<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Single-cell RNA-seq denoising using a deep count autoencoder | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Single-cell RNA-seq denoising using a deep count autoencoder</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Single-cell RNA-seq denoising using a deep count autoencoder</h1><div class="post-meta"><a href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/#comments" class="comment-count"></a><p><span class="date">Apr 15, 2019</span><span><a href="/categories/Nature-Communications/" class="category">Nature Communications</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2019年发表在Nature Communications上的使用自编码器对单细胞RNA-Seq数据进行降噪的工作，通讯作者是<a href="https://www.helmholtz-muenchen.de/icb/institute/staff/staff/ma/2494/index.html" target="_blank" rel="noopener">Fabian J. Theis</a>, 来自于德国。这篇文章对自编码器做了改进，常规的自编码器只有一层输出，该文章提出的DCA方法有三层输出。</p>
<p><a href="https://www.nature.com/articles/s41467-018-07931-2" target="_blank" rel="noopener">文章链接</a><br><a href="https://github.com/theislab/dca" target="_blank" rel="noopener">文章代码1</a><br><a href="https://scanpy.readthedocs.io/en/latest/api/index.html#imputation" target="_blank" rel="noopener">文章代码2</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>scRNA-seq (Single-cell RNA sequencing)使得研究人员可以在单个细胞分辨率水平上研究基因表达；然而由amplification以及dropout带来的噪音可能阻碍下游的分析，因此需要对scRNA-seq（维度高、稀疏）开发专门的方法；</li>
<li>作者在这里提出DCA(deep count autoencoder network)对scRNA-seq进行降噪；DCA使用<strong>binomial noise model with or without zero-inflation</strong>考虑到<strong>count distribution</strong>, <strong>overdispersion</strong> and <strong>sparsity of the data</strong>；</li>
<li>使用模拟数据和真实数据作者证明了DCA降噪可以提高各种各样传统的scRNA-seq分析，DCA在数据填充和速度上也比目前已有的方法。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>单细胞转录组的进步使得研究人员可以发现新的细胞类型，研究复杂细胞分化、发育过程追溯以及提高对人类疾病的理解；</li>
<li>单细胞测序技术虽然有了很大的提升，但是由各种各样的技术因素（例如：amplification bias, cell cycle effects, library size differences and especially low RNA capture rate）导致在scRNA-seq实验中存在着大量的噪音；这些技术由于测序深度较低所以非常稀疏；总之，技术因此带来了大量的噪音，这可能会破坏本身的生物学信号因此阻碍下游分析；</li>
<li>RNA捕获率低会导致检测不到某个基因的表达，也就是所谓的”false” zero count, 叫做”<strong>dropout event</strong>“。但是也有一些zero count是由于该基因在这个细胞中没有表达，因此在scRNA-seq中并不是所有的zero count都是缺失值。在统计学上，缺失值通常可以补充，要么通过随机替换要么是根据数据结构推断， 由于true zero count与false zero count之间的不平凡的差别，传统的方法对于scRNA-seq的imputation并不合适；</li>
<li>降噪的概念在图像领域被普遍用来从噪音中描绘信号；降噪通过从原始图像中抑制或者去除噪音来提升图片的质量；作者这里假设，从无噪数据得来的结果可以反应潜在的生物学过程以及细胞状态。然而，诸如成像或测序的测量技术会产生corrupted representation (<strong>Figure 1A</strong>)；<br><img src="/images/20190415-Fig1.png" alt="Figure 1"><br><img src="/images/20190415-Fig2.png" alt="Figure 1-legend"></li>
<li>目前已有一些针对于scRNA-seq的填充或者降噪方法。这些方法依赖于单细胞基因表达数据之间的相关性结构利用细胞之间或基因之间的相关性来推断矫正好的基因表达。例如：<a href="https://www.nature.com/articles/s41467-018-03405-7" target="_blank" rel="noopener">scImpute</a>使用混合模型定义可能的dropout values，然后将likely dropout values替换；而<a href="https://www.cell.com/cell/fulltext/S0092-8674(18" target="_blank" rel="noopener">MAGIC</a>30724-4)和<a href="https://www.nature.com/articles/s41592-018-0033-z" target="_blank" rel="noopener">SAVER</a>对每个基因和细胞输入输出降噪后的结果。但是这些方法都忽略了数据之间的count的非线性关系，而且，随着scRNA数据量的增大，这些方法不能分析上百万个细胞，这些方法分析不了如此大尺度的数据；</li>
<li>人工神经网络中的自编码器可以以一种非监督的模式学习到数据有效的压缩。自编码器的压缩部分强迫自编码器只能学习到重要的潜在特征，这种重构可以忽略随机噪音，目前自编码器在分子生物学领域已有很多应用；</li>
<li>作者为了能将自编码器应用于scRNA数据的填充而开发了<strong>deep count autoencoder</strong>, 方法的核心是将重构的错误定义为噪音模型的概率分布而不是输入数据本身 (<strong>FIgure 1B</strong>)，在训练的过程中, DCA通过最小化重构错误来学习基因特异性的分布参数；DCA有三个隐层可以捕获scRNA-seq数据中的复杂性以及非线性；</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-Count-noise-model-is-neccessary-to-denoise-scRNA-seq-data"><a href="#3-1-Count-noise-model-is-neccessary-to-denoise-scRNA-seq-data" class="headerlink" title="3.1 Count noise model is neccessary to denoise scRNA-seq data"></a>3.1 Count noise model is neccessary to denoise scRNA-seq data</h4><ul>
<li>作者首先将DCA应用于模拟scRNA-seq(使用<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1305-0" target="_blank" rel="noopener">Splatter</a>)数据探究DCA方法的属性；首先可以获得有dropout和没有dropout的数据,作者模拟了两组数据:<ul>
<li>两个细胞类型(2000个细胞)中的200个基因， 63%被设置为0；</li>
<li>六个细胞类型(2000个细胞)中的200个基因，35%被设置为0；</li>
</ul>
</li>
<li>为了指导用户选择合适的噪声模型，作者提出检查从同一cluster或细胞类型的细胞计算的基因平均值和经验dropout之间的关系，通过构建NB（negative binomial distribution without zero-inflation）<br>与ZINB（negative binomial distribution with zero-inflation）思然比检验来决定zero-inflation是否存在从而决定DCA选择的噪音模型的参数；</li>
<li><strong>Figure 2A-2B:</strong> 从模拟数据结果来看，dropout带来了大量的噪音阻碍了西北类型的发现，但是在用DCA降噪之后又可以恢复之前的细胞类型；为了检验count-based损失函数是不是必要的，作者比较了DCA与传统的自编码器（MSE作为损失函数，log转换），从最右边的图可以看出基于自编码器的MSE不能恢复细胞类型，表明<strong>specialized count loss function</strong>对于scRNA-seq数据的必要性<br><img src="/images/20190415-Fig3.png" alt="Figure 2"><br><img src="/images/20190415-Fig4.png" alt="Figure 2-legend"></li>
<li><strong>Figure 2C-2D:</strong> 上述结论在更复杂的六个细胞类型的数据上也得到了验证；</li>
<li>模拟数据的优点就是可以进行各种各样的评估，包括潜在的overimpudtation的评估；</li>
</ul>
<h4 id="3-2-DCA可以捕获真实数据中细胞群体结构-DCA-captures-cell-population-structure-in-real-data"><a href="#3-2-DCA可以捕获真实数据中细胞群体结构-DCA-captures-cell-population-structure-in-real-data" class="headerlink" title="3.2  DCA可以捕获真实数据中细胞群体结构 (DCA captures cell population structure in real data)"></a>3.2  DCA可以捕获真实数据中细胞群体结构 (DCA captures cell population structure in real data)</h4><ul>
<li>作者在这里使用了68579个外周血单核细胞和1000个highly variable genes （92%都是0）(<strong>Figure 3A</strong>)。<br><img src="/images/20190415-Fig5.png" alt="Figure 3"><br><img src="/images/20190415-Fig6.png" alt="Figure 3-legend"></li>
<li><strong>Figure 3B:</strong> 作者将自编码器的瓶颈层限制成2个神经元然后可视化其输出；</li>
<li><strong>Figure 3C-3F:</strong> 已知的细胞类型marker基因表现出了cluster-specific表达，表明DCA在真是数据集中捕获到了细胞群体的结构；</li>
</ul>
<h3 id="4-方法"><a href="#4-方法" class="headerlink" title="4. 方法"></a>4. 方法</h3><h4 id="4-1-自编码器"><a href="#4-1-自编码器" class="headerlink" title="4.1 自编码器"></a>4.1 自编码器</h4><ul>
<li>自编码器使用三层来预测输入数据，分别是：输入层、隐层(瓶颈层)以及输出层。输入输出都是相同大小维度，隐层是输入数据的低维展示，也就是降维，传统的自编码器通过最小化MSE损失函数来优化权重，自编码器中的压缩模型强制其只能学习到重要的潜在特征；PCA可以看作是以MSE为损失函数的线性自编码器；</li>
</ul>
<h4 id="4-2-噪音模型"><a href="#4-2-噪音模型" class="headerlink" title="4.2 噪音模型"></a>4.2 噪音模型</h4><ul>
<li>ZINB分布模拟高度稀疏和过度分散的count数据，其考虑了两个成分：<ul>
<li>点质量为零，表示数据中的过零值 (a point mass at zero which represents excess zero values in the data)；</li>
<li>表示计数分布的负二项式分量 (a negative binomial component representing the count distribution)</li>
</ul>
</li>
<li>成分1可捕获dropout事件；</li>
<li>成分2代表从潜在的分子中抽样reads的过程；</li>
<li>ZINB分布用下式表示：<script type="math/tex; mode=display">NB(x;\mu,\theta)=\frac{\Gamma(x+\theta)}{\Gamma(\theta)}(\frac{\theta}{\theta+\mu})^\theta(\frac{\mu}{\theta+\mu})^x</script><script type="math/tex; mode=display">ZINB(x;\pi,\mu,\theta)=\pi\sigma_0(x)+(1-\pi)NB(x;\mu,\theta)</script></li>
<li>\(\mu\)和\(\theta\)分别代表负二项分布的均值和离散度；</li>
</ul>
<h4 id="4-3-模型结构与训练"><a href="#4-3-模型结构与训练" class="headerlink" title="4.3 模型结构与训练"></a>4.3 模型结构与训练</h4><ul>
<li>接下来作者用自编码器框架以每个基因输入数据为条件估计ZINB分布的三个参数，因此与传统的自编码器不同的是，这个模型除了估计均值\(\mu\)外，还要估计dropout(\(\pi\))和dispersion参数(\(\theta\));</li>
<li>与传统的自编码器不同的是DCA有三个输出层，分别代表每个基因的三个参数，三个参数组成了gene-specific损失函数来比较这个基因原始输入与输出的差异；</li>
<li>对而分类模型来说，输出层可以解释为逻辑回归，因此DCA输出层可以看作是ZINB回归，predictor就是细胞的重新展示：<script type="math/tex; mode=display">E=ReLU(\bar{X}W_E)</script><script type="math/tex; mode=display">B=ReLU(EW_B)</script><script type="math/tex; mode=display">D=ReLU(BW_D)</script><script type="math/tex; mode=display">\bar{M}=exp(DW_{\mu})</script><script type="math/tex; mode=display">\prod=sigmoid(DW_{\pi})</script><script type="math/tex; mode=display">\Theta=exp(DW_\theta)</script></li>
<li>E, B, D分别代表encoder、bottleneck和decoder层，\(\bar{x}\)代表log转换的表达值的z-score。每个细胞的大小因子(size factor), 用每个细胞总的count数目除以每个细胞counts的median，\(\bar{X}\)通过一下公式来计算：<script type="math/tex; mode=display">\bar{X}=zscore(log(diag(s_i)^{-1}X+1))</script></li>
<li>X代表基因表达count矩阵，”zscore”代表z-score标准化；</li>
<li>输出激活以矩阵的形式展现</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">https://jingjingzhai.github.io/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/深度学习/">深度学习</a><a href="/tags/scRNA-Seq/">scRNA-Seq</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/" class="pre">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a><a href="/2019/04/12/Opportunities-and-challenges-for-transcriptome-wide-association-studies/" class="next">Opportunities and challenges for transcriptome- wide association studies</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-Count-noise-model-is-neccessary-to-denoise-scRNA-seq-data"><span class="toc-text">3.1 Count noise model is neccessary to denoise scRNA-seq data</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-DCA可以捕获真实数据中细胞群体结构-DCA-captures-cell-population-structure-in-real-data"><span class="toc-text">3.2  DCA可以捕获真实数据中细胞群体结构 (DCA captures cell population structure in real data)</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#4-方法"><span class="toc-text">4. 方法</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#4-1-自编码器"><span class="toc-text">4.1 自编码器</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-2-噪音模型"><span class="toc-text">4.2 噪音模型</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-3-模型结构与训练"><span class="toc-text">4.3 模型结构与训练</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/07/17/A-Deep-Neural-Network-for-Predictingand-Engineering-Alternative-Polyadenylation/">A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">Single-cell RNA-seq denoising using a deep count autoencoder</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/APA预测/" style="font-size: 15px;">APA预测</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/08/">八月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>