<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</h1><div class="post-meta"><a href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/#comments" class="comment-count"></a><p><span class="date">Aug 11, 2019</span><span><a href="/categories/The-Plant-Cell/" class="category">The Plant Cell</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是来自于明尼苏达大学的Chad L. Myers在The Plant Cell发表的关于整合共表达网络与GWAS来挖掘玉米中的causal genes.<br><a id="more"></a><br><a href="http://www.plantcell.org/content/30/12/2922" target="_blank" rel="noopener">文章链接</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>在不同物种中，GWAS已经确定出了很多与性状相关的位点，然而，由于每个确定的位点周围连锁不平衡的存在，我们并不知道causal genes到底是哪些。这个问题在非人类，非模式物种中尤其明显，因为这些物种中的功能注释相对较少，只有很少的信息可以用来挖掘候选基因；</li>
<li>本文作者开发了一个方法叫Camoco, 从大规模的玉米GWAS中挖掘候选基因。值得注意的是，作者发现他们方法的性能依赖于共表达网络的类型：<strong>使用群体RNA-Seq构建的共表达网络的性能比其他网络都要好</strong>；</li>
<li>Camoco确定的两个候选基因可以通过突变体实验验证。这项研究证明了共表达网络可以为从GWAS中挖掘causal genes提供有力的基础，但是也表明了这种策略依赖于所用的基因表达数据。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>GWAS是理解性状变异遗传基础的有力工具。以玉米为例，GWAS确定了大概40个与开花时间相关的loci，89个与株高有关的loci，36个与leaf length有关的loci，32 loci for resistance to southern leaf blight, and 26 loci for kernel protein. 虽然在统计上已经确定了很多与性状相关的loci，但是causal genes的确定已经与这些loci相关的功能等位基因的生物学解释仍然存在很大挑战；</li>
<li>为GWAS提供动力的连锁不平衡（LD）是限制因果基因鉴定的主要障碍。遗传标记由GWAS鉴定，但通常位于基因边界之外，而且可能距离真实的causal snp很远。在玉米中，LD变化可以从1kb到1Mb，在其他农作物中这个范围可能更大，而且，越来越多的证据表明<strong>基因调控区域在功能变异中扮演着重要的角色，导致causal variants落在基因边界的外部。</strong>在玉米中，一经报道有些QTL由非编码序列组成。这些因素意味着即使一个位点与性状很相关，在确定真正的causal genes之前仍然会有很多看似可信的候选基因被确定；</li>
<li>对于基因注释不完整的农作物来说，缩小候选基因的范围非常重要。举个例子，在玉米中，只有大约～1%的基因有突变体的功能注释；</li>
<li>对于功能注释来说，一个简单而且有信息的策略就是基因表达。使用不同的遗传背景或者不同的组织大雨可以帮助我们确立基因的生物学功能。比较两个基因的表达（共表达）可以衡量基因的联合响应。而且共表达的分析已被成功应用于很多功能相关基因的确定，而且已被应用于解析拟南芥GWAS的结果；</li>
<li>由于共表达为功能关系提供了全局的测量，因此可以作为解析GWAS候选位点的有力工具。尤其是，作者期望在同一生物学过程中不同基因表达的变化与给定表型是相关的。因此，如果GWAS捕获的SNP是由共调控的基因编码的话，这些数据集的重叠就不会是随机的。虽然并不是所有的功能都具有共表达关系，但是这些数据将会提供有价值的线索。<strong>而且在人类和小鼠中，共表达已经被作为理解GWAS的基础</strong>；</li>
<li>作者开发了一个计算框架Camoco (Coanalysis of molecular components),通过整合基因共表达网络和GWAS挖掘候选基因。Camoco用传统的GWAS评估候选SNPs，然后确定一组高置信度的候选基因；</li>
<li>作者将Camoco应用在玉米上，作者关注到玉米籽粒中17个元素(Al, As, B, Ca, Cd, Fe, K, Mg, Mn, Mo, Na, Ni, Rb, S, Se, Sr, and Zn)的累积。植物必须摄取除土壤中碳和氧以外的所有元素，使植物离子组成为理解植物环境响应，谷物营养品质和植物生理学的关键组成部分；</li>
<li>作者评估了三个不同类型共表达网络的利用并且用模拟数据证明了Camoco的有效性。这项研究也证明了共表达网络中的功能模块与GWAS候选SNP的一致性。同时确定了高置信度的候选基因，并通过单基因突变体证明了方法的有效性。</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-Camoco-整合GWAS结果与共表达网络的框架-Camoco-A-framework-for-integrating-GWAS-results-and-comparing-coexpression-networks"><a href="#3-1-Camoco-整合GWAS结果与共表达网络的框架-Camoco-A-framework-for-integrating-GWAS-results-and-comparing-coexpression-networks" class="headerlink" title="3.1 Camoco: 整合GWAS结果与共表达网络的框架 (Camoco: A framework for integrating GWAS results and comparing coexpression networks)"></a>3.1 Camoco: 整合GWAS结果与共表达网络的框架 (Camoco: A framework for integrating GWAS results and comparing coexpression networks)</h4><ul>
<li>Camoco输入的是：一组与性状相关的SNPs和一个基因表达的表，输出GWAS信号周围高可信度的具有很强共表达关系的候选基因。如下图所示：Camoco框架有三个模块:</li>
<li><strong>Figure 1A:</strong> SNP-to-gene mapping, 用户指定一个window size以及每个SNP最大的候选基因数目，然后确定所有的candidate loci;</li>
<li><strong>Figure 1B:</strong> 共表达网络的构建与分析；</li>
<li><p><strong>Figure 1C:</strong> Camoco的overlap算法，使用两个网络打分策略：<strong>subnetwork density and subnetwork locality</strong>. 前者衡量GWAS信号附近基因的两两组和的平均相互作用强度。<strong>所谓density计算是subnetwork中平均的相互作用分值然后在用subnetwork的大小做矫正</strong>；后者计算：</p>
<blockquote>
<p><strong>Subnetwork locality measures the proportion of significant (Z &gt; 3) coex- pression interactions among genes within a GWAS-derived subnetwork (local interactions) as compared with the number of global interactions with other genes in the genome (global interactions)</strong></p>
</blockquote>
</li>
<li><p>locality的计算：</p>
<blockquote>
<p><strong>Specifically, locality is obtained by first fitting a linear regression between all genes’ local degree (among the subnetwork of interest) and their global degree and measuring the mean of the residual for genes in the subnetwork (Equation 2).</strong></p>
</blockquote>
</li>
<li><p><strong>Figure 1D:</strong> 通过比较GWAS确定的基因的共表达强度与随机网络中共表达的强度来计算子网络相互作用的共表达强度。<br><img src="/images/20190811-Fig1.png" alt="Figure 1"><br><img src="/images/20190811-Fig2.png" alt="Figure 1-legend"></p>
</li>
</ul>
<h4 id="3-2-从不同类型的转录组数据中建立共表达网络-Generating-coexpression-networks-from-diverse-transcriptional-data"><a href="#3-2-从不同类型的转录组数据中建立共表达网络-Generating-coexpression-networks-from-diverse-transcriptional-data" class="headerlink" title="3.2 从不同类型的转录组数据中建立共表达网络 (Generating coexpression networks from diverse transcriptional data)"></a>3.2 从不同类型的转录组数据中建立共表达网络 (Generating coexpression networks from diverse transcriptional data)</h4><ul>
<li>作者推断不同的表达谱来源可能对共表达网络解释GWAS捕获的遗传变异的效用产生很大影响，作者在这部分就建了多个共表达网络评估了Camoco框架的有效性。作者使用了三组不同的数据，分别是：<ul>
<li>503 diverse inbred lines (maize pan-genome);</li>
<li>不同组织、发育时间点的基因表达数据；</li>
<li>ionomics GWAS research program中的数据。</li>
</ul>
</li>
</ul>
<h4 id="3-3-考虑顺式基因相互作用-Accounting-for-cis-gene-interactions"><a href="#3-3-考虑顺式基因相互作用-Accounting-for-cis-gene-interactions" class="headerlink" title="3.3 考虑顺式基因相互作用 (Accounting for cis gene interactions)"></a>3.3 考虑顺式基因相互作用 (Accounting for cis gene interactions)</h4><ul>
<li>顺式基因比反式基因拥有更强的共表达。<br><img src="/images/20190811-Fig3.png" alt="Figure 2"></li>
</ul>
<h4 id="3-4-Camoco框架的评估-Evaluation-of-the-Camoco-Framework"><a href="#3-4-Camoco框架的评估-Evaluation-of-the-Camoco-Framework" class="headerlink" title="3.4 Camoco框架的评估 (Evaluation of the Camoco Framework)"></a>3.4 Camoco框架的评估 (Evaluation of the Camoco Framework)</h4><p><img src="/images/20190811-Fig4.png" alt="Figure 3"></p>
<h3 id="4-讨论"><a href="#4-讨论" class="headerlink" title="4. 讨论"></a>4. 讨论</h3><h3 id="5-方法"><a href="#5-方法" class="headerlink" title="5. 方法"></a>5. 方法</h3><h4 id="5-1-共表达网络的构建与质控-Construction-and-quality-control-of-coexpression-networks"><a href="#5-1-共表达网络的构建与质控-Construction-and-quality-control-of-coexpression-networks" class="headerlink" title="5.1 共表达网络的构建与质控(Construction and quality control of coexpression networks)"></a>5.1 共表达网络的构建与质控(Construction and quality control of coexpression networks)</h4><ul>
<li><strong>ZmPAN:</strong> 24,756个基因用于构建网络，对于每一对基因计算Pearson相关系数，共产生大概306 million个边。然后Pearson相关系数然后被Fisher转换以及标准化(Z-score)从而可以实现cross-networkcomparison. Z&gt;=3被认为是全局显著的。为了评估网络总体的可靠程度，作者采用了以下几个方法：<ul>
<li>玉米注释的GO terms中基因的z scores与包含相同数目基因的1000个随机term进行比较；</li>
<li>Degree distribution.</li>
</ul>
</li>
<li><strong>ZmSAM:</strong> 与上述类似</li>
<li><strong>ZmRoot:</strong> 与上述类似</li>
</ul>
<h4 id="5-2-SNP-to-Gene-Mapping-and-Effective-Loci"><a href="#5-2-SNP-to-Gene-Mapping-and-Effective-Loci" class="headerlink" title="5.2 SNP-to-Gene Mapping and Effective Loci"></a>5.2 SNP-to-Gene Mapping and Effective Loci</h4><ul>
<li>两个参数：<strong>candidate window size and maximum number of flanking genes</strong></li>
</ul>
<blockquote>
<p><strong>Candidate genes were ranked by the absolute value of their distance to the center of their parental effective locus. Algorithms implementing the SNP-to-gene mapping used here are accessible through the Camoco command line interface.</strong></p>
</blockquote>
<h4 id="5-3-Calculating-subnetwork-density-and-locality"><a href="#5-3-Calculating-subnetwork-density-and-locality" class="headerlink" title="5.3 Calculating subnetwork density and locality"></a>5.3 Calculating subnetwork density and locality</h4><ul>
<li>Subnetwork density计算公式如下：<script type="math/tex; mode=display">Subnetwork\ Density\ (subnetwork\ S)=\frac{\sum\limits_{all\ gene\ pairs\ i,j\ \in S\ i \neq j}W_{ij}}{\frac{1}{\sqrt{N_e}}}</script></li>
<li>其中，\(W_{ij}\)是基因i与j的共表达分值, \(N_e\)是pairwise的数目。</li>
</ul>
<blockquote>
<p> <strong>To quantify network locality, both local and global degree are calculated for each gene within a subnetwork where local degree is the number of interactions to other genes in the subnetwork and global degree is the total number of interactions a gene has. To account for degree bias, where genes with a high global degree are more likely to have more local interactions, a linear regression is calculated on local degree using global degree (designated local ; global), and regression residuals for each gene are analyzed:</strong></p>
</blockquote>
<script type="math/tex; mode=display">Subnetwork\ Locality\ (subnetwork\ S)=\frac{\sum\limits_{all\ genes\ i \in S}Gene-Specific\ Locality\ (gene\ i)}{N_g}</script><ul>
<li>其中：\(N_g\)是subnetwork中的基因的数目<script type="math/tex; mode=display">Gene-Specific\ Density\ (gene\ i)=\frac{\sum\limits_{all\ genes j \neq i}W_{ij}}{N_g-1}</script></li>
<li>\(W_{ij}\): 共表达网络分值</li>
<li>\(N_g\): 共表达网络中基因的数目<script type="math/tex; mode=display">Gene-Specific\ Locality\ (gene\ i) = \epsilon_i</script></li>
<li>\(\epsilon_i\)是回归模型中gene \(i\) 的残差：<script type="math/tex; mode=display">degree_{local}(gene\ j)=\alpha*degree_{global}(gene\ j)+\epsilon_j</script></li>
<li>\(degree_{local}(gene\ j)\)是子网络中满足阈值的相互作用数目</li>
<li>\(degree_{global}(gene\ j)\)是基因j与基因组中所有基因相互作用满足阈值的数目</li>
</ul>
<h4 id="5-4-Simulating-GWAS-Using-GO-Terms"><a href="#5-4-Simulating-GWAS-Using-GO-Terms" class="headerlink" title="5.4 Simulating GWAS Using GO Terms"></a>5.4 Simulating GWAS Using GO Terms</h4><ul>
<li>选取具有50-100个基因的term，同时选取具有显著共表达的term用于分析, 选取MCR和FCR用于模拟噪音。<script type="math/tex; mode=display">MCR = 1-\frac{True\ Candidate\ Genes}{Candadate\ Genes}</script><script type="math/tex; mode=display">FCR = \frac{Candidate\ Genes - True\ Candidate\ Genes}{Candidate\ Genes}</script></li>
</ul>
<h4 id="5-5-Simulating-MCR"><a href="#5-5-Simulating-MCR" class="headerlink" title="5.5 Simulating MCR"></a>5.5 Simulating MCR</h4><blockquote>
<p><strong>The effects of MCR were evaluated by subjecting GO terms with significant coexpression (P # 0.05; described above) to varying levels of MCRs. True GO term genes were replaced with random genes at varying rates (MCR: 0, 10, 20, 50, 80, 90, and 100%). The effect of MCR was evaluated by as- sessing the number of GO terms that retained significant coexpression (compared with 1000 randomizations) at each level of MCR.</strong></p>
</blockquote>
<h4 id="5-6-Adding-False-Candidate-Genes-by-Expanding-SNP-to-Gene-Mapping-Parameters"><a href="#5-6-Adding-False-Candidate-Genes-by-Expanding-SNP-to-Gene-Mapping-Parameters" class="headerlink" title="5.6 Adding False Candidate Genes by Expanding SNP-to-Gene Mapping Parameters"></a>5.6 Adding False Candidate Genes by Expanding SNP-to-Gene Mapping Parameters</h4><blockquote>
<p><strong>To determine how false candidates due to imperfect SNP-to-gene map- ping affected the ability to detect coexpressed candidate genes linked to a GWAS trait, GO terms with significantly coexpressed genes were re- assessed after incorporating false candidate genes. Each gene in a GO term was treated as an SNP and remapped to a set of candidate genes using the different SNP-to-gene mapping parameters (all combinations of 50, 100, and 500 kb and one, two, or five flanking genes). Effective FCR at each SNP-to-gene mapping parameter setting was calculated by dividing the number of true GO genes with candidates identified after SNP-to-gene mapping. Since varying SNP-to-gene mapping pa- rameters changes the number of candidate genes considered within a term, each term was considered independently for each parameter combination.</strong></p>
</blockquote>
<h4 id="5-7-Maize-Ionone-GWAS"><a href="#5-7-Maize-Ionone-GWAS" class="headerlink" title="5.7 Maize Ionone GWAS"></a>5.7 Maize Ionone GWAS</h4><h4 id="5-8-Identifying-Ionome-HPO-Genes-and-HPO-Genes"><a href="#5-8-Identifying-Ionome-HPO-Genes-and-HPO-Genes" class="headerlink" title="5.8 Identifying Ionome HPO Genes and HPO+ Genes"></a>5.8 Identifying Ionome HPO Genes and HPO+ Genes</h4></div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/">https://jingjingzhai.github.io/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/数量遗传/">数量遗传</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/" class="next">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-Camoco-整合GWAS结果与共表达网络的框架-Camoco-A-framework-for-integrating-GWAS-results-and-comparing-coexpression-networks"><span class="toc-text">3.1 Camoco: 整合GWAS结果与共表达网络的框架 (Camoco: A framework for integrating GWAS results and comparing coexpression networks)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-从不同类型的转录组数据中建立共表达网络-Generating-coexpression-networks-from-diverse-transcriptional-data"><span class="toc-text">3.2 从不同类型的转录组数据中建立共表达网络 (Generating coexpression networks from diverse transcriptional data)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-考虑顺式基因相互作用-Accounting-for-cis-gene-interactions"><span class="toc-text">3.3 考虑顺式基因相互作用 (Accounting for cis gene interactions)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-Camoco框架的评估-Evaluation-of-the-Camoco-Framework"><span class="toc-text">3.4 Camoco框架的评估 (Evaluation of the Camoco Framework)</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#4-讨论"><span class="toc-text">4. 讨论</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#5-方法"><span class="toc-text">5. 方法</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#5-1-共表达网络的构建与质控-Construction-and-quality-control-of-coexpression-networks"><span class="toc-text">5.1 共表达网络的构建与质控(Construction and quality control of coexpression networks)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#5-2-SNP-to-Gene-Mapping-and-Effective-Loci"><span class="toc-text">5.2 SNP-to-Gene Mapping and Effective Loci</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#5-3-Calculating-subnetwork-density-and-locality"><span class="toc-text">5.3 Calculating subnetwork density and locality</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#5-4-Simulating-GWAS-Using-GO-Terms"><span class="toc-text">5.4 Simulating GWAS Using GO Terms</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#5-5-Simulating-MCR"><span class="toc-text">5.5 Simulating MCR</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#5-6-Adding-False-Candidate-Genes-by-Expanding-SNP-to-Gene-Mapping-Parameters"><span class="toc-text">5.6 Adding False Candidate Genes by Expanding SNP-to-Gene Mapping Parameters</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#5-7-Maize-Ionone-GWAS"><span class="toc-text">5.7 Maize Ionone GWAS</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#5-8-Identifying-Ionome-HPO-Genes-and-HPO-Genes"><span class="toc-text">5.8 Identifying Ionome HPO Genes and HPO+ Genes</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/07/17/A-Deep-Neural-Network-for-Predictingand-Engineering-Alternative-Polyadenylation/">A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/24/Functional-architecture-of-low-frequency-variants-highlights-strength-of-negative-selection-across-coding-and-non-coding-annotations/">Functional architecture of low-frequency variants highlights strength of negative selection across coding and non-coding annotations</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a> <a href="/tags/APA预测/" style="font-size: 15px;">APA预测</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/08/">八月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>