<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>A resource of variant effect predictions of single nucleotide variants in model organisms | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">A resource of variant effect predictions of single nucleotide variants in model organisms</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">A resource of variant effect predictions of single nucleotide variants in model organisms</h1><div class="post-meta"><a href="/2019/03/04/A-resource-of-variant-effect-predictions-of-SNV-in-model-organism/#comments" class="comment-count"></a><p><span class="date">Mar 04, 2019</span><span><a href="/categories/MSB/" class="category">MSB</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2018年12月发表在期刊<a href="http://msb.embopress.org/" target="_blank" rel="noopener">molecular system biology</a>上的一篇关于SNV效应预测的资源数据库。通讯作者和第一作者都来自<a href="https://www.ebi.ac.uk/" target="_blank" rel="noopener">European Bioinformatics Institute</a>的<a href="https://www.embl.de" target="_blank" rel="noopener">分子生物学实验室</a>。<a id="more"></a>该文章的亮点主要在于从分子机制层面解析SNV，在人类疾病相关的SNP研究中，早在2012年就有相关的对SNP进行注释的工作（以<a href="https://genome.cshlp.org/content/22/9/1790.abstract" target="_blank" rel="noopener">RegulomeDB</a>为代表），那时还只是针对SNP与功能基因组在物理位置的overlap来注释。随后也有各种基于机器学习/深度学习对各种调控序列（TFBS、DHS、Methylation）建模, 然后根据SNP上下游的序列预测两个分值，再根据这个分值构建二分类模型，从而实现注释SNP。这样的软件有很多，随后又有文章<a href="https://www.nature.com/articles/s41467-017-01261-5" target="_blank" rel="noopener">FUMA</a>整合了各种各样的预测软件，但个人认为FUMA这篇文章虽然发表了NC，而且引用也很高，但是却没有MSB这篇文章的意义、创新性大。</p>
<p><a href="http://msb.embopress.org/content/14/12/e8430" target="_blank" rel="noopener">文章链接</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>在遗传学中，不论是编码区域还是非编码区域的SNV的效应都是生物学家感兴趣的。目前虽然也有很多计算方法用于阐述SNV在细胞机制中的效应，但是并没有直接全面覆盖SNV的分子效应 (molecular effects);</li>
<li>为了解决这个问题，作者汇编(compile; to collect information from different places and arrange it in a book, report, or list)了基于序列和基于结构的SNP效应predictor，并预测了人类和酵母中相对于参考基因组几乎所有的氨基酸和核苷酸变异；</li>
<li>这篇文章涉及到的机制研究包括: <strong>protein stability</strong>, <strong>interaction interfaces</strong>, <strong>post-translational modifications</strong> and <strong>transcription factor binding sites</strong>。作者向我们展示了如何使用这个数据资源来生成<strong>protein complex burden scores</strong> 进而与表型进行关联;</li>
<li>数据资源可以通过以下链接访问：www.mutfunc.com</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>GWAS是将表型与基因型联系的重要手段，但是GWAS既不能找到causal variant，又难以解释causal variant的机制</li>
<li>编码区和非编码区的SNP都可以影响各种各样的生物学功能；</li>
<li>通过预测的手段去挖掘遗传变异改变的机制将是省时有省财的方法；虽然目前也有很多关于这方面的预测，但是都不能提供全面的预测效应以及解析机制而且使用起来需要专业人员操作；</li>
<li>因此这篇文章中作者大量的预测了人类、酵母和大肠杆菌中几乎所有SNV的变异，从不同层面（包括<strong>the context of conserved protein regions, protein stability, protein– protein interaction (PPI) interfaces, PTMs, kinase–substrate interactions, short linear motifs (SLiMs), start and stop codons, and tran- scription factor (TF) binding sites (TFBSs)</strong>）解析SNP的机制。</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-在酵母和人类个体中功能基因组区域展现出了进化约束-Functional-genomic-regions-display-evolutionary-constraint-across-yeast-and-human-individuals"><a href="#3-1-在酵母和人类个体中功能基因组区域展现出了进化约束-Functional-genomic-regions-display-evolutionary-constraint-across-yeast-and-human-individuals" class="headerlink" title="3.1 在酵母和人类个体中功能基因组区域展现出了进化约束 (Functional genomic regions display evolutionary constraint across yeast and human individuals)"></a>3.1 在酵母和人类个体中功能基因组区域展现出了进化约束 (Functional genomic regions display evolutionary constraint across yeast and human individuals)</h4><ul>
<li>这部分作者主要从RSA、PTM、TFBS三个层面论证有功能的区域富集的自然变异更少</li>
</ul>
<p><img src="/images/20190304-Fig1.png" alt="Population-level sequence constraint in genome functional elements"></p>
<ul>
<li><strong>RSA: relative surface accessibility,蛋白质表面氨基酸残基的可及性。</strong></li>
<li><strong>Figure 1A-1B:</strong> 文章根据RSA将氨基酸残基分成bin，统计这个bin里自然变异（natural variation）的数目,与随机选取的残基进行比较（也就是纵坐标表示的expected variant count), 然后随机1000次（permutation），计算p值。发现隐藏（buried region; <strong>对应1A</strong>, RSA越小表明是隐藏区域）与结合区域（interface region;对应<strong>对应1B</strong>, delta RSA越大表明事结合区域）与暴露在外面的区域相比，前者的自然变异数目更少；</li>
<li><strong>Figure 1C:</strong> 这个图主要是想分析与随机相比，不同类型的PTM (翻译后修饰；去了上下游5个残基)富集的自然变异是否有差异；</li>
<li><strong>Figure 1D:</strong> 从图C中我们并没有看出不同类型的PTM有很强的约束性，随后作者又分析了每个PTM上下游5个残基的自然变异富集情况，发现如果PTM的周围也有其他PTM存在的话，约束会变强（自然变异的数目变少）;</li>
<li><strong>Figure 1E:</strong> 接下来作者分析了TFBS内自然变异的富集情况，预测出来的TFBS里的SNV与chip-seq peak里的其他区域进行比较，有些转录因子结合区域是非常constraint；</li>
<li><strong>Figure 1F:</strong> 进一步研究分析发现，那些TFBS聚成cluster的位点约束性更强；</li>
<li><strong>Figure 1G:</strong> 作者首先使用了information content (IC)概念，是指：position-specific information content (IC) of the TF specificity position weight matrices，根据PWM计算出来的值，越大代表与TF结合能力越强。这个图表明IC越大，约束性越强，与预期的结果一致；</li>
</ul>
<p><img src="/images/20190304-Fig9.png" alt="Figure 1H"></p>
<ul>
<li><p><strong>Figure 1H:</strong> 这部分作者举了四个例子表明高IC与低IC位置的约束性差异。</p>
</li>
<li><p>其他信息：(A, B, F) P-values represent a one-sided Wilcoxon test. (A, B, C, D, F) Error bars represent the standard deviation. One hundred random samples were used. (G) P-value shown is computed using a one-sided Kolmogorov–Smirnov test.</p>
</li>
</ul>
<h4 id="3-2-单核苷酸变异的机制效应的综合资源-A-comprehensive-resource-of-mechanistic-effects-of-single-nucleotide-variants"><a href="#3-2-单核苷酸变异的机制效应的综合资源-A-comprehensive-resource-of-mechanistic-effects-of-single-nucleotide-variants" class="headerlink" title="3.2 单核苷酸变异的机制效应的综合资源 (A comprehensive resource of mechanistic effects of single nucleotide variants)"></a>3.2 单核苷酸变异的机制效应的综合资源 (A comprehensive resource of mechanistic effects of single nucleotide variants)</h4><p><img src="/images/20190304-Fig2.png" alt="Fig2A"></p>
<ul>
<li><strong>Figure 2A:</strong> 大范围的预测SNV对于保守蛋白区域、蛋白稳定性、蛋白相互作用表面、激酶-底物磷酸化以及其他的PTMs、linear motifs、TFBS、start and stop codons</li>
</ul>
<h4 id="3-3-功能重要的位置富含预测的有害变异-Functionally-important-positions-are-enriched-in-predicted-deleterious-variants"><a href="#3-3-功能重要的位置富含预测的有害变异-Functionally-important-positions-are-enriched-in-predicted-deleterious-variants" class="headerlink" title="3.3 功能重要的位置富含预测的有害变异 (Functionally important positions are enriched in predicted deleterious variants)"></a>3.3 功能重要的位置富含预测的有害变异 (Functionally important positions are enriched in predicted deleterious variants)</h4><p><img src="/images/20190304-Fig3.png" alt="Fig2BCD"></p>
<ul>
<li><strong>Figure 2B:</strong> 首先用essential gene中是否拥有更少的致病自然变异，结论与之前一致，essential gene中致病的自然变异更少；</li>
<li><strong>Figure 2C:</strong> 致病自然变异与MAF的关系，从图中可以看出，MAF越大，致病的比例越低，这也暗示了罕见变异的重要性；</li>
<li><strong>Figure 2D:</strong> 此部分评估了不同软件对于致病变异的分类能力，SIFT拥有最好的分类性能。SIFT performed the best at discriminating pathogenic variants from benign (AUC H. sapiens = 0.87, S. cerevisiae = 0.92), followed by FoldX interfaces (AUC H. sapi- ens = 0.64, S. cerevisiae = 0.72) and FoldX stability (AUC H. sapi- ens = 0.70, S. cerevisiae = 0.62, Fig 2D).</li>
</ul>
<p><img src="/images/20190304-Fig4.png" alt="Fig2EF"></p>
<ul>
<li><strong>Figure 2E-2F:</strong> 虽然功能重要的区域中（preserves motif, disrupts motif, disrupts non-PTM res, disrupts PRM res, Start lost, Stop lost, Stop gain）致病自然变异的数目很少，但从统计意义来说，这些区域还是富含质变变异的。</li>
</ul>
<h4 id="3-4-对于不确定显著性的SNP预测其机制效应-Predicting-mechanistic-impacts-of-variants-of-uncertain-significance"><a href="#3-4-对于不确定显著性的SNP预测其机制效应-Predicting-mechanistic-impacts-of-variants-of-uncertain-significance" class="headerlink" title="3.4 对于不确定显著性的SNP预测其机制效应 (Predicting mechanistic impacts of variants of uncertain significance)"></a>3.4 对于不确定显著性的SNP预测其机制效应 (Predicting mechanistic impacts of variants of uncertain significance)</h4><p><img src="/images/20190304-Fig5.png" alt="Fig3"></p>
<ul>
<li><strong>VUS: variants of uncertain significance</strong>, 已经通过遗传检验检测出来的SNP，但是还不确定是benign或者pathogenic。</li>
<li>Focus到影响蛋白结构完整性的变异</li>
<li>举例子证明找到的会影响蛋白结构的VUS</li>
</ul>
<h4 id="3-5-酿酒酵母菌株间的基因组差异是表型相似性的重要但弱的预测因子"><a href="#3-5-酿酒酵母菌株间的基因组差异是表型相似性的重要但弱的预测因子" class="headerlink" title="3.5 酿酒酵母菌株间的基因组差异是表型相似性的重要但弱的预测因子"></a>3.5 酿酒酵母菌株间的基因组差异是表型相似性的重要但弱的预测因子</h4><ul>
<li>这部分内容用来阐述如何使用<a href="http://www.mutfunc.com" target="_blank" rel="noopener">mutfunc</a>来进行基因型-表型的关联分析。</li>
</ul>
<p><img src="/images/20190304-Fig6.png" alt="Fig4AC"></p>
<ul>
<li><strong>Figure 4A:</strong> 两个重复之间的S分值的一致性说明了数据一致性、可靠性。</li>
<li><strong>Figure 4C:</strong> 93个测序菌株的基因型距离以及表型距离的散点图。</li>
</ul>
<p><img src="/images/20190304-Fig7.png" alt="Fig4B"></p>
<ul>
<li><strong>Figure 4B:</strong> S-score在不同菌株不同条件下的热图。</li>
</ul>
<h4 id="3-6-Gene-and-complex-disruption-scores-for-genotype-to-phenotype-associations"><a href="#3-6-Gene-and-complex-disruption-scores-for-genotype-to-phenotype-associations" class="headerlink" title="3.6 Gene and complex disruption scores for genotype-to- phenotype associations"></a>3.6 Gene and complex disruption scores for genotype-to- phenotype associations</h4><p>由于绝大部分变异都是中性的，因此此部分作者使用SIFT(for conservation), FoldX(protein stability)和protein truncating variants (PTVs)从基因水平计算<strong>a total gene burden</strong>或者<strong>disruption score</strong></p>
<p><img src="/images/20190304-Fig8.png" alt="Fig5ABC"></p>
<ul>
<li><strong>Figure 5A:</strong> 基因水平计算<strong>a total gene burden</strong>或者<strong>disruption score</strong>的方法。</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/03/04/A-resource-of-variant-effect-predictions-of-SNV-in-model-organism/">https://jingjingzhai.github.io/2019/03/04/A-resource-of-variant-effect-predictions-of-SNV-in-model-organism/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/SNP-Annotation/">SNP Annotation</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/03/07/Evolutionarily-informed-deep-learning-methods-forpredicting-relative-transcript-abundance-fromDNA-sequence/" class="pre">Evolutionarily informed deep learning methods for predicting relative transcript abundance from DNA sequence</a><a href="/2019/02/22/Anchor trans-cell type prediction of transcription factor binding sites/" class="next">Anchor trans-cell type prediction of transcription factor binding sites</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-在酵母和人类个体中功能基因组区域展现出了进化约束-Functional-genomic-regions-display-evolutionary-constraint-across-yeast-and-human-individuals"><span class="toc-text">3.1 在酵母和人类个体中功能基因组区域展现出了进化约束 (Functional genomic regions display evolutionary constraint across yeast and human individuals)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-单核苷酸变异的机制效应的综合资源-A-comprehensive-resource-of-mechanistic-effects-of-single-nucleotide-variants"><span class="toc-text">3.2 单核苷酸变异的机制效应的综合资源 (A comprehensive resource of mechanistic effects of single nucleotide variants)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-功能重要的位置富含预测的有害变异-Functionally-important-positions-are-enriched-in-predicted-deleterious-variants"><span class="toc-text">3.3 功能重要的位置富含预测的有害变异 (Functionally important positions are enriched in predicted deleterious variants)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-对于不确定显著性的SNP预测其机制效应-Predicting-mechanistic-impacts-of-variants-of-uncertain-significance"><span class="toc-text">3.4 对于不确定显著性的SNP预测其机制效应 (Predicting mechanistic impacts of variants of uncertain significance)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-5-酿酒酵母菌株间的基因组差异是表型相似性的重要但弱的预测因子"><span class="toc-text">3.5 酿酒酵母菌株间的基因组差异是表型相似性的重要但弱的预测因子</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-6-Gene-and-complex-disruption-scores-for-genotype-to-phenotype-associations"><span class="toc-text">3.6 Gene and complex disruption scores for genotype-to- phenotype associations</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/07/16/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">Single-cell RNA-seq denoising using a deep count autoencoder</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/12/Opportunities-and-challenges-for-transcriptome-wide-association-studies/">Opportunities and challenges for transcriptome- wide association studies</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/09/Virtual-ChIP-seq-predicting-transcription-factor-binding-by-learning-from-the-transcriptome/">Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>