<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation</h1><div class="post-meta"><a href="/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/#comments" class="comment-count"></a><p><span class="date">Mar 29, 2019</span><span><a href="/categories/The-Plant-Cell/" class="category">The Plant Cell</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2017年发表于The Plant Cell期刊上的用共线性网络分析MADS-box基因家族的文章，通讯作者是荷兰瓦赫宁根大学<a href="https://www.wur.nl/en/Persons/Eric-prof.dr.-ME-Eric-Schranz.htm?subpage=projects" target="_blank" rel="noopener">M. Eric Schranz</a>。同时作者还在<strong>Current Opinion in Plant Biology</strong>发表一篇关于共线性网络的<a href="https://www.sciencedirect.com/science/article/pii/S1369526616302230" target="_blank" rel="noopener">综述</a>。<a id="more"></a> 文章创新点在于将网络分析与共线性结合起来，可以发现一些新的进化模式。</p>
<p><a href="http://www.plantcell.org/content/29/6/1278" target="_blank" rel="noopener">文章链接</a><br><a href="https://github.com/zhaotao1987/SynNet-Pipeline" target="_blank" rel="noopener">文章代码</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>保守的基因组背景为比较进化分析提供了关键信息。随着植物物种测序数目的增多，共线性分析可以为基因家族的进化提供新的视角；</li>
<li>作者在本文利用<strong>网络分析方法</strong>组织以及解释大量成对的共线性关系，具体来说，作者通过使用51个植物基因组分析了MADS-box转录因子基因家族的共线性网络，与系统发生树结合分析，发现了几个新的进化模式并通过共线性网络clusters进行可视化；</li>
<li>作者发现了一些lineage-specific clusters，这些clusters来源于Brassicales中花发育调节因子（APETALA3和PI）和开花时间（FLC）以及Poales根发育调节因子（AGL17）的转座事件；</li>
<li>同时作者也确定了两个大的gene clusters，他们共同包含<strong>many key phenotypic regulatory Type II MADS-box gene clades</strong>。基因簇以及基因树都表明，这些基因来自于老的串联重复，其可能早于种子植物的辐射，然后通过随后的多倍体事件扩展；</li>
<li>作者也发现了其他几个研究较少的分支的被子植物范围的同线性保守性；</li>
<li>这些发现为MADS-box基因家族成员的基因组来源、生物学保守性以及分化提供了新的假设。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>保守的基因顺序可以保留数亿年，并且为conserved genomic context以及基因组和基因的进化提供关键信息。举个例子，”Hox gene cluster”,regulates the animal body plan(???), 在动物王国中大部分都是共线性的。<strong>Synteny</strong>一词最初被定义为<strong>a set of genes from two species located on same chromosome, but not necessarily in the same order</strong>; 但目前广泛使用的<strong>Synteny</strong>一词定义如下：<strong>conserved collinearity and genomic context</strong>;</li>
<li>共线性数据已经被广泛用于建立老的多倍化事件的出现、确定染色体重排、检查基因家族的扩张或收缩以及建立基因直系同源。共线性既能反映出功能上又能反映出调节上的重要关系，也被用作<strong>proxy for the conservation or constraint of gene function</strong>。 因此，多个物种之间的共线性关系可以为基因家族进化调节的发育途径提供重要信息。举个例子，morphological novelty的起源与动物中Hox基因的关键调控转录因子的重复有关，也与植物中的MADS-box基因有关。然而，基因cluster在某些谱系(lineage)中经常被分散或“分解”，如章鱼和蟾蜍基因组中的Hox簇，这种分散有助于形成divergent的基因表达和morphological novelties;</li>
<li>在植物中，MADS-box是调节花器官、生殖器官以及其他性状发育模式的关键转录因子(<strong>Figure 1A-1B</strong>)；例如：floral organ identity大部分由MADS-box基因控制。MADS-box基因的共线性数据一杯用来推断B-和C-功能以及A-和E-功能基因的祖先遗传组成。但是这些研究只分析了不到十个物种, 结果以以parallel coordinate plots来展示(<strong>Figure 1C</strong>)。目前还没有关于很多物种MADS-box基因家族的进化分析研究，这是因为基因家族经历了大量的duplications，产生了复杂的直系同源、旁系同源以及功能同源。因此对所有可能的共线性关系的系统挖掘是很有挑战的，而且随着分析的物种数目的增加，组织以及展示这样的共线性关系也会变的更加的困难；<br><img src="/images/20190329-Fig1.jpg" alt="Figure 1"><br><img src="/images/20190329-Fig2.png" alt="Figure 1-legend"></li>
<li>因此，作者在这里展示了共线性网络cluster的新方法然后用于gene ancestry的分析。与<strong>Figure 1C</strong>不同的是，作者将基因共线性block（从种内和种间比较得到）抽象为顶点（节点或点）和边（点之间的线）。靶基因的<strong>Syntelogs (syntenic homologous genes)</strong>或者感兴趣的基因家族可以在一个图中highlight（<strong>Figure 1D</strong>），而不用展示flanking genes。例如：<strong>Figure 1C</strong>中的<strong>Gene 2</strong>可以展示成一个网络（<strong>Figure 1D cluster 1</strong>），边代表他们共线性的关系；如果一个基因经历了额外的duplication事件，例如串联重复或者多倍化事件，这些duplicated syntelogs被当作一个nodes而不是添加额外的线（<strong>Figure 1D cluster 2</strong>）；</li>
<li>通过共线性网络分析，潜在的老的串联重复也可以被展示出来。例如：Cluster 3展示了一个<strong>Gene 4</strong>和<strong>Gene 5</strong>都在一个cluster的例子（<strong>Figure 1D，cluster 3</strong>），当<strong>Gene 4</strong>和<strong>Gene 5</strong>属于同一个基因家族而且包含同一个蛋白结构域的时候会出现这种情况；与<strong>Cluster 2</strong>不同的是，这种情况可能来源于老的串联重复，因此在基因序列水平进化出了一定程度上的差异；举个例子，<strong>Gene A4</strong>例与<strong>Gene B5</strong>同源，因为在物种B中最好的选择<strong>基因B4</strong>”可能已丢失，因此我们可以获得更多的物种间的links而不是物种内的links(<strong>Figure 1D。cluster 3</strong>）。</li>
<li>值得提出的是，有时候一个特定的node也可以连接到其他多个没有连接的节点例如<strong>Gene E7</strong> radiates to seven other nodes of syntelogs of “<strong>Gene 8</strong>“ (<strong>Figure 1D cluster 4</strong>)，这是因为<strong>Gene 7</strong>包含<strong>Gene 7</strong>和<strong>Gene 8</strong>的结构域，或者是由于注释错误导致的；</li>
<li>作者接下来向我们介绍了最近发布的构建共线性网络的通用方法，主要分为以下三步：<ul>
<li><strong>Step 1:</strong> pairwise whole-genome comparisons</li>
<li><strong>Step 2:</strong> detection of syntenic blocks and data fusion</li>
<li><strong>Step 3:</strong> network clustering<br>前两步提供了旁系同源基因的共线性关系，MCScan进一步进行共线性检测，最后一步可以充分利用聚类算法，即使共线性网络分析的核心。最终得到的cluster将随使用的方法不同而不同；在这里，作者使用<strong>CFinder</strong>聚类pairwise synteny data，这个方法使用<strong>k-clique percolation</strong>可以检测网络中具有overlap的群体。K-clique对应着由k个节点组成的子网络（例如：k=3就是一个三角形）；两个k-cliques被认为是临近的如果他们共享k-1个节点；</li>
</ul>
</li>
<li>为了进一步说明这个方法，作者分析了MADS-box基因家族，这个家族主要的clades之间的关系已经被很多研究证明过了（<strong>Figure 1B</strong>）。但是这些研究还不能完全解决MADS-box树上最深的那些点。拟南芥的MADS-box家族由107个基因，来源于多个gene duplication事件。MADS基因可以被分为两个clades，叫做Type I和Type II。Type II进一步分为MIKC<sup>C</sup>和MIKC<sup>*</sup>。MADS-box基因已经被研究很多了尤其是MIKC<sup>C</sup>。为了方便起见，在此之后将SQUA，FLC和TM8样基因的假设共同祖先基因称为SFT基因；</li>
<li>接下来，作者就要讨论在51个植物基因组中检测到的所有的MADS-box基因的共线性网络，这个网络包括来自这51种物种中共有但也独立的多倍体事件的种内和种间同线性块。与系统发生数组合分析可以发现一些之前没有检测到的基因转座、duplication，shared deep ancestry。</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-共线性网络流程-Overview-of-the-synteny-network-pipeline"><a href="#3-1-共线性网络流程-Overview-of-the-synteny-network-pipeline" class="headerlink" title="3.1 共线性网络流程 (Overview of the synteny network pipeline)"></a>3.1 共线性网络流程 (Overview of the synteny network pipeline)</h4><ul>
<li><strong>Figure 2A:</strong> 作者主要分析了分析了51个植物基因组，包括绿藻(green algae)，苔藓(mosses)，裸子植物(gymnosperms)和被子植物(angiosperms),使用蛋白序列做全基因组的intra- and interspecies comparisions。然后通过MCScanX得到syntenic genomic blocks进而产生共线性gene pairs中所有的link，这个数据库总过包含921074 nodes and 8045487 edges；然后使用HMMER扫描51个基因组的蛋白序列确定MADS-box的candidate genes，这个基因list用于后续的提取subnetwork；提取的subnetwork包含3458 nodes，25500个边；<br><img src="/images/20190329-Fig3.jpg" alt="Figure 2"><br><img src="/images/20190329-Fig4.png" alt="Figure 2-legend"></li>
<li><strong>Figure 2B:</strong> 用Gephi进化subnetwork的可视化，使用k-clique percolation进行聚类，从这个图中可很直观的看出，MADS-box基因在所有被子植物谱系中关系，<strong>但是没有跟非被子植物的关系</strong>，点的大小代表连接数目的多少；</li>
<li><strong>Figure 2C:</strong> <strong>为了进一步解释distant gene clades</strong>之间的关系，就用所有基因家族构建的基因树去展示，作者发现，<strong>genes from distal gene clades (shown in Figure 1B) that are syntenically connected, such as SEP1-like (floral E genes) with SQUA-like (floral A genes) genes, AGL6-like with TM3 (SOC1-like) genes, and StMADS11 (SVP-like) with AGL17-like genes</strong>；<br><img src="/images/20190329-Fig5.jpg" alt="Figure 3"><br><img src="/images/20190329-Fig6.png" alt="Figure 3-legend"></li>
<li><strong>Figure 3A:</strong> 作者在Phylogenetic profiling选取了26 example clusters 从95 clusters（k=3），红色的格子代表在不同物种中至少有一个syntelog；在小麦和大麦中却没有发现任何与其他植物基因组具有共线性的区域，这可能是当时这两个基因组注释的质量不太好；</li>
<li><strong>Figure 3B:</strong> 通过使用Cytoscape，作者又向我们展示了一个undirected and unweighted network with related gene clade names；</li>
<li>接下来，作者主要从三个方面论述从MADS-box基因家族的共线性网络分析中的三个新发现。</li>
</ul>
<h4 id="3-2-谱系特异性的共线性关系-Lineage-specific-synteny-relationships"><a href="#3-2-谱系特异性的共线性关系-Lineage-specific-synteny-relationships" class="headerlink" title="3.2 谱系特异性的共线性关系 (Lineage-specific synteny relationships)"></a>3.2 谱系特异性的共线性关系 (Lineage-specific synteny relationships)</h4><ul>
<li>重要的被子植物家族（如禾本科，菊科，豆科，十字花科和茄科）很容易通过独特的性状和花卉特征进行鉴定。这些主要植物家族的特征还在于其起源具有<strong>独立的老的多倍体事件</strong>。因此，形态学变化可能与这些古老的多倍体事件或特定的基因转座事件有关，这些事件将关键的调节因子置于新的基因组背景中。我们的同线网络方法可以通过聚类和系统发育分析来识别基因的这种谱系特异性转座事件。</li>
</ul>
<h5 id="3-2-1-B-Function-AP3-and-PI-Genes-in-the-Brassicaceae-十字花科-and-Cleomaceae-醉蝶花科-Families"><a href="#3-2-1-B-Function-AP3-and-PI-Genes-in-the-Brassicaceae-十字花科-and-Cleomaceae-醉蝶花科-Families" class="headerlink" title="3.2.1 B-Function (AP3 and PI) Genes in the Brassicaceae (十字花科) and Cleomaceae (醉蝶花科) Families"></a>3.2.1 B-Function (AP3 and PI) Genes in the Brassicaceae (十字花科) and Cleomaceae (醉蝶花科) Families</h5><ul>
<li>AP3和PI基因对花瓣(petal)和雄蕊(stamen)的规格很重要, 在本文中，作者发现AP3基因落在一个单独的cluster (<strong>Figure 3B. cluster 9</strong>)，这个cluster由单子叶植物和双子叶植物的一些旁系同源组成。但是这个cluster缺乏十字花科（Brassicaceae）家族的AP3旁系同源；相反，来自十字花科的AP3基因形成了一个独立的cluster；</li>
<li>与此同时，PI基因也出现了非常相似的情况，来自于六个十字花科物种的PI旁系同源聚到了一起（<strong>Figure 3 cluster 24</strong>）</li>
</ul>
<h5 id="3-2-2-FLC-Like-genes-cluster-in-Brassicaceae"><a href="#3-2-2-FLC-Like-genes-cluster-in-Brassicaceae" class="headerlink" title="3.2.2 FLC-Like genes cluster in Brassicaceae"></a>3.2.2 FLC-Like genes cluster in Brassicaceae</h5><ul>
<li>在拟南芥中，FLC基因及其密切相关的MAF基因是花抑制因子和开花时间的主要调节因子；在本文中，作者发现了一个由FLC和MAF基因的21个syntelogs组成的cluster（<strong>Figure 3, cluster 23</strong>）;</li>
</ul>
<h5 id="3-2-3-AGL17-Like-genes-cluster-in-monocots"><a href="#3-2-3-AGL17-Like-genes-cluster-in-monocots" class="headerlink" title="3.2.3 AGL17-Like genes cluster in monocots"></a>3.2.3 AGL17-Like genes cluster in monocots</h5><ul>
<li>AGL17-like genes也形成了一个明显的共线性cluster（<strong>Figure 3 cluster 14</strong>）；</li>
</ul>
<h4 id="3-3-老的串联基因排列的推断"><a href="#3-3-老的串联基因排列的推断" class="headerlink" title="3.3 老的串联基因排列的推断"></a>3.3 老的串联基因排列的推断</h4><ul>
<li>除了上面描述的独特的lineage-specific cluster之外，当使用适当的聚类方法时（例如k-clique percolation），也可以获得包含互连子聚类（具有力导向或有机布局）的更大聚类。 ）。如cluster 3（<strong>Figure 1D</strong>）所示，这个cluster表明所涉及基因的长保守的紧密基因组接近（代表各自的亚群），因此有助于建立基因进化的轨迹。</li>
</ul>
<h5 id="3-3-1-Angiosperm-wide-conserved-SEP1-SQUA-and-SEP3-SFT-tandems"><a href="#3-3-1-Angiosperm-wide-conserved-SEP1-SQUA-and-SEP3-SFT-tandems" class="headerlink" title="3.3.1 Angiosperm-wide conserved SEP1-SQUA and SEP3-SFT tandems"></a>3.3.1 Angiosperm-wide conserved SEP1-SQUA and SEP3-SFT tandems</h5><ul>
<li>作者确定的最大一个cluster包含475个节点，由AGL2-like和SFT-like基因组成（<strong>Figure 3B cluster 1 and 4A</strong>）；这个cluster又可以分成两个subgroups，左边的是SEP1- and SQUA-like genes，右边的是SEP3-，FLC- and TM8-like genes（<strong>Figure 4A</strong>）；<br><img src="/images/20190329-Fig7.jpg" alt="Figure 4"><br><img src="/images/20190329-Fig8.png" alt="Figure 4-legend"></li>
</ul>
<h5 id="3-3-2-Angiosperm-wide-conserved-AGL6-TM3-tandem"><a href="#3-3-2-Angiosperm-wide-conserved-AGL6-TM3-tandem" class="headerlink" title="3.3.2 Angiosperm-wide conserved AGL6-TM3 tandem"></a>3.3.2 Angiosperm-wide conserved AGL6-TM3 tandem</h5><ul>
<li>作者确定的第二个最大的cluster包含305个节点，包含AGL6- and TM3-like genes (<strong>Figure 3B, cluster 2 and 4B</strong>), 作者发现AGL6-TM3 串联基因重排在被子植物中非常广泛（<strong>Figures 4B and 4C</strong>）；</li>
</ul>
<h4 id="3-4-Synteny-relations-across-angiosperms-for-overlooked-MADS-Box-gene-clades"><a href="#3-4-Synteny-relations-across-angiosperms-for-overlooked-MADS-Box-gene-clades" class="headerlink" title="3.4 Synteny relations across angiosperms for overlooked MADS-Box gene clades"></a>3.4 Synteny relations across angiosperms for overlooked MADS-Box gene clades</h4><ul>
<li>除了许多功能上被研究很好的MADS基因外，大部分基因家族成员的功能都没有被研究，例如MADS-box基因家族的MIKC<sup>*</sup>型和I型基因。但是，共线性网络可以为这些基因随着进化时间的共线性保守性提供证据，从而推断保守基因的功能；</li>
</ul>
<h5 id="3-4-1-MIKE-type-genes"><a href="#3-4-1-MIKE-type-genes" class="headerlink" title="3.4.1 MIKE*-type genes"></a>3.4.1 MIKE<sup>*</sup>-type genes</h5><ul>
<li>使用共线性网络分析，作者发现两个高度连接的网络，包含：<br>i: the angiosperm AGL30-, AGL65, and AGL94-like genes (MIKEC<sup>*</sup>-P clade) (<strong>Figure 3, cluster 5</strong>);<br>i:: the AGL66-, AGL67-, and AGL104-like genes (MIKC<sup>*</sup>-S clade) (<strong>Figure 3, cluster 10</strong>)</li>
<li>这两个cluster都包含双子叶植物和单子叶植物，但是MIKC<sup>*</sup>-S custer看起来在单子叶植物植物中由扩张，然后N. nucifera在这个cluster中缺乏，这意味着这两个clades在被子植物中很保守；</li>
</ul>
<h5 id="3-4-2-Type-I-MADS-Box-genes"><a href="#3-4-2-Type-I-MADS-Box-genes" class="headerlink" title="3.4.2 Type I MADS-Box genes"></a>3.4.2 Type I MADS-Box genes</h5><ul>
<li>XXX<h5 id="StMADS11-SVP-Like-Genes"><a href="#StMADS11-SVP-Like-Genes" class="headerlink" title="StMADS11 (SVP-Like) Genes"></a>StMADS11 (SVP-Like) Genes</h5></li>
<li>XXX</li>
</ul>
<h3 id="4-讨论"><a href="#4-讨论" class="headerlink" title="4. 讨论"></a>4. 讨论</h3><ul>
<li>XXX</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/">https://jingjingzhai.github.io/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/进化/">进化</a><a href="/tags/Synteny-Network/">Synteny Network</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/03/31/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/" class="pre">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</a><a href="/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/" class="next">Deep-learning augmented RNA-seq analysis of transcript splicing</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-共线性网络流程-Overview-of-the-synteny-network-pipeline"><span class="toc-text">3.1 共线性网络流程 (Overview of the synteny network pipeline)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-谱系特异性的共线性关系-Lineage-specific-synteny-relationships"><span class="toc-text">3.2 谱系特异性的共线性关系 (Lineage-specific synteny relationships)</span></a><ol class="toc-child"><li class="toc-item toc-level-5"><a class="toc-link" href="#3-2-1-B-Function-AP3-and-PI-Genes-in-the-Brassicaceae-十字花科-and-Cleomaceae-醉蝶花科-Families"><span class="toc-text">3.2.1 B-Function (AP3 and PI) Genes in the Brassicaceae (十字花科) and Cleomaceae (醉蝶花科) Families</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-2-2-FLC-Like-genes-cluster-in-Brassicaceae"><span class="toc-text">3.2.2 FLC-Like genes cluster in Brassicaceae</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-2-3-AGL17-Like-genes-cluster-in-monocots"><span class="toc-text">3.2.3 AGL17-Like genes cluster in monocots</span></a></li></ol></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-老的串联基因排列的推断"><span class="toc-text">3.3 老的串联基因排列的推断</span></a><ol class="toc-child"><li class="toc-item toc-level-5"><a class="toc-link" href="#3-3-1-Angiosperm-wide-conserved-SEP1-SQUA-and-SEP3-SFT-tandems"><span class="toc-text">3.3.1 Angiosperm-wide conserved SEP1-SQUA and SEP3-SFT tandems</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-3-2-Angiosperm-wide-conserved-AGL6-TM3-tandem"><span class="toc-text">3.3.2 Angiosperm-wide conserved AGL6-TM3 tandem</span></a></li></ol></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-Synteny-relations-across-angiosperms-for-overlooked-MADS-Box-gene-clades"><span class="toc-text">3.4 Synteny relations across angiosperms for overlooked MADS-Box gene clades</span></a><ol class="toc-child"><li class="toc-item toc-level-5"><a class="toc-link" href="#3-4-1-MIKE-type-genes"><span class="toc-text">3.4.1 MIKE*-type genes</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-4-2-Type-I-MADS-Box-genes"><span class="toc-text">3.4.2 Type I MADS-Box genes</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#StMADS11-SVP-Like-Genes"><span class="toc-text">StMADS11 (SVP-Like) Genes</span></a></li></ol></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#4-讨论"><span class="toc-text">4. 讨论</span></a></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/04/03/Dysregulation-of-expression-correlates-with-rare-allele-burden-and-fitness-loss-in-maize/">Dysregulation of expression correlates with rare-allele burden and fitness  loss in maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/03/DeepCAPE-a-deep-convolutional-neural-network-for-the-accurate-prediction-of-enhancers/">DeepCAPE: a deep convolutional neural network for the accurate prediction of enhancers</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/">Comprehensive functional genomic resource and integrative model for the human brain</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/01/A-universal-sNP-and-small-indel-variant-caller-using-deep-neural-networks/">A universal SNP and small-indel variant caller using deep neural networks</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/31/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/">Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/">Deep-learning augmented RNA-seq analysis of transcript splicing</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/">Predicting splicing from primary sequence with deep learning</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/">DeepTACT: Predicting 3D chromatin contacts via bootstrapping deep learning</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/13/Network-propagation-a-universal-amplifier-of-genetic-associations/">Network propagation: a universal amplifier of genetic associations</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">1</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>