<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Common SNPs explain a large proportion of the heritability for human height | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Common SNPs explain a large proportion of the heritability for human height</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Common SNPs explain a large proportion of the heritability for human height</h1><div class="post-meta"><a href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/#comments" class="comment-count"></a><p><span class="date">Apr 26, 2019</span><span><a href="/categories/Nature-Genetics/" class="category">Nature Genetics</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p><a href="https://www.nature.com/articles/ng.608" target="_blank" rel="noopener">文章链接</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>由GWAS找到的SNP只能解释人类群体中复杂性状的一小部分遗传变异，那么剩余的遗传力在哪里呢？作者在这里使用线性模型分析了3925个不相关的个体中由294831个SNP解释的人类身高的方差的比例，并使用模拟方法验证了这个估计方法；</li>
<li>当同时考虑所有的SNP时，45%的方差可以被解释，因此大部分的遗传力并不是丢失了，而是没有被检测出来，可能是因为效应太小了没有办法通过严格的显著性检验；</li>
<li>作者说明剩余的遗传力是由于causal variants与所有genotyped SNPs之间的不完全的连锁不平衡，而且causal variants往往具有更低的等位基因频率，这也就加剧了剩余遗传力。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>人类群体中GWAS已经鉴定出了成百上千个与复杂性状显著相关的SNPs，然而对于任何性状来说，他们只占据遗传变异的一小部分，那么剩余的遗传力在哪里呢？也就是基因组中所谓的<strong>dark matter</strong>。可能的解释包括：基因与基因、基因与环境之间的相互作用或者是可遗传的表观因子导致了亲缘之间的相似。</li>
<li>然而，以验证的SNP所解释的遗传方差比狭义遗传力 (narrow-sense heritability; the proportion of phenotypic variance due to additive heritability) 小得多。对于验证的SNP不能解释估计的遗传力，有两种解释：<ul>
<li>causal variants的效应小，没有达到严格的显著性阈值；</li>
<li>causal variants与genotyped SNPs之间不完全的连锁不平衡，causal variants比genotyped SNPs相比MAF更低</li>
</ul>
</li>
<li>接下来作者将验证了上述两种假设，使用人类身高作为理想的复杂性状估计每一种假设对身高遗传力的贡献。</li>
<li>身高是一个非常经典的数量性状，很容易测量，因此多年来一直被当做理想复杂性状。身高的遗传力被估计为0.8。罕见变异可以导致极高或者极低的身高，但是这些不能解释在一般群体中的变异，最近的GWAS检测出了大约50个变异与群体中身高相关，但是这些只能解释总表型方差的5%。</li>
<li>由GWAS得到的统计关联分析只能检测单个SNP与复杂性状的关联，考虑到多重假设检验，就需要设定严格的阈值，这在一定程度上降低了假阳性，但同时也丢掉了很多真阳性，尤其是当SNP对表型的贡献很小的时候。关联分析的时候，SNP的效应被认为是随机的，然后估计由所有SNP解释的方差。作者在这里使用的这种方法，不尝试检验单个SNP的显著性而是使用所有的SNP估计无偏的方差。</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-估计由全基因组范围内SNP解释的遗传方差-Estimating-genetic-variance-explained-by-genome-wide-SNPs"><a href="#3-1-估计由全基因组范围内SNP解释的遗传方差-Estimating-genetic-variance-explained-by-genome-wide-SNPs" class="headerlink" title="3.1 估计由全基因组范围内SNP解释的遗传方差 (Estimating genetic variance explained by genome-wide SNPs)"></a>3.1 估计由全基因组范围内SNP解释的遗传方差 (Estimating genetic variance explained by genome-wide SNPs)</h4><ul>
<li>作者使用4259个个体的群体数据，然后对每个个体做pairwise relationship比较去除大于0.025的个体，最终保留了3925个个体。作者使用REML (Restricted maximum likelihood)估计由SNP解释的遗传方差。<br><img src="/images/20190426-Fig1.png" alt="table 1"></li>
<li>如<strong>表1</strong>所示：由这些SNP可以解释表型方差的比例是0.45。</li>
</ul>
<h4 id="3-2-矫正因果变异与SNPs之间不完全的LD-Correcting-for-incomplete-LD-between-SNPs-and-causal-variants"><a href="#3-2-矫正因果变异与SNPs之间不完全的LD-Correcting-for-incomplete-LD-between-SNPs-and-causal-variants" class="headerlink" title="3.2 矫正因果变异与SNPs之间不完全的LD (Correcting for incomplete LD between SNPs and causal variants)"></a>3.2 矫正因果变异与SNPs之间不完全的LD (Correcting for incomplete LD between SNPs and causal variants)</h4><ul>
<li>45%的遗传变异仍然与表型方差的80%(加性遗传方差，也就是估算的遗传力)相比仍然有很大出入。<strong>那么为什么这些SNPs不能解释所有估算的遗传力呢？</strong>一个原因是：所有的SNPs与causal variants并不是完全连锁的。SNP解释表型方差的能力依赖于causal variants与所有SNP之间的LD信息。</li>
<li>但是真是的causal variants是未知的，我们不能直接估计他们与观测到的SNPs之间的LD信息，但是我们可以通过考虑所有的SNPs之间的LD来模拟它。主要是因为causal vatiants与其他SNPs有不同的属性，因此SNPs之间的LD可以作为causal variants与SNPs之间的一个导向。</li>
<li>为了挖掘\(G_{jk}\) (由causal variants计算的基因组relationship)与\(A_{jk}\) (由所有SNPs计算的relationship)之间的差异如何，作者随机从成年人群体中选取了五组SNPs (50K, 100K, …, 250K, K = 1000), 从青春期少年的群体中选取10组(50K, 100K, …, 500K). 对于每一组SNP，将SNPs随机分成两组，第一组代表SNPs，第二组代表causal variants，然后使用第一个组中SNPs的遗传关系 (\(A_{jk}\)), 使用第二组中的SNPs (\(MAF &lt; \theta\); where \(\theta = 0.1, 0.2, 0.3, 0.4, 0.5\))计算\(G_{jk}\)。<br><img src="/images/20190426-Fig2.png" alt="Figure 1"></li>
<li>从图2可以看出，校正后的遗传关系不随着SNP数目的增多而增加。<br><img src="/images/20190426-Fig3.png" alt="Figure 2"></li>
</ul>
<h4 id="3-3-模拟研究-Simulation-studies"><a href="#3-3-模拟研究-Simulation-studies" class="headerlink" title="3.3 模拟研究 (Simulation studies)"></a>3.3 模拟研究 (Simulation studies)</h4><ul>
<li>模拟研究验证这个方法，作者在这里基于3925个个体的294831个SNP用两种方式模拟了数量性状：<ul>
<li>从所有SNP中随机选取causal variants；</li>
<li>从MAF小于0.1的SNP中随机选取causal variants；</li>
</ul>
</li>
<li>从表2可以看出情景1是估计的遗传力偏差更小；<br><img src="/images/20190426-Fig4.png" alt="Table 2"></li>
</ul>
<h3 id="4-讨论"><a href="#4-讨论" class="headerlink" title="4. 讨论"></a>4. 讨论</h3><ul>
<li>GWAS只能解释5%的表型方差，而本文中发现所有的SNPs可以解释40%的表型方差，因此88%（40/45）由SNP导致的方差在GWAS中都没有被检测出来（由于没有达到统计显著性）；本文的研究表明：45%与总共80%的遗传力之间的矛盾来源于causal variants与SNPs之间不安全的LD信息，可能是因为causal variants通常具有更低的MAF。</li>
</ul>
<h3 id="5-方法"><a href="#5-方法" class="headerlink" title="5. 方法"></a>5. 方法</h3><h4 id="5-1-统计框架"><a href="#5-1-统计框架" class="headerlink" title="5.1 统计框架"></a>5.1 统计框架</h4><ul>
<li>对于一个数量性状的GWAS来说，单个SNP与性状的关联可以用以下回归模型：<script type="math/tex; mode=display">y_j=\mu+x_{ij}a_i+e_j</script></li>
<li>其中：<ul>
<li>\(y_j\): 第j个个体的表型值；</li>
<li>\(\mu\): 均值项；</li>
<li>\(a_i\): SNP i的等位基因替换效应；</li>
<li>\(x_{ij}\): SNP i的基因型(0, 1, 2);</li>
<li>\(e_j\): 残差效应，服从均值为0，方差为\(\sigma_e^2\)的正态分布。</li>
</ul>
</li>
<li>假定我们可以对所有的causal variants进行genotype，我们可以把所有的causal variants包含进模型中：<script type="math/tex; mode=display">y_j=\mu+g_{j}+e_j</script><script type="math/tex; mode=display">g_j=\sum_{i=1}^mz_{ij}u_i</script></li>
<li>其中：<ul>
<li>\(g_i\)是个体j总的遗传效应；</li>
<li>\(m\): causal loci的数目；</li>
<li>\(u_i\): 第i个causal variant的加性效应；</li>
<li>\(z_{ij}\)可以取\(\frac{-2f_i}{\sqrt{2f_i(1-f_i)}}\), \(\frac{1-2f_i}{\sqrt{2f_i(1-f_i)}}\), \(\frac{2(1-f_i)}{\sqrt{2f_i(1-f_i)}}\)中的一个，也就是说第j个个体在位点i的基因型是0，1，2的时候，分别取上面三个值。\(f_i\)是等位基因的频率。可以知道：\(E(z_{ij}=0)\); \(var(z_{ij}=1)\)。</li>
</ul>
</li>
<li>写成矩阵的形式：<script type="math/tex; mode=display">\textbf{y}=\mu\textbf{1}+\textbf{g}+\textbf{e}</script><script type="math/tex; mode=display">\textbf{g}=\textbf{Z}\textbf{u}</script></li>
<li><strong>注意：上式中\(\mu\)是常数，所以要乘以identity matrix（一列）才是向量</strong></li>
<li>将\(u\)看作是随机效应，假定\(u \sim N(0,I\sigma_u^2)\), 这里\(\sigma_u^2\)是causal effects的方差，因此y的方差-协方差矩阵可以表示为：</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">https://jingjingzhai.github.io/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/数量遗传/">数量遗传</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/" class="pre">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a><a href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/" class="next">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-估计由全基因组范围内SNP解释的遗传方差-Estimating-genetic-variance-explained-by-genome-wide-SNPs"><span class="toc-text">3.1 估计由全基因组范围内SNP解释的遗传方差 (Estimating genetic variance explained by genome-wide SNPs)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-矫正因果变异与SNPs之间不完全的LD-Correcting-for-incomplete-LD-between-SNPs-and-causal-variants"><span class="toc-text">3.2 矫正因果变异与SNPs之间不完全的LD (Correcting for incomplete LD between SNPs and causal variants)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-模拟研究-Simulation-studies"><span class="toc-text">3.3 模拟研究 (Simulation studies)</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#4-讨论"><span class="toc-text">4. 讨论</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#5-方法"><span class="toc-text">5. 方法</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#5-1-统计框架"><span class="toc-text">5.1 统计框架</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/07/17/A-Deep-Neural-Network-for-Predictingand-Engineering-Alternative-Polyadenylation/">A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">Single-cell RNA-seq denoising using a deep count autoencoder</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/APA预测/" style="font-size: 15px;">APA预测</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/08/">八月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>