<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</h1><div class="post-meta"><a href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/#comments" class="comment-count"></a><p><span class="date">Apr 17, 2019</span><span><a href="/categories/The-Plant-Cell/" class="category">The Plant Cell</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2019年3月发表于The Plant Cell期刊上关于研究假基因的进化起源以及他们与调控序列关系的文章。通讯作者是来自于北京林业大学的<a href="http://biology.bjfu.edu.cn/szdw/bqwrcgcgjjrx/269650.html" target="_blank" rel="noopener">张德强</a>。</p>
<p><a href="http://www.plantcell.org/content/31/3/563" target="_blank" rel="noopener">文章简介</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>假基因，功能基因的非功能亲属，由重复或者反转录转座以及基因功能确实形成，进化分析为假基因的起源以及对基因表达的影响提供了线索；</li>
<li>然而，关于植物假基因的系统研究却很少实施。作者在这里研究了7个被子植物中的假基因的起源、进化以及表达模式还有他们与非编码序列的关系；</li>
<li>作者确定了大约250000个假基因，与蛋白编码基因相比，大多数是lineage-specific。假基因在染色体上的分布表明基因组重组对于假基因的消除有贡献，大多数的假基因从序列以及表达水平来看进化很快，表现出了组织或者发育阶段特异性的表达模式；</li>
<li>作者发现相当大一部分的非转座子非编码调控RNAs(microRNAs和long noncoding RNAs)来源于假基因上游的邻近区域，表明假基因通过提供TFBS调控基因组的进化；</li>
<li>因此作者提出，假的转录调控区域的快速重写是主要新的调控模块起源的主要机制。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>假基因是蛋白编码基因的disabled copies，通常被当作genomic fossils。如果蛋白编码的基因发生了例如移码、in-frame stop codons and truncations of full-length genes等特征时就可能会变成假基因。根据重复事件的机制，假基因可以被分成两类：和processed；<ul>
<li>nonprocessed: 来源于基因组DNA重复或者不平等交叉互换；</li>
<li>processed: 反转录和整合事件；</li>
</ul>
</li>
<li>假基因被认为是没有功能的序列因此在进化上倾向于中性，与此一致的是，在人类、线虫、果蝇基因组中，大部分的假基因都是中性进化；</li>
<li>虽然假基因是蛋白编码基因的disabled copies，但是有一小部分的假基因表现出了各种各样的调控功能，影响调控RNA的产生。例如：目前已有研究表明假基因可以作为endogenous small interfering RNAs的source；也有研究表明假基因调控基因表达通过隐蔽microRNAs。这些研究表明假基因在基因表达中伴着调控角色因此促进科学家去挖掘假基因在不同物种中的功能；</li>
<li>假基因的进化分析已经为他们亲缘特异性的基因组进化历史以及遗传机制提供了重要的线索，但目前这样的线索还比较少；</li>
<li>进化导致假基因在染色体分布的机制目前尚不清楚。基因组重复(paleopolyploidy)事件在开花植物中很普遍。paleopolyploidy的长期进化总是设计大量的基因组反转录转座以及重复基因的大量消除。这可能会在植物基因组中产生成千上万个假基因。重组被认为是影响基因组特征的一个关键因素。人类和过硬的基因组中重组率很低但是在线虫中却表现出相反的趋势，这些观测在特定的基因组中重组事件的差异；</li>
<li>全基因范围内从进化速率分析假基因的研究目前还没少，因此在paleopoly物种中进化力量如何改变假基因的模式还不清楚。目前在植物中唯一的跨物种假基因的比较是拟南芥和水稻，因此植物假基因的研究还需进一步深入。</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-七个被子植物物种中假基因的确定-Identification-of-psi-in-seven-angiosperm-species"><a href="#3-1-七个被子植物物种中假基因的确定-Identification-of-psi-in-seven-angiosperm-species" class="headerlink" title="3.1 七个被子植物物种中假基因的确定 (Identification of \(\psi\ in seven angiosperm species))"></a>3.1 七个被子植物物种中假基因的确定 (Identification of \(\psi\ in seven angiosperm species))</h4><ul>
<li>为了系统确定七个物种(拟南芥、二穗短柄草、大豆、蒺藜状苜蓿、水稻、毛果杨以及高粱)中的候选假基因区域，作者在这里使用旁系同源搜索组合以及严格的筛选条件来最小化噪音增加阳性信号(<strong>Figure 1</strong>)。<br><img src="/images/20190417-Fig1.png" alt="Figure 1"></li>
<li>首先使用RepeatMasker (RM)将基因组中基因间隔区的重复序列mask；</li>
<li>以identity大于0.2, match length &gt; 0.05为阈值确定了90,000到800,000基因间隔区域的旁系contigs，然后检测在cutoff(30-107氨基酸，match length coverage ratio 0.050-0.052)附近检测假基因然后发现了8个来源于WGD的假基因，三个是共线性的假基因；</li>
<li><strong>Figure 2A:</strong> 应用这个严格的筛选条件，每个物种获得了5128到73811个假基因，其中146-2524个假基因来源于WGD事件，同时还发现11.6%-25%的假基因由内含子；</li>
<li><strong>Figure 2B:</strong> 与预期一致，作者发现假基因与七个物种的蛋白编码基因的密度由很强的相关性；</li>
<li><strong>Figure S3, 2C:</strong> 大豆在最近六千万年里经历了两个WGD，一次比较近的是在1300万年前导致了高度重复的基因组，大约75%的基因都有多个拷贝，因此大豆中假基因的可能来源于recent WGD之后不久的基因的丢失，于此一致的是：作者发现包含假基因的基因对Ks大约为0.13，对应着大豆lineage-specific paleotetraploidization. 同时在毛果杨中发现假基因的与最近功能同源的比对coverage最高，通过比较Ka、Ks以及二者之间的比例作者衡量进化速率, 总体来看，大部分的假基因-FP(Functional paralog)之间Ka:Ks比例都要比FG-FG之间高一点。<br><img src="/images/20190417-Fig2.png" alt="Figure 2"><br><img src="/images/20190417-Fig3.png" alt="Figure 2-legend"><br><img src="/images/20190417-Fig4.png" alt="Figure S3"></li>
</ul>
<h4 id="3-2-老的全长假基因的不对称消除率-Asymmetric-elimination-rate-of-ancient-full-length-psi"><a href="#3-2-老的全长假基因的不对称消除率-Asymmetric-elimination-rate-of-ancient-full-length-psi" class="headerlink" title="3.2 老的全长假基因的不对称消除率 (Asymmetric elimination rate of ancient full-length \(\psi\))"></a>3.2 老的全长假基因的不对称消除率 (Asymmetric elimination rate of ancient full-length \(\psi\))</h4><ul>
<li>接下来，作者通过检测假基因序列与FP之间的序列相似性来推断假基因的年龄。</li>
<li><strong>Figure 2D:</strong> 作者观察到在相似时间点上假基因的数目一步一步增加然后再一步一步降低；</li>
<li>由于假基因aare expected中性进化，作者假下来检测了三种类型的disablements: 插入、缺失以及终止密码子；<br><img src="/images/20190417-Fig5.png" alt="Figure S5"><br>-通过上图作者观测到：在P. trichocarpa, M. truncatula中stop codons的密度更高；Arabidopsis, B. distachyon rice, and S. bicolor中插入的比例更高；</li>
<li>考虑到假基因的中性进化，在老得全长假基因的处所有位点的选择压力应该是一样的。但是当作者检测假基因片段与FPs的重叠的时候，作者观测到老得全长假基因的不对称的消除速率，两端peaks处有更高的保留(<strong>Figure S6</strong>)<br><img src="/images/20190417-Fig6.png" alt="Figure S6"></li>
</ul>
<h4 id="3-3-不同物种中假基因的动态指令-Dynammic-repertoire-of-psi-in-different-species"><a href="#3-3-不同物种中假基因的动态指令-Dynammic-repertoire-of-psi-in-different-species" class="headerlink" title="3.3 不同物种中假基因的动态指令 (Dynammic repertoire of \(\psi\) in different species)"></a>3.3 不同物种中假基因的动态指令 (Dynammic repertoire of \(\psi\) in different species)</h4><ul>
<li>作者在这里比较了假基因与他们的FPs之间的分布模式，发现只有23.3%的基因由假基因，导致了高度不均匀的假基因的分布；</li>
</ul>
<h4 id="3-4-重组对假基因的消除有贡献-Recombination-contributes-to-psi-elimination"><a href="#3-4-重组对假基因的消除有贡献-Recombination-contributes-to-psi-elimination" class="headerlink" title="3.4 重组对假基因的消除有贡献 (Recombination contributes to \(\psi\) elimination)"></a>3.4 重组对假基因的消除有贡献 (Recombination contributes to \(\psi\) elimination)</h4><ul>
<li>大多数假基因没有选择性约束，可以自由地积累非基因样特征，如移码和终止密码子，因此作者想知道重组率是否会影响假基因分布的模式；在<strong>抑制重组区域</strong>有害变异更容易积累，例如中心体周围，因此作者期望观测到低重组区域和中心体附近有假基因的富集。</li>
<li>与预期抑制，作者发现假基因相对富集在中心体区域，在大豆，水稻、拟南芥中心体区域往往都是低重组率的，大豆基因组中一个有力的特征是57%基因组序列都在中心体区域内。</li>
<li><strong>Figure 4A:</strong> 大豆中重组率与假基因的密度之间有明显的负相关；<br><img src="/images/20190417-Fig7.png" alt="Figure 4A"></li>
<li>之间已经有研究表明在高重组区域与低重组区域蛋白编码基因存在着不对称的进化，作者的研究将分析延伸到比较假基因与FP之间，作者开始将大豆中注释的假基因与他们的FPs利用经验密码子模型进行比较，去除低置信度的假基因-FP pairs。随后，计算每一对之间的Ka、Ks。</li>
<li><strong>Figure 4B:</strong> 由图可以看出来在染色体臂上的pair用有更高的Ka。<br><img src="/images/20190417-Fig8.png" alt="Figure 4B"></li>
</ul>
<h4 id="3-5-顺式调控元件富集在假基因的邻近上游区域-Cis-regulatory-elements-are-enriched-in-the-proximal-upstream-regions-of-psi"><a href="#3-5-顺式调控元件富集在假基因的邻近上游区域-Cis-regulatory-elements-are-enriched-in-the-proximal-upstream-regions-of-psi" class="headerlink" title="3.5 顺式调控元件富集在假基因的邻近上游区域 (Cis-regulatory elements are enriched in the proximal upstream regions of \(\psi\))"></a>3.5 顺式调控元件富集在假基因的邻近上游区域 (Cis-regulatory elements are enriched in the proximal upstream regions of \(\psi\))</h4><ul>
<li>虽然很多假基因看起来有很高的turnover rate而且不编码蛋白质，但是有一些仍然产生RNA。为了检测假基因的表达，作者重新分析了六个物种的RNA-seq数据获得了链特异性的RNA-seq for Poplus under four abiotic stress treatments。<br><img src="/images/20190417-Fig9.jpg" alt="Figure 5"><br><img src="/images/20190417-Fig10.png" alt="Figure 5-legend"></li>
<li><strong>Figure 5A,5C:</strong> 在七个物种中，假基因的表达的中值与编码蛋白的基因要低，而且一些假基因表现出了高度组织特异性；</li>
<li><strong>Figure 5B:</strong> 更详细的分析表明，有表达的假基因倾向于与自己的FPs有更高的序列相似性，这表明，有表达的假基因可能来源于相对比较近的重复事件。一个可能的解释是老的祖先顺式调控元件功能没有完全退化，在这种情况下，假基因的表达与FPs高度相关；</li>
<li><strong>Figure 5D:</strong> 在七个物种中，假基因-FP表达的平均相关系数是0.19，比随机选的要高比观测到的WGDs对要低；</li>
<li><strong>Figure 5E:</strong> 假基因的高度的组织特异性以及低的表达水平引来一个问题：<strong>顺式调控元件是不是富集在假基因上游区域？</strong>为了验证这个假设，作者分析了TFBS的频率，作者发现假基因临近上游区域与TFBS显著相关；</li>
<li><strong>Figure 5F:</strong> 作者接下来分析假基因表达模式的进化保守性。为了达到此目的，作者首先估计了物种之间的共享转录活动的出现。在水稻假基因组家族中只检测到了52.1%在B. distachyon中表达，52.4%在大豆中表达的在M. truncatula中表达。</li>
<li>Overall, these results indicate that rapid transcriptional evolution is a genuine feature of Cs。</li>
</ul>
<h4 id="3-6-非编码RNA基因与mRNA基因和假基因相关-Noncoding-RNA-genes-are-associated-with-mRNA-genes-and-psi"><a href="#3-6-非编码RNA基因与mRNA基因和假基因相关-Noncoding-RNA-genes-are-associated-with-mRNA-genes-and-psi" class="headerlink" title="3.6 非编码RNA基因与mRNA基因和假基因相关 (Noncoding RNA genes are associated with mRNA genes and \(\psi\))"></a>3.6 非编码RNA基因与mRNA基因和假基因相关 (Noncoding RNA genes are associated with mRNA genes and \(\psi\))</h4><ul>
<li>为了探索假基因对于非编码RNA的组成以及调节的贡献，作者构建了一个lncRNA的数据集来探究其与假基因的关系；</li>
<li><strong>Figure 6A:</strong> 对于非TE的lncRNAs大部分离基因更近一小部分离假基因更近。大多数非TE的lncRNAs与mRNA基因更近的被发现起源于TSS周围2-kb的区域；<br><img src="/images/20190417-Fig11.png" alt="Figure 6"><br><img src="/images/20190417-Fig12.png" alt="Figure 6"></li>
<li><strong>Figure 6B:</strong> 大多数lncRNA都是从假基因的临近上游区域转录而来的，举个例子：拟南芥中的lncRNA转录本来源于假基因位点。</li>
<li><strong>Figure 6C-6F</strong>: 作者随机选取了30个假基因的临近上游区域做了实验验证，21个检测到了GFP信号；</li>
</ul>
<h3 id="4-讨论"><a href="#4-讨论" class="headerlink" title="4. 讨论"></a>4. 讨论</h3><h4 id="4-1-假基因在植物中有很高的转换率-psi-have-a-high-turnover-rate-in-plants"><a href="#4-1-假基因在植物中有很高的转换率-psi-have-a-high-turnover-rate-in-plants" class="headerlink" title="4.1 假基因在植物中有很高的转换率 (\(\psi\) have a high turnover rate in plants)"></a>4.1 假基因在植物中有很高的转换率 (\(\psi\) have a high turnover rate in plants)</h4><ul>
<li>假基因是高度lineage specific而且有很高的turnover rate，表明植物中很大数目的假基因都在relaxed selective constraints进化，因此在假基因化的过程中快速消失；</li>
<li>在七个物种中，作者检测到了43个核心的假基因家族；</li>
<li>挖掘了假基因在三个植物物种中的分布，发现他们富集在中心体区域；</li>
<li>作者进一步分析了大豆中的基因组重组事件，结果表明假基因的分布与局部重组率显著负相关；</li>
</ul>
<h4 id="4-2-基因间隔区域的非编码RNAs来源于分化的假基因-Intergenic-noncoding-RNAs-are-derived-from-divergent-psi"><a href="#4-2-基因间隔区域的非编码RNAs来源于分化的假基因-Intergenic-noncoding-RNAs-are-derived-from-divergent-psi" class="headerlink" title="4.2 基因间隔区域的非编码RNAs来源于分化的假基因 (Intergenic noncoding RNAs are derived from divergent \(\psi\))"></a>4.2 基因间隔区域的非编码RNAs来源于分化的假基因 (Intergenic noncoding RNAs are derived from divergent \(\psi\))</h4><ul>
<li>有表达的假基因不可能仅仅代表转录组的噪音，因为很多假基因展现出了特定的表达模式而且与非生物斜坡相关；</li>
<li>在本研究中，一部分假基因是转录激活，这些假基因的上游邻近区域富集TFBSs；</li>
</ul>
<h4 id="4-3-新的调控序列来源于从头调控序列还是已经存在的调控序列-Do-novel-regulatory-sequences-originate-de-novo-or-from-preexisting-regulatory-sequences"><a href="#4-3-新的调控序列来源于从头调控序列还是已经存在的调控序列-Do-novel-regulatory-sequences-originate-de-novo-or-from-preexisting-regulatory-sequences" class="headerlink" title="4.3 新的调控序列来源于从头调控序列还是已经存在的调控序列 (Do novel regulatory sequences originate de novo or from preexisting regulatory sequences)"></a>4.3 新的调控序列来源于从头调控序列还是已经存在的调控序列 (Do novel regulatory sequences originate de novo or from preexisting regulatory sequences)</h4><ul>
<li>基因表达新的模式可以通过两种机制形成：从头进化和从已经存在的调控信息上重写。第二种机制可以分成三类：<ul>
<li>转座；</li>
<li>启动子交换；</li>
<li>co-option</li>
</ul>
</li>
<li>基因的调节是通过转录因子在TFBS的协调完成的，在很多物种中，TFBSs倾向于以同型或者异型簇出现，拥有复杂的调控motifs；</li>
<li>这些基因间隔区域的伸张总是拥有各种不同的TFBSs，这些区域可以获得一系列随机的点突变、小的indels或者TEs。</li>
<li>作者认为：<strong>从已有的调控序列生成新的新的调控序列比从头生成更有可能</strong>。</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">https://jingjingzhai.github.io/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/进化/">进化</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/" class="next">Single-cell RNA-seq denoising using a deep count autoencoder</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-七个被子植物物种中假基因的确定-Identification-of-psi-in-seven-angiosperm-species"><span class="toc-text">3.1 七个被子植物物种中假基因的确定 (Identification of \(\psi\ in seven angiosperm species))</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-老的全长假基因的不对称消除率-Asymmetric-elimination-rate-of-ancient-full-length-psi"><span class="toc-text">3.2 老的全长假基因的不对称消除率 (Asymmetric elimination rate of ancient full-length \(\psi\))</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-不同物种中假基因的动态指令-Dynammic-repertoire-of-psi-in-different-species"><span class="toc-text">3.3 不同物种中假基因的动态指令 (Dynammic repertoire of \(\psi\) in different species)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-重组对假基因的消除有贡献-Recombination-contributes-to-psi-elimination"><span class="toc-text">3.4 重组对假基因的消除有贡献 (Recombination contributes to \(\psi\) elimination)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-5-顺式调控元件富集在假基因的邻近上游区域-Cis-regulatory-elements-are-enriched-in-the-proximal-upstream-regions-of-psi"><span class="toc-text">3.5 顺式调控元件富集在假基因的邻近上游区域 (Cis-regulatory elements are enriched in the proximal upstream regions of \(\psi\))</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-6-非编码RNA基因与mRNA基因和假基因相关-Noncoding-RNA-genes-are-associated-with-mRNA-genes-and-psi"><span class="toc-text">3.6 非编码RNA基因与mRNA基因和假基因相关 (Noncoding RNA genes are associated with mRNA genes and \(\psi\))</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#4-讨论"><span class="toc-text">4. 讨论</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#4-1-假基因在植物中有很高的转换率-psi-have-a-high-turnover-rate-in-plants"><span class="toc-text">4.1 假基因在植物中有很高的转换率 (\(\psi\) have a high turnover rate in plants)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-2-基因间隔区域的非编码RNAs来源于分化的假基因-Intergenic-noncoding-RNAs-are-derived-from-divergent-psi"><span class="toc-text">4.2 基因间隔区域的非编码RNAs来源于分化的假基因 (Intergenic noncoding RNAs are derived from divergent \(\psi\))</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-3-新的调控序列来源于从头调控序列还是已经存在的调控序列-Do-novel-regulatory-sequences-originate-de-novo-or-from-preexisting-regulatory-sequences"><span class="toc-text">4.3 新的调控序列来源于从头调控序列还是已经存在的调控序列 (Do novel regulatory sequences originate de novo or from preexisting regulatory sequences)</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">Single-cell RNA-seq denoising using a deep count autoencoder</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/12/Opportunities-and-challenges-for-transcriptome-wide-association-studies/">Opportunities and challenges for transcriptome- wide association studies</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/09/Virtual-ChIP-seq-predicting-transcription-factor-binding-by-learning-from-the-transcriptome/">Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/03/Dysregulation-of-expression-correlates-with-rare-allele-burden-and-fitness-loss-in-maize/">Dysregulation of expression correlates with rare-allele burden and fitness  loss in maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/03/DeepCAPE-a-deep-convolutional-neural-network-for-the-accurate-prediction-of-enhancers/">DeepCAPE: a deep convolutional neural network for the accurate prediction of enhancers</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/">Comprehensive functional genomic resource and integrative model for the human brain</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/01/A-universal-sNP-and-small-indel-variant-caller-using-deep-neural-networks/">A universal SNP and small-indel variant caller using deep neural networks</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/31/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/">Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>