<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Comprehensive functional genomic resource and integrative model for the human brain | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Comprehensive functional genomic resource and integrative model for the human brain</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Comprehensive functional genomic resource and integrative model for the human brain</h1><div class="post-meta"><a href="/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/#comments" class="comment-count"></a><p><span class="date">Apr 01, 2019</span><span><a href="/categories/Science/" class="category">Science</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p><a href="http://science.sciencemag.org/content/362/6420/eaat8464" target="_blank" rel="noopener">文章链接</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>尽管在确定精神疾病的遗传风险方面取得了进展，但其分子机制仍然难以捉摸；为了解决这个问题，PsychENCODE联盟生成了包含1866个体人脑的全面在线资源；</li>
<li>PsychENCODE包含大约79000个brain-active enhancers, sets of Hi-C linkages and TADs; 很多细胞类型的单细胞表达图谱；eQTLs，与染色质、剪切、细胞类型比例相关的QTLs；</li>
<li>整合分析表明，不同的细胞类型比例在很大程度上解释了表达中的跨种群变异（重建准确度&gt; 88％），然后构建了基因调控网络，可以将GWAS得到的变异与基因关联起来；</li>
<li>最后将这个调控网络嵌入到一个可解释的深度学习模型，该模型将疾病预测与多基因风险评分相比提高了约6倍，而且确定了精神疾病中关键的基因和通路。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>大脑紊乱影响着全世界约1/5的人口，虽然通过GWAS已经确定了很多与精神疾病相关的遗传变异，如Psychiatric Genomics Consortium (PGC)已经鉴定了到142个GWAS位点与精神分裂症有关（SCZ），然而人们对其分子机制了解很少。</li>
<li>许多这些遗传变异处于非编码区域，大量研究表明遗传和表观遗传活动的变化与这些遗传变异相关。 特别是，基因型-组织表达（GTEx）项目已把许多非编码变异和eQTLs关联起来，以及ENCODE和路线图表观基因组学（roadmap）表观基因组项目已经确定了非编码区域的增强子和启动子。</li>
<li>然而上述项目对于人脑的相关研究甚少。之前的一些研究工作关于脑特异性功能提供了一些并不全面的见解。 除此之外，新的研究方法、技术如Hi-C和单细胞测序，还没有<br>与大脑基因组学数据完全整合。因此PsychENCODE联盟应运而生。</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-资源构建-Resource-construction"><a href="#3-1-资源构建-Resource-construction" class="headerlink" title="3.1 资源构建 (Resource construction)"></a>3.1 资源构建 (Resource construction)</h4><ul>
<li>如下图所示：PsychENCODE按层次结构组织数据 (organizes data hierarchically). 底层原始文件整合了来自于PsychENCODE所有的成年大脑数据，并将他们与ENCODE、CMC、GTEx、Roadmap以及单细胞数据进行整合，总共包含来自于PsychENCODE的3810个基因型、转录组、染色质、Hi-C数据集，以及来自于其他脸们的1662个数据集。总体而言，<strong>前额叶皮层（PFC）的数据集涉及1866个人的样本；</strong></li>
<li>同时还有PsychENCODE中的18025个单细胞RNA-Seq数据、14012个其他来源的单细胞RNA-Seq数据；<br><img src="/images/20190401-Fig1.png" alt="Figure 0"></li>
</ul>
<h4 id="3-2-转录组分析-Transcriptome-analysis-Bulk-and-single-cell"><a href="#3-2-转录组分析-Transcriptome-analysis-Bulk-and-single-cell" class="headerlink" title="3.2 转录组分析 (Transcriptome analysis: Bulk and single cell)"></a>3.2 转录组分析 (Transcriptome analysis: Bulk and single cell)</h4><ul>
<li>为了确定大脑特异基因组调控元件，作者使用了一个保守的方法，即：ENCODE建立标准的RNA-Seq pipeline来处理PcychENCODE、GTEx和Roadmap中的RNA-Seq数据；<br><img src="/images/20190401-Fig2.png" alt="Figure S2"></li>
<li>脑组织中有各种各样基础的细胞类型组成，<strong>组织水平上观察到的表达水平的变化可能是由于基础细胞水平上表达的变化</strong>；但是，细胞比例的这些变化如何影响个体中观察到的组织水平基因表达的变化仍然不清楚。为了解决这个问题，作者使用了在1866个个体对应的群体中使用了两个互补的策略：<ul>
<li><strong>策略1:</strong> 使用标准pipeline处理PsychENCODE和其他数据资源中的single-cell RNA-seq数据，然后进行拼接脑细胞类型的概况，包括<strong>兴奋性和抑制性神经元（根据先前的惯例分别表示为Ex1至Ex9和In1至In8）</strong>，<strong>主要的非神经元类型（例如，小胶质细胞和星形胶质细胞）</strong>，以及相关的其他<strong>与发育相关的细胞类型</strong>；作者提供了TPM值以及UMI count两种基因表达profile；对于每种profile，都可以生成一个expression signatures的表达矩阵\(C\), 由marker gene以及他们在不同细胞类型中的表达水平组成；在这个矩阵中，作者发现一些基因在细胞水平上变化比组织水平上变化范围大很多，例如多巴胺受体基因DRD3（<strong>Figure 2A</strong>），这表明<strong>个体之间细胞类型的改变对于个体水平表达差异的变化贡献很大</strong>；</li>
<li><strong>策略2:</strong> 使用非监督分析确定bulk expression variation（也就是使用非scRNA-Seq）的主要成分，使用NMF (nonnegative matrix factorization;\(B\approx VH\))确定排名靠前的components (捕获了\(V\)中大部分的变异<strong>Figure 2B</strong>)是否与单细胞得到的结果一致（<strong>Figure 2C</strong>）；<br><img src="/images/20190401-Fig3.jpg" alt="Figure 2"><br><img src="/images/20190401-Fig4.png" alt="Figure 2-legend"></li>
</ul>
</li>
<li>从<strong>Figure 2C</strong>可以看出NMF-TCs与矩阵\(C\)（不论是TPM还是UMI）高度相关，例如：NMF-17与Ex2细胞类型相关（相关系数\(r=0.63\)）, 这表明<strong>从bulk expression的无监督分析可以粗略地概括单细胞特征。</strong></li>
<li>接下来作者检查<strong>基础细胞类型比例的变化</strong>对于<strong>大量基因表达的变化</strong>有多大的贡献，为达此目的，作者首先计算了每个组织中不同细胞类型的相对比例(<strong>cell fractions</strong>)。<strong>特别地：</strong>作者通过使用单细胞signatures去估计个体间cell fractions (\(W\))来实现组织水平表达矩阵的去卷积(\(B\approx CW\)), 然后用之前实验验证的NEU<sup>+/-</sup>细胞证实结果的可靠性，总之，这部分分析说明<strong>细胞类型的变化对于bulk variation贡献很大</strong>，也就是说单细胞signatures的加权组合占到群体表达变异的大部分，精度大于88% (<strong>Figure 2D; \(1-||B-CW||^2/||B||^2&gt;88\%\)</strong>)；</li>
<li>作者确定于与不同性状相关的细胞比例的变化(<strong>Figure 2E</strong>), 例如：兴奋性神经元和抑制性神经元在男性和女性中比例不同；而且在ASD患者中，兴奋性神经元<strong>Ex5</strong>的比例更高，而在少突胶质细胞中比例较低、小胶质细胞和星型胶质细胞比例相应增加；</li>
<li>最后作者还发现与年龄的关系，随着年龄的增大，兴奋性神经元<strong>Ex3、Ex4</strong>显著增加，一些非神经元类型的比例降低（<strong>Figure 2F</strong>），这些改变可能与一些基因的差异表达有关，例如生长激素抑制剂(somatostatin; SST)基因，已被证明与衰老和神经船体有关，而且，随着年龄增大，SST基因展现出启动子去甲基化的增加，这也就解释了基因表达为什么会随着年龄增大而降低；其他基因如EGR1 (early growth response)和CP展现了不同的趋势(<strong>Figure 2F</strong>)；</li>
</ul>
<h4 id="3-3-增强子"><a href="#3-3-增强子" class="headerlink" title="3.3 增强子"></a>3.3 增强子</h4><ul>
<li>在这部分，作者为了鉴定脑中有活性的增强子，使用了Roadmap中PFC样本的DNase-seq和ChIP-seq数据集，使用ENCODE ChIP-seq标准pipeline进行处理，与ENCODE一致，将富集在H3K27ac中的开放染色质区域而在H3K4me3中缺乏的定义为<strong>active enhancers</strong> (<strong>Figure 3A</strong>), 共得到79056个增强子;</li>
<li>与差异表达相比，差异enhancers更难，因为增强子的边界可以延长或者收缩；作者为了挖掘这个信息，对于来自于PFC、TC和CB等50个个体的的H3K27ac数据进行标准化处理；将这些数据汇总起来共得到27761个 H3K27 peaks in the PFC, 42,683 in the TC, and 26,631 in the CB, 每一个peak都在至少一半的个体中存在；比较发现：PFC与TC更相似，这个结果与之前的研究一致；</li>
<li>作者随后确定了在reference brain中有活性的增强子在自己研究群体的个体中有多少是有活性的，与预期相符合：并不是每一个reference enhancer在个体中都是有活性的；只有70%的增强子在一个个体中是活性的 (<strong>Figure 3B</strong>)；<br><img src="/images/20190401-Fig5.jpg" alt="Figure 3"><br><img src="/images/20190401-Fig6.png" alt="Figure 3-legend"></li>
</ul>
<h4 id="3-4-一致性比较：转录组和表观基因组-Consistent-comparison-Transcriptome-and-epigenome"><a href="#3-4-一致性比较：转录组和表观基因组-Consistent-comparison-Transcriptome-and-epigenome" class="headerlink" title="3.4 一致性比较：转录组和表观基因组 (Consistent comparison: Transcriptome and epigenome)"></a>3.4 一致性比较：转录组和表观基因组 (Consistent comparison: Transcriptome and epigenome)</h4><ul>
<li>作者在这部分从比较了大脑与其他器官的转录组变异与表观基因组变异（<strong>Figure 3C-F</strong>），使用了PCA，t-SNE和RCA(reference component analysis)，然后确定一个最好的方法来比较，这部分详细内容可见图注，在此不再赘述；</li>
</ul>
<h4 id="3-5-QTL分析"><a href="#3-5-QTL分析" class="headerlink" title="3.5 QTL分析"></a>3.5 QTL分析</h4><ul>
<li>这部分作者利用这些数据确定影响基因表达和染色质活动的QTLs，分别计算了影响表达的eQTL，影响剪切的isoQTLs、影响细胞类型比例的fQTLs和影响染色质的cQTLs；</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/">https://jingjingzhai.github.io/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/深度学习/">深度学习</a><a href="/tags/Human-brain/">Human brain</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/04/03/DeepCAPE-a-deep-convolutional-neural-network-for-the-accurate-prediction-of-enhancers/" class="pre">DeepCAPE: a deep convolutional neural network for the accurate prediction of enhancers</a><a href="/2019/04/01/A-universal-sNP-and-small-indel-variant-caller-using-deep-neural-networks/" class="next">A universal SNP and small-indel variant caller using deep neural networks</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-资源构建-Resource-construction"><span class="toc-text">3.1 资源构建 (Resource construction)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-转录组分析-Transcriptome-analysis-Bulk-and-single-cell"><span class="toc-text">3.2 转录组分析 (Transcriptome analysis: Bulk and single cell)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-增强子"><span class="toc-text">3.3 增强子</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-一致性比较：转录组和表观基因组-Consistent-comparison-Transcriptome-and-epigenome"><span class="toc-text">3.4 一致性比较：转录组和表观基因组 (Consistent comparison: Transcriptome and epigenome)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-5-QTL分析"><span class="toc-text">3.5 QTL分析</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/07/17/A-Deep-Neural-Network-for-Predictingand-Engineering-Alternative-Polyadenylation/">A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/24/Functional-architecture-of-low-frequency-variants-highlights-strength-of-negative-selection-across-coding-and-non-coding-annotations/">Functional architecture of low-frequency variants highlights strength of negative selection across coding and non-coding annotations</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/APA预测/" style="font-size: 15px;">APA预测</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/08/">八月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>