<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>DeepTACT: Predicting 3D chromatin contacts via bootstrapping deep learning | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">DeepTACT: Predicting 3D chromatin contacts via bootstrapping deep learning</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">DeepTACT: Predicting 3D chromatin contacts via bootstrapping deep learning</h1><div class="post-meta"><a href="/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/#comments" class="comment-count"></a><p><span class="date">Mar 18, 2019</span><span><a href="/categories/NAR/" class="category">NAR</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2019年3月18日发表于NAR期刊上使用深度学习预测3D基因组相互作用的文章。通讯作者为清华大学自动化系的<a href="http://bioinfo.au.tsinghua.edu.cn/member/ruijiang/chinese/index.html" target="_blank" rel="noopener">江瑞老师实验室</a>。这篇文章在网络结构上使用了网络merge的策略，整合序列以及DNase特征，merge之后还使用了BLSTM进一步提取特征，同时还使用了目前火热的attention layer。 在数据上使用集成学习(Bootstrap)以及Data augmentation保证深度学习的数据量。<br><a id="more"></a></p>
<p><a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz167/5380496" target="_blank" rel="noopener">文章链接</a><br><a href="https://github.com/liwenran/DeepTACT" target="_blank" rel="noopener">文章代码</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>调控元件的相互作用对于理解转录调控以及疾病机制的解析至关重要(of crucial importance),然而目前的HiC技术只有超高分辨率才能捕获这种相互作用；</li>
<li>作者在这里提出了一种基于bootstrapping的深度学习模型DeepTACT，整合基因组序列以及染色质可及性预测调控元件chromatin contacts。DeepTACT不仅可以推断promoter-enhancer相互作用，还可以推断promoter-promoter相互作用；</li>
<li>DeepTACT确定了一类核心启动子，与不同细胞系的转录激活有关。通过与GWAS结合分析，IFNA2与冠状动脉疾病的关联说明了chrimatin contact在人类疾病研究中的应用。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>阐述调控元件相互作用对于SNP功能注释、疾病机制解析的的重要性；</li>
<li>3D基因组技术的发展，3C、ChIA-PET、HiC等；</li>
<li>计算生物学方法用于提高Hi-C分辨率以及在调控元件水平检测物理相互作用。i) HiC-Plus使用高分辨率的图像建模提高低分辨率HiC的分辨率，但最多只能提高到10kb；ii) EpiTensor,从组蛋白修饰、染色质可及性以及RNA-Seq的1D图谱中确定3D相互作用； iii) 3DEpiLoop从表观基因组数据以及转录因子profile预测chromatin loop；iv）TargetFinder整合TFs、histone marks、DNase-seq、expression and DNA methylation预测promoter-enhancer相互作用。这些方法都需要大量的表观基因组数据；</li>
<li>深度学习在计算生物学的应用；</li>
<li>DeepTACT使用bootstrapping deep learning在单个调控元件水平上使用序列特征和染色质可及性预测染色质contacts。DeepTACT能够将5-20kb的数据分辨率提升到1kb。</li>
</ul>
<h3 id="3-材料与方法"><a href="#3-材料与方法" class="headerlink" title="3. 材料与方法"></a>3. 材料与方法</h3><h4 id="3-1-数据收集与预处理-Data-collection-and-preprocessing"><a href="#3-1-数据收集与预处理-Data-collection-and-preprocessing" class="headerlink" title="3.1 数据收集与预处理 (Data collection and preprocessing)"></a>3.1 数据收集与预处理 (Data collection and preprocessing)</h4><ul>
<li>PCHi-C(Promoter capture Hi-C)数据,涉及到多个细胞类型。每个cell type有25148个high confident chromatin loops;</li>
<li>DNase-seq数据集；</li>
<li>Enhancer from FANTOM5，最后得到65 432 permissive enhancers；</li>
<li>ChIA-PET， 194467个loops；</li>
<li>eQTL、PPI、RNA-Seq、Histone等。</li>
</ul>
<h4 id="3-2-DeepTACT模型设计与训练策略"><a href="#3-2-DeepTACT模型设计与训练策略" class="headerlink" title="3.2 DeepTACT模型设计与训练策略"></a>3.2 DeepTACT模型设计与训练策略</h4><ul>
<li><p><strong>chromatin accessibility score</strong>的计算<br><img src="/images/20190318-Fig3.png" alt="Figure"></p>
<p>其中，N是落在该点的reads数目，W为该点附近的背景window size，落在window W区域的reads数目。</p>
</li>
</ul>
<ul>
<li>如下图所示，共有三个模块。其中<strong>Sequence Module</strong>中使用了两个卷积神经网络，分别代表相互作用的两个片段；<strong>Openness Module</strong>中使用<strong>chromatin accessibility score</strong>作为模型输入，同样是两个CNN；第三个模块将它们整合，同事还是用了BLSTM以及Attention Layer<br><img src="/images/20190318-Fig1.png" alt="Figure 1ABC"></li>
</ul>
<h4 id="3-3-深度神经网络"><a href="#3-3-深度神经网络" class="headerlink" title="3.3 深度神经网络"></a>3.3 深度神经网络</h4><p>在将四个CNN合并之后，作者使用bidirectional long- and short-term memory (BLSTM)进一步从pooled sequence patterns提取特征，随后还使用了attention layer，最后使用全连接层预测分类。还使用了batch normalization加速训练以及dropout避免过拟合。</p>
<h4 id="3-4-Bootstrapping策略"><a href="#3-4-Bootstrapping策略" class="headerlink" title="3.4 Bootstrapping策略"></a>3.4 Bootstrapping策略</h4><ul>
<li>采用有放回抽样（20次）、然后数据扩充（<strong>参见3.4</strong>）、分别训练、在使用集成学习（ensemble learning）对预测结果整合（取平均）。<br><img src="/images/20190318-Fig2.png" alt="Figure 1D"></li>
</ul>
<h4 id="3-5-数据扩充"><a href="#3-5-数据扩充" class="headerlink" title="3.5 数据扩充"></a>3.5 数据扩充</h4><p>为了获得大量训练数据，作者将原始训练数据扩充了20倍。以1kb的窗口滑动50bp与扫描正样本中（2kb）两个序列，将会得到400个组合（20×20），然后随机选20个作为正样本。对于负样本，基于与正样本的距离，生成20次。</p>
<h4 id="3-6-使用gene-enhancer网络对疾病相关的调控元件排序"><a href="#3-6-使用gene-enhancer网络对疾病相关的调控元件排序" class="headerlink" title="3.6 使用gene-enhancer网络对疾病相关的调控元件排序"></a>3.6 使用gene-enhancer网络对疾病相关的调控元件排序</h4><p>使用gene-enhancer网络中的随机游走策略为genes/enhancers打分。</p>
<h3 id="4-结果"><a href="#4-结果" class="headerlink" title="4. 结果"></a>4. 结果</h3><h4 id="4-1-DeepTACT可精确预测chromatin-contacts-DeepTACT-accurately-predicts-chromatin-contacts"><a href="#4-1-DeepTACT可精确预测chromatin-contacts-DeepTACT-accurately-predicts-chromatin-contacts" class="headerlink" title="4.1 DeepTACT可精确预测chromatin contacts (DeepTACT accurately predicts chromatin contacts)"></a>4.1 DeepTACT可精确预测chromatin contacts (DeepTACT accurately predicts chromatin contacts)</h4><ul>
<li>在六个细胞系中DeepTACT模型比较<br><img src="/images/20190318-Fig4.png" alt="Figure 2"></li>
</ul>
<h4 id="4-2-DeepTACT从启动子捕获Hi-C数据中获得了很好的启动子-启动子相互作用-DeepTACT-provides-finer-mapping-of-promoter-promoter-interactions-from-promoter-capture-Hi-C-data"><a href="#4-2-DeepTACT从启动子捕获Hi-C数据中获得了很好的启动子-启动子相互作用-DeepTACT-provides-finer-mapping-of-promoter-promoter-interactions-from-promoter-capture-Hi-C-data" class="headerlink" title="4.2 DeepTACT从启动子捕获Hi-C数据中获得了很好的启动子-启动子相互作用 (DeepTACT provides finer mapping of promoter-promoter interactions from promoter capture Hi-C data)"></a>4.2 DeepTACT从启动子捕获Hi-C数据中获得了很好的启动子-启动子相互作用 (DeepTACT provides finer mapping of promoter-promoter interactions from promoter capture Hi-C data)</h4><ul>
<li>PCHi-C技术可以确定相互作用的区域，但是每个区域的长度取决于测序的分辨率。这部分主要讲述如何应用上述构建的模型得到低分辨率的相互作用。<br><img src="/images/20190318-Fig5.png" alt="Figure 3"></li>
<li><strong>Figure 3ABC:</strong> 对于PCHi-C测序捕获到的长的相互作用，先搜索其中的enhancer以及promoter，两两组合预测其相互作用的可能性;</li>
<li><strong>Figure 3DE:</strong> 使用不同类型的数据对预测结果进行验证。</li>
</ul>
<h4 id="4-3-DeepTACT从启动子捕获Hi-C数据中获得了更好的启动子-增强子相互作用-DeepTACT-provides-finer-mapping-of-promoter-enhancer-interactions-from-promoter-capture-Hi-C-data"><a href="#4-3-DeepTACT从启动子捕获Hi-C数据中获得了更好的启动子-增强子相互作用-DeepTACT-provides-finer-mapping-of-promoter-enhancer-interactions-from-promoter-capture-Hi-C-data" class="headerlink" title="4.3 DeepTACT从启动子捕获Hi-C数据中获得了更好的启动子-增强子相互作用 (DeepTACT provides finer mapping of promoter-enhancer interactions from promoter capture Hi-C data)"></a>4.3 DeepTACT从启动子捕获Hi-C数据中获得了更好的启动子-增强子相互作用 (DeepTACT provides finer mapping of promoter-enhancer interactions from promoter capture Hi-C data)</h4><ul>
<li>这部分主要用于证明DeepTACT可用来预测<strong>启动子-增强子</strong>相互作用<br><img src="/images/20190318-Fig6.png" alt="Figure 4"></li>
</ul>
<h4 id="4-4-通过预测得出的核心启动子的描述-Characterization-of-hub-promoters-defined-by-predicted-interactions"><a href="#4-4-通过预测得出的核心启动子的描述-Characterization-of-hub-promoters-defined-by-predicted-interactions" class="headerlink" title="4.4 通过预测得出的核心启动子的描述 (Characterization of hub promoters defined by predicted interactions)"></a>4.4 通过预测得出的核心启动子的描述 (Characterization of hub promoters defined by predicted interactions)</h4><ul>
<li>这部分内容主要用于描述核心启动子的功能，<strong>核心启动子</strong>是作者根据预测结果选取的前10%最易发生相互作用的启动子。<br><img src="/images/20190318-Fig7.png" alt="Figure 5ABD"></li>
<li><strong>Figure 5A:</strong> 首先查看核心启动子在转录激活区域的overlap情况，使用了四个组蛋白信号（H3K4me3, H3K27ac, H3K4me1和H3K4me2);</li>
<li><strong>Figure 5B:</strong> 从ENCODE中收集了579个转录因子，计算启动子在cell type中是激活的对应的细胞类型的数目；还分析了与Housekeeping基因的关系；</li>
<li><strong>Figure 5D:</strong> GO富集分析推测核心启动子的功能；<br><img src="/images/20190318-Fig8.png" alt="Figure 5ABD"></li>
<li><strong>Figure 5C:</strong> 核心启动子在PPI网络中平均degree的分析；</li>
<li><strong>Figure 5D:</strong> REACTOME富集分析推测核心启动子的功能；</li>
<li><strong>Figure 5F:</strong> 每个细胞类型中核心启动子的聚类；</li>
<li><strong>Figure 5G-5H:</strong> 不同细胞类型中核心启动子的overlap，表明核心启动子可以反映细胞类型的相似性。</li>
</ul>
<h4 id="4-5-使用预测的相互作用确定疾病相关的调控原件-Identification-of-disease-related-regulatory-elements-using-predicted-interactions"><a href="#4-5-使用预测的相互作用确定疾病相关的调控原件-Identification-of-disease-related-regulatory-elements-using-predicted-interactions" class="headerlink" title="4.5 使用预测的相互作用确定疾病相关的调控原件 (Identification of disease-related regulatory elements using predicted interactions)"></a>4.5 使用预测的相互作用确定疾病相关的调控原件 (Identification of disease-related regulatory elements using predicted interactions)</h4><ul>
<li>最后一部分作者使用random walk with restart算法对网络中的调控元件进行排序，现将GWAS的SNP映射到对应的调控元件上，然后使用p值作为网络随机游走的初始概率，通过迭代来不断获得每个节点的稳态分布时对应的概率。下图展示了一个例子，就是在原始GWAS信号中不显著的位点，通过随机游走排名第二，而且发现<em>IFNA2</em>这个基因没有显著的SNP。</li>
</ul>
<p><img src="/images/20190318-Fig9.png" alt="Figure 7"></p>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/">https://jingjingzhai.github.io/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/深度学习/">深度学习</a><a href="/tags/chromatin-contacts-prediction/">chromatin contacts prediction</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/" class="pre">Predicting splicing from primary sequence with deep learning</a><a href="/2019/03/13/Network-propagation-a-universal-amplifier-of-genetic-associations/" class="next">Network propagation: a universal amplifier of genetic associations</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-材料与方法"><span class="toc-text">3. 材料与方法</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-数据收集与预处理-Data-collection-and-preprocessing"><span class="toc-text">3.1 数据收集与预处理 (Data collection and preprocessing)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-DeepTACT模型设计与训练策略"><span class="toc-text">3.2 DeepTACT模型设计与训练策略</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-深度神经网络"><span class="toc-text">3.3 深度神经网络</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-Bootstrapping策略"><span class="toc-text">3.4 Bootstrapping策略</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-5-数据扩充"><span class="toc-text">3.5 数据扩充</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-6-使用gene-enhancer网络对疾病相关的调控元件排序"><span class="toc-text">3.6 使用gene-enhancer网络对疾病相关的调控元件排序</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#4-结果"><span class="toc-text">4. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#4-1-DeepTACT可精确预测chromatin-contacts-DeepTACT-accurately-predicts-chromatin-contacts"><span class="toc-text">4.1 DeepTACT可精确预测chromatin contacts (DeepTACT accurately predicts chromatin contacts)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-2-DeepTACT从启动子捕获Hi-C数据中获得了很好的启动子-启动子相互作用-DeepTACT-provides-finer-mapping-of-promoter-promoter-interactions-from-promoter-capture-Hi-C-data"><span class="toc-text">4.2 DeepTACT从启动子捕获Hi-C数据中获得了很好的启动子-启动子相互作用 (DeepTACT provides finer mapping of promoter-promoter interactions from promoter capture Hi-C data)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-3-DeepTACT从启动子捕获Hi-C数据中获得了更好的启动子-增强子相互作用-DeepTACT-provides-finer-mapping-of-promoter-enhancer-interactions-from-promoter-capture-Hi-C-data"><span class="toc-text">4.3 DeepTACT从启动子捕获Hi-C数据中获得了更好的启动子-增强子相互作用 (DeepTACT provides finer mapping of promoter-enhancer interactions from promoter capture Hi-C data)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-4-通过预测得出的核心启动子的描述-Characterization-of-hub-promoters-defined-by-predicted-interactions"><span class="toc-text">4.4 通过预测得出的核心启动子的描述 (Characterization of hub promoters defined by predicted interactions)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-5-使用预测的相互作用确定疾病相关的调控原件-Identification-of-disease-related-regulatory-elements-using-predicted-interactions"><span class="toc-text">4.5 使用预测的相互作用确定疾病相关的调控原件 (Identification of disease-related regulatory elements using predicted interactions)</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/07/17/A-Deep-Neural-Network-for-Predictingand-Engineering-Alternative-Polyadenylation/">A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">Single-cell RNA-seq denoising using a deep count autoencoder</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/APA预测/" style="font-size: 15px;">APA预测</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/08/">八月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>