<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome</h1><div class="post-meta"><a href="/2019/04/09/Virtual-ChIP-seq-predicting-transcription-factor-binding-by-learning-from-the-transcriptome/#comments" class="comment-count"></a><p><span class="date">Apr 09, 2019</span><span><a href="/categories/bioRxiv/" class="category">bioRxiv</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p><a href="https://www.biorxiv.org/content/10.1101/168419v4" target="_blank" rel="noopener">文章链接</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li><strong>动机</strong>: 确定转录因子结合位点是解析非编码突变破坏转录因子调控功能并促进疾病研究的第一步；ChIP-Seq是确定结合位点的最普遍的方法，但是在病人样本中实施ChIP-Seq实验会受到生物学材料以及价格的限制；已经存在的调控元件预测的计算方法主要根据与已知的TF binding sequences之间的相似性预测；但是大部分结合位点跟已知的转录因子序列motifs不相似而且很多转录因子甚至不是序列特异的；</li>
<li><strong>结果</strong>: 作者开发了Virtual ChIP-seq，利用人工神经网络通过整合其他细胞类型的ChIP-Seq数据和染色质可及性数据来预测新的细胞类型里的ChIP-seq结果；同时，Virtual ChIP-seq还学习到了特定基因组区域中基因表达与转录因子结合位点之间的相关性；这个方法比单纯的基于序列效果要好。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>转录因子调控基因表达，每个转录因子通过结合到基因组上来协调基因的表达。细胞内的机器利用这些regulators调节关键的细胞过程并适应环境的刺激；一个TF的序列或者量发生改变会影响很多基因的表达；事实上，这些改变也是遗传性疾病 (hereditary disorders)、复杂疾病、自身免疫缺陷以及癌症的主要原因；</li>
<li>转录因子结合到DNA序列是基于氨基酸残基与核苷酸序列之间比较弱的非共价相互作用；DNA的原始序列、二级结构 (shape)、三级结构 (conformation) 对于TF binding都发挥着作用；很多TF与其他TF或者染色质结合蛋白形成复合体间接的结合到DNA上；很多体外的高通量实验（HT-SELEX or PBM）已经对于TF结合序列提供了全面的理解；但是，用体外数据训练的模型应用到体内实验的时候性能表现很差；</li>
<li>ChIP-Seq、ChIP-exo、ChIP-nexus可以将给定TF的结合序列找出来，但是这些实验至少需要1,000,000到100,000,000个细胞，而且还去决定这个TF是否有抗体；在临床实验中，这么大量的细胞并不一定可以获得；但是ATAC-seq只需要几百到几千个细胞，在临床样本中可以获得。然而染色质可及性不能决定到底哪个转录因子会结合到DNA上，目前也有一些方法利用这些信息与TF binding sequences偏好、基因组保守性以及其他基因组特征来预测TF结合；</li>
<li>使用motif discovery工具预测TF binding可以帮助我们理解不同疾病中TF扮演的角色；</li>
<li>不同软件的评估：<ul>
<li>auROC不合适，因为TF binding数据是高度不平衡的；</li>
<li>benchmark数据集不同；</li>
</ul>
</li>
<li>RNA-Seq数据集可以获得病人样本的转录组数据，作者假设转录组的数据可以更好的用于TF binding的预测，之前有研究使用调控元件预测基因表达；也有人用基因表达预测染色质可及性或者用基因表达的差异确定特定条件下统计显著的sequence motifs，但是他们没有根据转录组的数据预测体内的TF binding；</li>
<li>因此作者提出了Virtual ChIP-Seq，通过学习公共数据库中的ChIP-seq实验从而预测TF binding；Virtual ChIP-Seq还学习了还学习到了基因组保守性、基因表达等信息；Virtual ChIP-seq在新的细胞类型中MCC最小值为0.3。</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-大多数TF结合位点缺乏motifs-Sequence-motifs-are-absent-in-most-TF-binding-sites"><a href="#3-1-大多数TF结合位点缺乏motifs-Sequence-motifs-are-absent-in-most-TF-binding-sites" class="headerlink" title="3.1 大多数TF结合位点缺乏motifs (Sequence motifs are absent in most TF binding sites)"></a>3.1 大多数TF结合位点缺乏motifs (Sequence motifs are absent in most TF binding sites)</h4><h5 id="3-1-1-大多数ChIP-seq-peaks缺乏转录因子相关的序列motif-Most-ChIP-seq-peaks-lack-the-TF’s-relevant-sequence-motif"><a href="#3-1-1-大多数ChIP-seq-peaks缺乏转录因子相关的序列motif-Most-ChIP-seq-peaks-lack-the-TF’s-relevant-sequence-motif" class="headerlink" title="3.1.1 大多数ChIP-seq peaks缺乏转录因子相关的序列motif (Most ChIP-seq peaks lack the TF’s relevant sequence motif)"></a>3.1.1 大多数ChIP-seq peaks缺乏转录因子相关的序列motif (Most ChIP-seq peaks lack the TF’s relevant sequence motif)</h5><ul>
<li>大部分用来预测TF结合的工具都是使用序列偏好性数据，即PWM矩阵；但是只基于序列不能决定转录因子的结合。我们可以从高通量实验中获得一些额外的属性例如：co-binding partners；作者在这里使用了54个细胞类型的201个chromatin factors，作者首先探究是不是大多数的结合位点都match同一个转录因子的sequence motif；</li>
<li><strong>Figure 1A:</strong> 在这201个蛋白中，76个在JASPAR数据库中没有motif， 还有一些motif-free蛋白是chromatin-binding proteins但不是真正的TFs<br><img src="/images/20190409-Fig1.jpg" alt="Figure 1"><br><img src="/images/20190409-Fig2.png" alt="Figure 1-legend"></li>
<li><strong>Figure 1B:</strong> 对于序列特异性的TFs，peaks match序列motif的比例从4.55%(SIX5)到94.2%(CTCF), 均值为49.4%；</li>
</ul>
<h5 id="3-1-2-很多序列motifs都不是中心富集的-Many-sequence-motifs-are-not-centrally-enriched"><a href="#3-1-2-很多序列motifs都不是中心富集的-Many-sequence-motifs-are-not-centrally-enriched" class="headerlink" title="3.1.2 很多序列motifs都不是中心富集的 (Many sequence motifs are not centrally enriched)"></a>3.1.2 很多序列motifs都不是中心富集的 (Many sequence motifs are not centrally enriched)</h5><ul>
<li><strong>Figure 1C:</strong> 作者比较了low motif occupancy TFs与high motif occupancy中central enrichment的程度；low motif occupancy had waker central enrichment；</li>
<li><strong>Figure 1D:</strong> 一个示例；</li>
</ul>
<h4 id="3-2-模型、性能以及基准数据"><a href="#3-2-模型、性能以及基准数据" class="headerlink" title="3.2 模型、性能以及基准数据"></a>3.2 模型、性能以及基准数据</h4><h5 id="3-2-1-数据集"><a href="#3-2-1-数据集" class="headerlink" title="3.2.1 数据集"></a>3.2.1 数据集</h5><ul>
<li><strong>标签:</strong> 对于基因组上200bp的片段如果与peak summit overlap则认为是正样本；</li>
</ul>
<h5 id="3-2-2-Learning-from-the-transcriptome"><a href="#3-2-2-Learning-from-the-transcriptome" class="headerlink" title="3.2.2 Learning from the transcriptome"></a>3.2.2 Learning from the transcriptome</h5><ul>
<li><strong>Figure 2A-2C:</strong> 对于每一个chromatin factor，作者创建了相关性矩阵，计算特定chromatin factor在特定基因组区域的binding与表达值的相关系数；<br><img src="/images/20190409-Fig3.jpg" alt="Figure 2"><br><img src="/images/20190409-Fig4.png" alt="Figure 2-legend"></li>
</ul>
<h5 id="3-2-3-Learning-from-other-predictive-features"><a href="#3-2-3-Learning-from-other-predictive-features" class="headerlink" title="3.2.3 Learning from other predictive features"></a>3.2.3 Learning from other predictive features</h5><h5 id="3-2-4-超参选取与训练-Selecting-hyperparameters-and-training"><a href="#3-2-4-超参选取与训练-Selecting-hyperparameters-and-training" class="headerlink" title="3.2.4 超参选取与训练 (Selecting hyperparameters and training)"></a>3.2.4 超参选取与训练 (Selecting hyperparameters and training)</h5><ul>
<li>文章共用了以下特征：<strong>表达值(Figure 3A)</strong>、<strong>公共数据库中的证据信息(Figure 3B)</strong>、<strong>chromatin accessibility (Figure 3C)</strong>、<strong>基因组保守性(Figure 3D)</strong>、<strong>序列motif分值(Figure 3E)</strong>;</li>
<li>chr5, chr10, chr15, and chr20用于训练；<br><img src="/images/20190409-Fig5.jpg" alt="Figure 3"></li>
</ul>
<h5 id="3-2-5-多层感知机"><a href="#3-2-5-多层感知机" class="headerlink" title="3.2.5 多层感知机"></a>3.2.5 多层感知机</h5></div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/04/09/Virtual-ChIP-seq-predicting-transcription-factor-binding-by-learning-from-the-transcriptome/">https://jingjingzhai.github.io/2019/04/09/Virtual-ChIP-seq-predicting-transcription-factor-binding-by-learning-from-the-transcriptome/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/TFBS-Prediction/">TFBS Prediction</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/04/12/Opportunities-and-challenges-for-transcriptome-wide-association-studies/" class="pre">Opportunities and challenges for transcriptome- wide association studies</a><a href="/2019/04/03/Dysregulation-of-expression-correlates-with-rare-allele-burden-and-fitness-loss-in-maize/" class="next">Dysregulation of expression correlates with rare-allele burden and fitness  loss in maize</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-大多数TF结合位点缺乏motifs-Sequence-motifs-are-absent-in-most-TF-binding-sites"><span class="toc-text">3.1 大多数TF结合位点缺乏motifs (Sequence motifs are absent in most TF binding sites)</span></a><ol class="toc-child"><li class="toc-item toc-level-5"><a class="toc-link" href="#3-1-1-大多数ChIP-seq-peaks缺乏转录因子相关的序列motif-Most-ChIP-seq-peaks-lack-the-TF’s-relevant-sequence-motif"><span class="toc-text">3.1.1 大多数ChIP-seq peaks缺乏转录因子相关的序列motif (Most ChIP-seq peaks lack the TF’s relevant sequence motif)</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-1-2-很多序列motifs都不是中心富集的-Many-sequence-motifs-are-not-centrally-enriched"><span class="toc-text">3.1.2 很多序列motifs都不是中心富集的 (Many sequence motifs are not centrally enriched)</span></a></li></ol></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-模型、性能以及基准数据"><span class="toc-text">3.2 模型、性能以及基准数据</span></a><ol class="toc-child"><li class="toc-item toc-level-5"><a class="toc-link" href="#3-2-1-数据集"><span class="toc-text">3.2.1 数据集</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-2-2-Learning-from-the-transcriptome"><span class="toc-text">3.2.2 Learning from the transcriptome</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-2-3-Learning-from-other-predictive-features"><span class="toc-text">3.2.3 Learning from other predictive features</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-2-4-超参选取与训练-Selecting-hyperparameters-and-training"><span class="toc-text">3.2.4 超参选取与训练 (Selecting hyperparameters and training)</span></a></li><li class="toc-item toc-level-5"><a class="toc-link" href="#3-2-5-多层感知机"><span class="toc-text">3.2.5 多层感知机</span></a></li></ol></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/07/17/A-Deep-Neural-Network-for-Predictingand-Engineering-Alternative-Polyadenylation/">A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/24/Functional-architecture-of-low-frequency-variants-highlights-strength-of-negative-selection-across-coding-and-non-coding-annotations/">Functional architecture of low-frequency variants highlights strength of negative selection across coding and non-coding annotations</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a> <a href="/tags/APA预测/" style="font-size: 15px;">APA预测</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/08/">八月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>