<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Predicting splicing from primary sequence with deep learning | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Predicting splicing from primary sequence with deep learning</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Predicting splicing from primary sequence with deep learning</h1><div class="post-meta"><a href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/#comments" class="comment-count"></a><p><span class="date">Mar 20, 2019</span><span><a href="/categories/Cell/" class="category">Cell</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2019年1月发表在Cell期刊用深度学习预测可变剪切。通讯作者来自于<a href="https://www.illumina.com/" target="_blank" rel="noopener">Illumina人工智能实验室</a>。文章使用了火热的ResNet预测可变剪切，模型足够复杂（32个卷积）而且预测结果看起来是真的好，top-k accuracy高达95%，甩其他软件几条街，结果的分析更是详实复杂。<a id="more"></a>最后说一句，CNS正刊的文章内容是真的丰富。</p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867418316295" target="_blank" rel="noopener">文章连接</a><br><a href="https://basespace.illumina.com/s/5u6ThOblecrh" target="_blank" rel="noopener">SpliceAI model training</a><br><a href="https://github.com/Illumina/SpliceAI" target="_blank" rel="noopener">SpliceAI software</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>前体mRNA的剪接非常精确，但是背后的机制却还不清晰，作者描述了一种深度神经网络精确的预测splice junctions；</li>
<li>在预测出来影响可变剪切的同义突变和内含子突变中，使用RNA-Seq数据验证很高比例的有害突变；而且在预测出来的突变（影响可变剪切）他们显著富集在自闭症和智力残疾的患者中；</li>
<li>作者估计，罕见遗传性疾病患者中9％-11％的致病突变是由此前未被充分认识的疾病变异引起的。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>外显子组测序改变了患有罕见遗传疾病的患者和家属的临床诊断，并且当作为一线测试时，显着减少了诊断odyssey(a long trip or period involving a lot of different and exciting activities, esp. while searching for something；来自Cambridge Dictionary)的时间和成本；但是外显子测序诊断罕见疾病只有25%-30%，仍然有大部分患者没有被诊断出来；</li>
<li>基因组的非编码区域在基因调控中扮演者重要的角色，占了causal disease loci的90%。事实上，非编码突变可以破坏mRNA的剪切尽管他们落在GT和AG之外，这些变异被成为<strong>cryptic splice variants</strong>, 长期以来被认为在罕见疾病中发挥着重要的角色。但是在临床治疗中，cryptic splice mutations却一直被忽略，主要是因为对于splice code理解不够深入，这也为精确的预测他们提供了很大的困难；</li>
<li>最近，RNA测序（RNA-seq）已经成为检测孟德尔疾病中剪接异常的有希望的测定方法，但迄今为止它在临床环境中的应用仍然局限于相关细胞类型已知且可进入活组织检查的少数病例。对任意前mRNA序列的剪接的预测可以使得精确预测cryptic splice variants成为可能，从而改善遗传疾病患者的诊断。虽然目前在core splicing motifs, exonic splice enhancers and silencer以及预测exon inclusion方面取得了一定进展，但是从原始序列构建剪切的预测模型还是很困难。</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-使用深度学习从原始序列中精确的预测剪切-Accurate-prediction-of-splicing-from-a-primary-sequence-using-deep-learning"><a href="#3-1-使用深度学习从原始序列中精确的预测剪切-Accurate-prediction-of-splicing-from-a-primary-sequence-using-deep-learning" class="headerlink" title="3.1 使用深度学习从原始序列中精确的预测剪切 (Accurate prediction of splicing from a primary sequence using deep learning)"></a>3.1 使用深度学习从原始序列中精确的预测剪切 (Accurate prediction of splicing from a primary sequence using deep learning)</h4><p><img src="/images/20190320-Fig2.png" alt="Figure 1A"></p>
<ul>
<li><p>文章中提出了可变剪切预测模型<strong>SpliceAI</strong>, 如上图所示, 文章使用了大名鼎鼎的<a href="https://zhuanlan.zhihu.com/p/33613512" target="_blank" rel="noopener">残差网络</a>, 使用32空洞卷积层，使用上下游共10k（例如：以GT为中心）的序列（真的很长了。。。）训练，然后预测出该位点是<strong>Splice Acceptor</strong>, <strong>Splice Donor</strong>或者<strong>Neither</strong>。上图只是提供了简单框架，关于模型的详细结构，如下图所示：<br><img src="/images/20190320-Fig1.jpg" alt="Figure S1"></p>
</li>
<li><p>其中上图中的每一个RB就是一个残差模块（Residual Block），参数N，W，D分别表示：卷积核数目、卷积核大小（window size）以及空洞卷积空洞率。有关空洞卷积与标准卷积的差别，查看下图，图片来自于<a href="https://www.zhihu.com/question/54149221" target="_blank" rel="noopener">知乎</a>：</p>
</li>
<li>标准卷积<br><img src="/images/convolution.gif" alt="标准卷积"></li>
<li><p>空洞卷积 (dilated convolution)<br><img src="/images/dilated-convolution.gif" alt="空洞卷积"></p>
</li>
<li><p><strong>！！！接下来，作者就要验证他模型的可靠性了</strong>，首先呢，作者的评估指标是<strong>top-k accuracy</strong>，简单来说：k是随着测试集中对应类别样本的数目发生变化的，比如测试集中Splice Donor有100个，那就看预测为Splice Donor的前100个中有多少个是真正的Splice Donor，作者说，SliceAI的<strong>top-k accuracy</strong>是0.95。然后就举了一个例子，基因<em>CFTR</em>，这个基因超过100kb，通过SpliceAI也能将它的基因结构准确的预测出来，如下图<strong>Figure 1B</strong>所示：<br><img src="/images/20190320-Fig4.png" alt="Figure 1B"></p>
</li>
<li><p><strong>Figure 1C:</strong> 对于测试集中GENCODE注释出来的每一个外显子，作者都检验了网络的预测分值与<strong>支持外显子包含reads数目除以支持外显子跳跃的reads数目（也就是图中的Exon inclussion rate）</strong>之间的相关性；<br><img src="/images/20190320-Fig5.png" alt="Figure 1CEF"><br><img src="/images/20190320-Fig6.png" alt="Figure 1DG"></p>
</li>
<li><p><strong>FIgure 1D:</strong> 接下来就要看sequence motif了，模型这么高的精度到底是哪些序列因素决定的。然后就做了一个模拟，对于已知外显子附近的核苷酸进行一一替换，然后看预测分值的变化。然后作者发现破坏splice donor motif会导致网络失去预测上游的splice acceptor位点的能力；</p>
</li>
<li><strong>Figure 1E:</strong> 这部分就是讲述上下游序列长度对于模型预测效果的影响，共做了四个梯度，80nt、400nt、2k和10k，其实2k的时候效果就已经很好了；</li>
<li><strong>Figure 1F:</strong> 通过比较80nt和10k的模型，检验long-range and short range specificity determinants。左图黄色是外显子长度分布、右图粉色是内含子长度分布，紫色线是10k模型的预测分值、蓝色线是80nt的预测分值；</li>
<li><strong>Figure 1G:</strong> 由于剪接发生共转录，染色质状态和共转录剪接之间的相互作用也可能指导外显子定义，并且有可能被网络利用到染色质状态可从前体序列预测的程度。全基因组核小体定位实验表明核小体占据在外显子区域更高。作者就看自己的模型是不是用了核小体定位的序列模式（这个图理解的并不是很透彻。。）。然后就用一对最优的acceptor和donor（间隔150nt；平均外显子长度），看这对motif是不是会导致外显子保留，发现哪些倾向于被预测为外显子的区域拥有很高的核小体定位，这表明SpliceAI也学习到了核小体定位的一些信息；</li>
<li><strong>Figure 1H:</strong> 虽然多项研究报道了外显子与核小体占据之间的相关性，但核小体定位在外显子定义中的因果作用尚未确定。然后作者使用GTEx中149个体的RNA-Seq和全基因组测序数据，确定了一些专属于个人的新的外显子，这些私有外显子与已经存在的核小体定位显著相关。</li>
</ul>
<h4 id="3-2-用RNA-Seq验证预测的cryptic-splice-mutation-Verification-of-Predicted-Cryptic-Splice-Mutations-in-RNA-Seq-Data"><a href="#3-2-用RNA-Seq验证预测的cryptic-splice-mutation-Verification-of-Predicted-Cryptic-Splice-Mutations-in-RNA-Seq-Data" class="headerlink" title="3.2 用RNA-Seq验证预测的cryptic splice mutation (Verification of Predicted Cryptic Splice Mutations in RNA-Seq Data)"></a>3.2 用RNA-Seq验证预测的cryptic splice mutation (Verification of Predicted Cryptic Splice Mutations in RNA-Seq Data)</h4><ul>
<li>在上一部分不同层面证明了模型的性能之后，接下来这部分就用来探讨SpliceAI预测cryptic splice mutation的能力；<br><img src="/images/20190320-Fig7.jpg" alt="Figure 2"><br><img src="/images/20190320-Fig8.png" alt="Figure 2-legend"></li>
<li><strong>Figure 2A:</strong> 对于一个突变，预测其reference transcript和alternative transcript分值的差异；</li>
<li><strong>Figure 2B:</strong> 同样是使用GTEx群体，首先关注rare，private mutations（只在一个个体中出现），发现罕见、私有变异显著富集在私有新的splice junctions中，表明预测的大部分都是有功能的。这里主要是通过RNA-Seq数据验证预测出来的突变的确是有功能的；</li>
<li><strong>Figure 2C:</strong> 量化创造了新剪切位点的变异，以PYGB基因中的一个变异为例（creates a novel donor site）,计算novel juction相对使用率；</li>
<li><strong>Figure 2D:</strong> 预测分值被RNA-Seq数据验证的比例，可以看出分值变化越大，验证的比例越高；</li>
<li><strong>Figure 2E:</strong> 使用SNP的effect size来验证，关于effect size的计算在方法中有详述；</li>
<li><strong>Figure 2F:</strong> 不同阈值下，SpliceAI的sensitivity；</li>
<li><strong>Figure 2G:</strong> 以sensitivity为指标，与其他软件比较。</li>
</ul>
<h4 id="3-3-组织特异性选择性剪接经常来自于weak-cryptic-splice-variants-Tissue-specific-alternative-splicing-frequently-arises-from-weak-cryptic-splice-variants"><a href="#3-3-组织特异性选择性剪接经常来自于weak-cryptic-splice-variants-Tissue-specific-alternative-splicing-frequently-arises-from-weak-cryptic-splice-variants" class="headerlink" title="3.3 组织特异性选择性剪接经常来自于weak cryptic splice variants (Tissue-specific alternative splicing frequently arises from weak cryptic splice variants)"></a>3.3 组织特异性选择性剪接经常来自于weak cryptic splice variants (Tissue-specific alternative splicing frequently arises from weak cryptic splice variants)</h4><ul>
<li>第一部分验证可变剪切模型的精准性，第二部分用RNA-Seq数据验证预测出来的变异的确是有功能的；<strong>！！！</strong>在这里作者就想利用模型预测出来的结果解释一下生物学现象，比如：可变剪切的组织特异性；<br><img src="/images/20190320-Fig9.jpg" alt="Figure 3"><br><img src="/images/20190320-Fig10.png" alt="Figure 3-legend"></li>
<li><strong>Figure 3A:</strong> 可变剪切是基因调控的一个主要模式，其增加了不同组织以及不同发育阶段转录本的多样性，基因表达的错误调节（dysregulation）与疾病过程相关。但是作者发现novel splice junctions （由cryptic splice mutations导致的）的相对使用在不同组织中变化很大。作者在这里展示了一个例子，位于基因CDC25B外显子区域内的一个杂合变异，导致了一个新的donor site。这个变异在GTEx群体中只在一个个体中出现，表现出了组织特异性（muscle和fibroblasts；纤维母细胞）的可变剪切。</li>
<li><strong>Figure 3B:</strong> 而且在不同个体中这种组织特异性是reproducible; 这里举了一个<strong>Novel acceptor site</strong>, 位于FAM229B基因的外显子区域，在三个个体中表现出了一致的组织特异性，表明组织特异性的确与这些疾病相关，而不是由于随机效应导致的；</li>
<li><strong>Figure 3C:</strong> 其次作者发现预测出来的weak and intermmediate (△score 0.35-0,8) cryptic splice variants在正常转录本和异常转录本中有明显的差异(Fisher精准检验), 也就是说预测出来的weak cryptic splice variants更容易导致可变剪切的组织特异性；</li>
</ul>
<h4 id="3-4-预测的cryptic-splice-variants在人类群体中是非常有害的-Predicted-cryptic-splice-variants-are-strongly-deleterious-in-human-population"><a href="#3-4-预测的cryptic-splice-variants在人类群体中是非常有害的-Predicted-cryptic-splice-variants-are-strongly-deleterious-in-human-population" class="headerlink" title="3.4 预测的cryptic splice variants在人类群体中是非常有害的 (Predicted cryptic splice variants are strongly deleterious in human population)"></a>3.4 预测的cryptic splice variants在人类群体中是非常有害的 (Predicted cryptic splice variants are strongly deleterious in human population)</h4><ul>
<li>尽管预测出来的cryptic splice variant在RNA-Seq数据上有很高的验证率，但在大多数情况下，他们不是完全penetrant (<strong>中文翻译为：渗透的，我的理解是导致了acceptor/donor site loss/gain，但并不完全，仍保留原有的转录本，只是量发生了改变</strong>)，因此也就有可能一部分变异可能没有显著的功能。因此这部分内容作者为了探究预测的cryptic splice variants的自然选择特征，作者对来自Exome Aggregation Consortium（ExAC）数据库的60,706个人类外显子组中存在的每个变异进行评分，并鉴定出会改变exon-intron边界的变异；</li>
<li><p>为了测量作用在predicted splice-altering variants上负选择（进化过程中淘汰有害的等位基因）的程度，作者计算了common allele frequencies中splice-altering变异数目，与ExAC中单个变异（在ExAC群体中只有一个）的数目比较。由于最近人口规模呈指数级增长，所以singleton经历的负选择的程度很低。<strong>因此，相对于单体变异，在common allele频谱中预测出来的cryptic variants的减少为functional splice-altering variants提供了一个证据</strong>。为了避免对蛋白质编码序列造成混淆影响，我们将分析局限于位于必需GT或AG二核苷酸之外的同义突变和内含子突变，排除了被预测具有splice-altering effects的错义突变；</p>
<p>  <img src="/images/20190320-Fig11.jpg" alt="Figure 4"><br>  <img src="/images/20190320-Fig13.png" alt="Figure 4-legend"></p>
</li>
<li><strong>Figure 4A:</strong> 在common allele和singleton中SNV的△score大于0.8和小于0.1比较；common allele显著缺乏splice-altering variants。预测分值很高的cryptic variants经历了很强的负选择；</li>
<li><strong>Figure 4B:</strong> 在△score=0.8这个阈值下，大多数变异在RNA-Seq数据中倾向于fully penetrant （<strong>Figure 2D</strong>），预测为synonymous和intronic sryptic splice的突变在common allele frequencies中有78%的降低，与frameshift、stop-gain和essential GT or AG splice-distrupting中的82%旗鼓相当。图中纵坐标是有害变异的比例；</li>
<li><strong>Figure 4C:</strong> 当考虑影响frameshifts的cryptic splice variants时，发现他们经历更强的负选择；在ExAC数据集中，synonymous和intronic cryptic splice gain variants中有害变异的比例，然后基于这个变异是不是导致frameshift来分割；也就是说：<strong>先对synonymous和intronic cryptic splice gain variants中有害变异统计，然后分割成两部分</strong>；</li>
<li><strong>Figure 4D:</strong> 这部分作者比较<strong>predicted deep intronic (&gt;50 nt from known exon-intron boundaries) cryptic splice variants in the gnomAD dataset</strong>中有害变异的数目；</li>
<li><strong>Figure 4E:</strong> 每个个体中罕见变异、protein-truncating变异以及cryptic splice变异的平均数目。</li>
</ul>
<h4 id="3-5-De-novo-cryptic-splice-mutations是罕见遗传疾病的主要原因-De-Novo-Cryptic-Splice-Mutations-are-a-major-cause-of-rare-genetic-disorders"><a href="#3-5-De-novo-cryptic-splice-mutations是罕见遗传疾病的主要原因-De-Novo-Cryptic-Splice-Mutations-are-a-major-cause-of-rare-genetic-disorders" class="headerlink" title="3.5 De novo cryptic splice mutations是罕见遗传疾病的主要原因 (De Novo Cryptic Splice Mutations are a major cause of rare genetic disorders)"></a>3.5 De novo cryptic splice mutations是罕见遗传疾病的主要原因 (De Novo Cryptic Splice Mutations are a major cause of rare genetic disorders)</h4><ul>
<li>自闭症谱系障碍和严重智力残疾患者的大规模测序研究证明了破坏神经发育途径中基因的从头蛋白编码突变（错义，无义，移码和必需剪接二核苷酸）的核心作用。为了评估通过改变剪接起作用的非编码突变的临床影响，我们应用神经网络预测来自解密发育障碍（DDD）的4,293名智力残疾人的从头突变的影响、来自Simons Simplex Collection和自闭症序列联盟的3,953名患有自闭症谱系障碍（ASD）的个体，以及来自Simons Simplex系列。</li>
<li><strong>Figure 5A:</strong> 与健康人群比较，intellectual disability和自闭症群体中分别有1.51和1.3倍的splice-altering变异的富集；</li>
<li><strong>Figure 5B:</strong> cryptic splice mutations占了自闭症（autism spectrum disorder; ASD）中致病变异的11%， DDD (Deciphering Developmental Disorders)的9%；</li>
<li><strong>Figure 5C:</strong> 大多数de novo predicted cryptic splice mutations的△score小于0.5，因此可能会产生mixture transcripts；</li>
<li><strong>Figure 5D:</strong> 新的致病基因；<br>  <img src="/images/20190320-Fig12.jpg" alt="Figure 5"><br>  <img src="/images/20190320-Fig14.png" alt="Figure 5-legend"><br>  <img src="/images/20190320-Fig15.png" alt="Figure 5-legend"></li>
</ul>
<h4 id="3-6-实验验证-Experimental-validation-of-de-novo-cryptic-splice-mutations-in-Autism-Patients"><a href="#3-6-实验验证-Experimental-validation-of-de-novo-cryptic-splice-mutations-in-Autism-Patients" class="headerlink" title="3.6 实验验证 (Experimental validation of de novo cryptic splice mutations in Autism Patients)"></a>3.6 实验验证 (Experimental validation of de novo cryptic splice mutations in Autism Patients)</h4><ul>
<li>这部分作者对部分结果进行了实验验证；</li>
<li><strong>Figure 5E:</strong> 预测出影响自闭症的cryptic splice mutations（RNA-Seq验证）的三个例子；</li>
<li><strong>Figure 5F:</strong> 通过RNA-Seq选出来的36个变异的验证状态。</li>
</ul>
<h3 id="4-方法细节"><a href="#4-方法细节" class="headerlink" title="4. 方法细节"></a>4. 方法细节</h3><h4 id="4-1-SpliceAI模型结构"><a href="#4-1-SpliceAI模型结构" class="headerlink" title="4.1 SpliceAI模型结构"></a>4.1 SpliceAI模型结构</h4><ul>
<li>文章主要使用ResNet结构，one-hot编码，具体的结构在<strong>3.1</strong>部分已经记录，此处不再赘述。</li>
</ul>
<h4 id="4-2-模型训练与测试"><a href="#4-2-模型训练与测试" class="headerlink" title="4.2 模型训练与测试"></a>4.2 模型训练与测试</h4><ul>
<li>共20287个基因，染色体2，4，6，8，10-22，X，Y用于模型训练（共13，384个基因，130796对donor-acceptor），训练集中随机抽10%作为Validation（用于调参和模型的earlystopping）。然后剩下的基因<strong>与训练集没有paralog的作为测试集</strong>。</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/">https://jingjingzhai.github.io/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/深度学习/">深度学习</a><a href="/tags/可变剪切/">可变剪切</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/" class="pre">Deep-learning augmented RNA-seq analysis of transcript splicing</a><a href="/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/" class="next">DeepTACT: Predicting 3D chromatin contacts via bootstrapping deep learning</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-使用深度学习从原始序列中精确的预测剪切-Accurate-prediction-of-splicing-from-a-primary-sequence-using-deep-learning"><span class="toc-text">3.1 使用深度学习从原始序列中精确的预测剪切 (Accurate prediction of splicing from a primary sequence using deep learning)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-用RNA-Seq验证预测的cryptic-splice-mutation-Verification-of-Predicted-Cryptic-Splice-Mutations-in-RNA-Seq-Data"><span class="toc-text">3.2 用RNA-Seq验证预测的cryptic splice mutation (Verification of Predicted Cryptic Splice Mutations in RNA-Seq Data)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-组织特异性选择性剪接经常来自于weak-cryptic-splice-variants-Tissue-specific-alternative-splicing-frequently-arises-from-weak-cryptic-splice-variants"><span class="toc-text">3.3 组织特异性选择性剪接经常来自于weak cryptic splice variants (Tissue-specific alternative splicing frequently arises from weak cryptic splice variants)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-预测的cryptic-splice-variants在人类群体中是非常有害的-Predicted-cryptic-splice-variants-are-strongly-deleterious-in-human-population"><span class="toc-text">3.4 预测的cryptic splice variants在人类群体中是非常有害的 (Predicted cryptic splice variants are strongly deleterious in human population)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-5-De-novo-cryptic-splice-mutations是罕见遗传疾病的主要原因-De-Novo-Cryptic-Splice-Mutations-are-a-major-cause-of-rare-genetic-disorders"><span class="toc-text">3.5 De novo cryptic splice mutations是罕见遗传疾病的主要原因 (De Novo Cryptic Splice Mutations are a major cause of rare genetic disorders)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-6-实验验证-Experimental-validation-of-de-novo-cryptic-splice-mutations-in-Autism-Patients"><span class="toc-text">3.6 实验验证 (Experimental validation of de novo cryptic splice mutations in Autism Patients)</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#4-方法细节"><span class="toc-text">4. 方法细节</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#4-1-SpliceAI模型结构"><span class="toc-text">4.1 SpliceAI模型结构</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-2-模型训练与测试"><span class="toc-text">4.2 模型训练与测试</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/07/16/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">Single-cell RNA-seq denoising using a deep count autoencoder</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/12/Opportunities-and-challenges-for-transcriptome-wide-association-studies/">Opportunities and challenges for transcriptome- wide association studies</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/09/Virtual-ChIP-seq-predicting-transcription-factor-binding-by-learning-from-the-transcriptome/">Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>