<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">学而不思则惘</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h2 class="post-title"><a href="/2019/04/03/Dysregulation-of-expression-correlates-with-rare-allele-burden-and-fitness-loss-in-maize/">Dysregulation of expression correlates with rare-allele burden and fitness  loss in maize</a></h2><div class="post-meta"><p><span class="date">2019-04-03</span><span><a href="/categories/Nature/" class="category">Nature</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2018年发表于Nature上的关于玉米基因表达的错误调节与罕见变异关系的文章。通讯作者是来自康奈尔大学<a href="https://www.panzea.org/pis" target="_blank" rel="noopener">Edward S. Buckler</a>。<br></p></div><p class="readmore"><a href="/2019/04/03/Dysregulation-of-expression-correlates-with-rare-allele-burden-and-fitness-loss-in-maize/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/04/03/DeepCAPE-a-deep-convolutional-neural-network-for-the-accurate-prediction-of-enhancers/">DeepCAPE: a deep convolutional neural network for the accurate prediction of enhancers</a></h2><div class="post-meta"><p><span class="date">2019-04-03</span><span><a href="/categories/bioRxiv/" class="category">bioRxiv</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是发布在biRxiv上的用深度学习预测增强子的文章。通讯作者为清华大学自动化系的<a href="http://bioinfo.au.tsinghua.edu.cn/member/ruijiang/chinese/index.html" target="_blank" rel="noopener">江瑞老师实验室</a>。这篇文章在网络结构上使用了网络merge的策略，同时还使用自编码器对数据降维；</p></div><p class="readmore"><a href="/2019/04/03/DeepCAPE-a-deep-convolutional-neural-network-for-the-accurate-prediction-of-enhancers/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/">Comprehensive functional genomic resource and integrative model for the human brain</a></h2><div class="post-meta"><p><span class="date">2019-04-01</span><span><a href="/categories/Science/" class="category">Science</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p><a href="http://science.sciencemag.org/content/362/6420/eaat8464" target="_blank" rel="noopener">文章链接</a></p></div><p class="readmore"><a href="/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/04/01/A-universal-sNP-and-small-indel-variant-caller-using-deep-neural-networks/">A universal SNP and small-indel variant caller using deep neural networks</a></h2><div class="post-meta"><p><span class="date">2019-04-01</span><span><a href="/categories/Nature-Biotechnology/" class="category">Nature Biotechnology</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p><a href="https://www.nature.com/articles/nbt.4235" target="_blank" rel="noopener">文章链接</a></p></div><p class="readmore"><a href="/2019/04/01/A-universal-sNP-and-small-indel-variant-caller-using-deep-neural-networks/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/03/31/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</a></h2><div class="post-meta"><p><span class="date">2019-03-31</span><span><a href="/categories/Cell/" class="category">Cell</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2018年12月发表在Cell期刊上关于突变对剪切效应影响的文章。通讯作者是来自于西班牙巴塞罗那科学技术学院的<a href="https://www.crg.eu/ben_lehner" target="_blank" rel="noopener">Ben Lehner</a>。</p></div><p class="readmore"><a href="/2019/03/31/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/">Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation</a></h2><div class="post-meta"><p><span class="date">2019-03-29</span><span><a href="/categories/The-Plant-Cell/" class="category">The Plant Cell</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2017年发表于The Plant Cell期刊上的用共线性网络分析MADS-box基因家族的文章，通讯作者是荷兰瓦赫宁根大学<a href="https://www.wur.nl/en/Persons/Eric-prof.dr.-ME-Eric-Schranz.htm?subpage=projects" target="_blank" rel="noopener">M. Eric Schranz</a>。同时作者还在<strong>Current Opinion in Plant Biology</strong>发表一篇关于共线性网络的<a href="https://www.sciencedirect.com/science/article/pii/S1369526616302230" target="_blank" rel="noopener">综述</a>。</p></div><p class="readmore"><a href="/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/">Deep-learning augmented RNA-seq analysis of transcript splicing</a></h2><div class="post-meta"><p><span class="date">2019-03-27</span><span><a href="/categories/Nature-Methods/" class="category">Nature Methods</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是发表在Nature Methods上的使用深度学习来扩充可变剪切的分析。通讯作者是来自于UCLA的<a href="http://xinglab.github.io" target="_blank" rel="noopener">邢毅</a>老师。</p></div><p class="readmore"><a href="/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/">Predicting splicing from primary sequence with deep learning</a></h2><div class="post-meta"><p><span class="date">2019-03-20</span><span><a href="/categories/Cell/" class="category">Cell</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2019年1月发表在Cell期刊用深度学习预测可变剪切。通讯作者来自于<a href="https://www.illumina.com/" target="_blank" rel="noopener">Illumina人工智能实验室</a>。文章使用了火热的ResNet预测可变剪切，模型足够复杂（32个卷积）而且预测结果看起来是真的好，top-k accuracy高达95%，甩其他软件几条街，结果的分析更是详实复杂。</p></div><p class="readmore"><a href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/">DeepTACT: Predicting 3D chromatin contacts via bootstrapping deep learning</a></h2><div class="post-meta"><p><span class="date">2019-03-18</span><span><a href="/categories/NAR/" class="category">NAR</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2019年3月18日发表于NAR期刊上使用深度学习预测3D基因组相互作用的文章。通讯作者为清华大学自动化系的<a href="http://bioinfo.au.tsinghua.edu.cn/member/ruijiang/chinese/index.html" target="_blank" rel="noopener">江瑞老师实验室</a>。这篇文章在网络结构上使用了网络merge的策略，整合序列以及DNase特征，merge之后还使用了BLSTM进一步提取特征，同时还使用了目前火热的attention layer。 在数据上使用集成学习(Bootstrap)以及Data augmentation保证深度学习的数据量。<br></p></div><p class="readmore"><a href="/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/">阅读更多</a></p></div><div class="post"><h2 class="post-title"><a href="/2019/03/13/Network-propagation-a-universal-amplifier-of-genetic-associations/">Network propagation: a universal amplifier of genetic associations</a></h2><div class="post-meta"><p><span class="date">2019-03-13</span><span><a href="/categories/Nature-Reviews-Genetics/" class="category">Nature Reviews Genetics</a></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是来自于以色列特拉维夫大学(Tel Aviv University) <a href="http://www.cs.tau.ac.il/~roded/index.html" target="_blank" rel="noopener">Roded Sharan课题组</a>在Nature reviews genetics发表的关于network propagation (网络扩散)在生物学领域另用的综述。</p></div><p class="readmore"><a href="/2019/03/13/Network-propagation-a-universal-amplifier-of-genetic-associations/">阅读更多</a></p></div><nav class="page-navigator"><a class="extend prev" rel="prev" href="/">上一页</a><a class="page-number" href="/">1</a><span class="page-number current">2</span><a class="page-number" href="/page/3/">3</a><a class="extend next" rel="next" href="/page/3/">下一页</a></nav></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/07/16/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/De-novo-domestication-an-alternative-route-toward-new-crops-for-the-future/">De novo domestication an alternative route toward new crops for the future</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/18/Phenotypic-and-genome-wide-association-with-the-local-environment-of-Arabidopsis/">Phenotypic and genome-wide association with the local environment of Arabidopsis</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/17/Evolutionary-Origins-of-Pseudogenes-and-Their-Association-with-Regulatory-Sequences-in-Plants/">Evolutionary Origins of Pseudogenes and Their Association with Regulatory Sequences in Plants</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">Single-cell RNA-seq denoising using a deep count autoencoder</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/12/Opportunities-and-challenges-for-transcriptome-wide-association-studies/">Opportunities and challenges for transcriptome- wide association studies</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/09/Virtual-ChIP-seq-predicting-transcription-factor-binding-by-learning-from-the-transcriptome/">Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/totop.js?v=2.0.3" async></script></body></html>