<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</h1><div class="post-meta"><a href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/#comments" class="comment-count"></a><p><span class="date">Aug 02, 2019</span><span><a href="/categories/Genome-Biology/" class="category">Genome Biology</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是来自于梅奥医学中心(Mayo clinic)的John D. Fryer课题组在Genome Biology上发表的的关于人类基因组中dark gene regions （比对上的reads很少）和camouflaged regions（多比对的区域）的系统分析与评估。<br><a id="more"></a></p>
<p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1707-2" target="_blank" rel="noopener">文章链接</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li><strong>背景:</strong> <ul>
<li>dark gene regions: 比对上的reads很少；</li>
<li>camouflaged regions: 模棱两可的比对，即多比对；</li>
</ul>
</li>
<li><strong>结果:</strong><ul>
<li>基于全基因组illumina测序数据，确定了36,794个dark regions in 6054个基因中。在这些中，8.7%是完全dark，35.2%有超过5%的dark；</li>
<li>Linked-read or long-read测序技术可以降低dark regions；</li>
<li>作者提出了一个算法解析camouflaged regions并将其应用于老年痴呆疾病，发现在CR1基因（老年痴呆的显著基因）有一个罕见的10个核苷酸的移码缺失。</li>
</ul>
</li>
<li><strong>结论：</strong><ul>
<li>虽然本文不能评估CR1移码突变与老年痴呆症的关系（样本量不够），但是本文的研究表明在基因组中有很多被忽视的区域可以通过long-read技术被解析。</li>
</ul>
</li>
</ul>
<h3 id="1-背景"><a href="#1-背景" class="headerlink" title="1. 背景"></a>1. 背景</h3><ul>
<li>研究人员很早就注意到人类基因组中包含“dark regions”，这些区域通过标准的二代高通量测序不能组装，因此这些区域如果有与人类健康或者疾病相关的变异的话就会很难确定。有一些“dark regions”文章称作”dark by depth”，也就是说比对上的reads很少或者没有比对上；还有一些称作“dark by mapping quality”，也就是多比对的reads。第一种情况可能是因为这些区域在化学水平上很难测到（例如：GC含量高）；而第二种情况不是测序的问题，而是下游生物信息学分析的问题。而且很多dark regions来源于基因组中的重复区域，作者把这部分区域叫做“camouflaged”。这些camouflaged regions要么是连续的串联重复或者更大片段的DNA重复（例如：基因重复）。事实上，人类基因组中很多基因对着进化的推移都发生了重复而且是具有转录活性的（例如：热激蛋白），然而也有一些不具有转录活性（例如：假基因）</li>
<li>对于多比对的reads，标准的二代测序比对工具（如：BWA）将read随机比对到基因组中一个位置，然后指定一个很低的比对分值，对于BWA来说，不能被唯一比对的reads的MAPQ是0；但是对于某些双端测序的情形，BWA将会给一个高一点的分值如果另外一端比对到附近的话。</li>
<li>最近关于camouflaged regions的研究证明了这个问题会影响所有的标准的RNA-Seq分析。Robert and Watson用12个不同的RNA-Seq处理方法证明了958个基因的表达水平被低估或者高估了，而且还证明了很多基因影响疾病。</li>
<li>在这篇文章中，作者系统分析了dark and camouflaged genes,强调了很多疾病相关的基因直接影响神经性疾病，例如：阿尔兹海默症，自闭症等。同时作者表明linked-read以及long-read测序技术可以大量的降低dark和camouflaged regions，而且作者提出一个解决camouglaged regions的方法，针对于标准的short-read测序数据也适用。然后作者将其应用于阿尔兹海默症的数据并且在CR1基因（阿尔兹海默症的top gene）的C3b和C4b binding domain上确定了一个罕见的十个核苷酸的移码缺失，这种缺失在5个cases中都有但是在controls中却没有。</li>
</ul>
<h3 id="2-结果"><a href="#2-结果" class="headerlink" title="2. 结果"></a>2. 结果</h3><h4 id="2-1-dark-and-camouflaged-regions的确定"><a href="#2-1-dark-and-camouflaged-regions的确定" class="headerlink" title="2.1 dark and camouflaged regions的确定"></a>2.1 dark and camouflaged regions的确定</h4><ul>
<li>数据：来自于ADSP(Alzheimer’s Disease Sequencing Project)的10个个体的全基因组测序数据，对每个样本扫描得到dark and camouflaged regions然后在是个样本中取平均得到最终的结果。因为这个研究只用了男性，所以可以评估Y染色体上的dark and camouflaged regions, 因为Y染色体大部分区域都是dark。</li>
<li>定义dark的两个原因：<ul>
<li>i: insufficient number of reads aligned to the genomic region (dark by depth)</li>
<li>ii: reads aligned to the re- gion, but with insufficient mapping quality for a variant caller to identify mutations in the region (dark by map- ping quality)</li>
</ul>
</li>
<li>具体定义指标如下图所示：<br><img src="/images/20190802-Fig1.png" alt="Figure 1"></li>
<li>使用MAPQ小于10是因为GATK用的就是这个标准。</li>
</ul>
<h4 id="2-2-标准的short-read测序在6054个基因中找到了36794个dark-regions-Standard-short-read-sequencing-leaves-36-794-dark-regions-across-6054-gene-bodies-including-protein-coding-exons-from-748-genes"><a href="#2-2-标准的short-read测序在6054个基因中找到了36794个dark-regions-Standard-short-read-sequencing-leaves-36-794-dark-regions-across-6054-gene-bodies-including-protein-coding-exons-from-748-genes" class="headerlink" title="2.2 标准的short-read测序在6054个基因中找到了36794个dark regions (Standard short-read sequencing leaves 36,794 dark regions across 6054 gene bodies, including protein- coding exons from 748 genes)"></a>2.2 标准的short-read测序在6054个基因中找到了36794个dark regions (Standard short-read sequencing leaves 36,794 dark regions across 6054 gene bodies, including protein- coding exons from 748 genes)</h4><ul>
<li><p>作者利用上述数据确定了6054个基因，3802个是protein<br>coding，1232个假基因，753个lincRNA (lone intergenic non-coding RNAs),具体如下图所示：<br><img src="/images/20190802-Fig2.png" alt="Figure 2"></p>
</li>
<li><p>作者接下来Focus到CDS regions，在748个protein-coding genes中确定了2855个dark CDS regions，其中117是完全dark (如下图所示)</p>
</li>
</ul>
<p><img src="/images/20190802-Fig3.png" alt="Figure 3"><br><img src="/images/20190802-Fig4.png" alt="Figure 3-legend"></p>
<h4 id="2-3-Dark-and-camouflaged-regions影响重要的通路以及基因家族-Important-pathways-and-gene-families-are-affected-by-dark-and-camouflaged-regions"><a href="#2-3-Dark-and-camouflaged-regions影响重要的通路以及基因家族-Important-pathways-and-gene-families-are-affected-by-dark-and-camouflaged-regions" class="headerlink" title="2.3 Dark and camouflaged regions影响重要的通路以及基因家族(Important pathways and gene families are affected by dark and camouflaged regions)"></a>2.3 Dark and camouflaged regions影响重要的通路以及基因家族(Important pathways and gene families are affected by dark and camouflaged regions)</h4><ul>
<li>这部分主要是对找到的基因做GO以及pathway enrichment富集分析。<br><img src="/images/20190802-Fig5.png" alt="Figure 4"></li>
</ul>
<h4 id="2-4-有76个基因与326个人类疾病的变异有关-There-are-76-dark-genes-with-known-mutations-associated-with-326-human-diseases"><a href="#2-4-有76个基因与326个人类疾病的变异有关-There-are-76-dark-genes-with-known-mutations-associated-with-326-human-diseases" class="headerlink" title="2.4 有76个基因与326个人类疾病的变异有关(There are 76 dark genes with known mutations associated with 326 human diseases)"></a>2.4 有76个基因与326个人类疾病的变异有关(There are 76 dark genes with known mutations associated with 326 human diseases)</h4><p><img src="/images/20190802-Fig6.png" alt="Figure 5"></p>
<h4 id="2-5-Camouflaged-genes-are-consistently-dark-in-gnomAD-but-dark-by-depth-genes-may-be-sample-or-dataset-specific"><a href="#2-5-Camouflaged-genes-are-consistently-dark-in-gnomAD-but-dark-by-depth-genes-may-be-sample-or-dataset-specific" class="headerlink" title="2.5 Camouflaged genes are consistently dark in gnomAD, but dark-by-depth genes may be sample or dataset specific"></a>2.5 Camouflaged genes are consistently dark in gnomAD, but dark-by-depth genes may be sample or dataset specific</h4><p><img src="/images/20190802-Fig7.png" alt="Figure 6"><br><img src="/images/20190802-Fig8.png" alt="Figure 6-legend"></p>
<h4 id="2-6-Many-camouflaged-regions-can-be-rescued-including-in-standard-short-read-sequencing-data"><a href="#2-6-Many-camouflaged-regions-can-be-rescued-including-in-standard-short-read-sequencing-data" class="headerlink" title="2.6 Many camouflaged regions can be rescued, including in standard short-read sequencing data"></a>2.6 Many camouflaged regions can be rescued, including in standard short-read sequencing data</h4><ul>
<li>作者提出了一种解决问题的方法：<blockquote>
<p><strong>(1)</strong> extract reads from camouflaged regions;<br><strong>(2)</strong> mask all highly similar regions in the reference genome, except one, and re-align the ex- tracted reads;<br><strong>(3)</strong> call mutations using standard methods, while accounting for increased ploidy and po- tential reference-based artifacts.</p>
</blockquote>
</li>
</ul>
<h3 id="3-结论"><a href="#3-结论" class="headerlink" title="3. 结论"></a>3. 结论</h3><ul>
<li>目前仍有数以千计的潜在重要基因组区域被短读序列所忽视，但很大程度上是通过linked-read或long-read技术解决的。虽然这些区域仅代表整个基因组或外显子组的一小部分，但已知许多这些区域在人类健康和疾病中是重要的。然而，同样重要的是，许多其他基因的影响完全未知，因为它们是100％黑暗的。我们提出了一种方法，可以解决大多数伪装区域，我们相信这些区域将帮助研究人员识别与疾病有关的突变。作为原理证明，我们在ADSP数据集中拯救了大约4214个变体，包括CR1中的10个核苷酸的移码突变。虽然我们无法正式评估阿尔茨海默病（样本量不足）中的CR1移码突变，但我们认为值得在更大的队列中进行调查。从长远来看，我们相信连锁和长读序列技术将是解决黑暗和伪装区域的最佳解决方案。</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">https://jingjingzhai.github.io/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/Dark-genes/">Dark genes</a><a href="/tags/Camouflaged-genes/">Camouflaged genes</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/" class="pre">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</a><a href="/2019/07/17/A-Deep-Neural-Network-for-Predictingand-Engineering-Alternative-Polyadenylation/" class="next">A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-背景"><span class="toc-text">1. 背景</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-结果"><span class="toc-text">2. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#2-1-dark-and-camouflaged-regions的确定"><span class="toc-text">2.1 dark and camouflaged regions的确定</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#2-2-标准的short-read测序在6054个基因中找到了36794个dark-regions-Standard-short-read-sequencing-leaves-36-794-dark-regions-across-6054-gene-bodies-including-protein-coding-exons-from-748-genes"><span class="toc-text">2.2 标准的short-read测序在6054个基因中找到了36794个dark regions (Standard short-read sequencing leaves 36,794 dark regions across 6054 gene bodies, including protein- coding exons from 748 genes)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#2-3-Dark-and-camouflaged-regions影响重要的通路以及基因家族-Important-pathways-and-gene-families-are-affected-by-dark-and-camouflaged-regions"><span class="toc-text">2.3 Dark and camouflaged regions影响重要的通路以及基因家族(Important pathways and gene families are affected by dark and camouflaged regions)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#2-4-有76个基因与326个人类疾病的变异有关-There-are-76-dark-genes-with-known-mutations-associated-with-326-human-diseases"><span class="toc-text">2.4 有76个基因与326个人类疾病的变异有关(There are 76 dark genes with known mutations associated with 326 human diseases)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#2-5-Camouflaged-genes-are-consistently-dark-in-gnomAD-but-dark-by-depth-genes-may-be-sample-or-dataset-specific"><span class="toc-text">2.5 Camouflaged genes are consistently dark in gnomAD, but dark-by-depth genes may be sample or dataset specific</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#2-6-Many-camouflaged-regions-can-be-rescued-including-in-standard-short-read-sequencing-data"><span class="toc-text">2.6 Many camouflaged regions can be rescued, including in standard short-read sequencing data</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结论"><span class="toc-text">3. 结论</span></a></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2020/02/13/Building-Transcription-Factor-Binding-Site-Models-to-Understand-Gene-Regulation-in-Plants/">Building transcription factor binding site models to understand gene regulation in plants</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/09/16/Deep-learning-sequence-based-ab-initio-prediction-of-variant-effects-on-expression-and-disease-risk/">Deep learning sequence-based ab initio prediction of variant effects on expression and disease risk </a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/17/Identification-of-the-expressome-by-machine-learning-on-omics-data/">Identification of the expressome by machine learning on omics data</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/12/Partitioning-heritability-by-functional-annotation-using-genome-wide-association-summary-statistics/">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/11/Integrating-Coexpression-Networks-with-GWAS-to-Prioritize-Causal-Genes-in-Maize/">Integrating Coexpression Networks with GWAS to Prioritize Causal Genes in Maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/08/02/Systematic-analysis-of-dark-and-camouflaged-genes-reveals-disease-relevant-genes-hiding-in-plain-sight/">Systematic analysis of dark and camouflaged genes reveals disease- relevant genes hiding in plain sight</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/07/17/A-Deep-Neural-Network-for-Predictingand-Engineering-Alternative-Polyadenylation/">A Deep Neural Network for Predicting and Engineering Alternative Polyadenylation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/05/23/Decomposing-Cell-Identity-for-Transfer-Learning-across-Cellular-Measurements-Platforms-Tissues-and-Species/">Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/26/Common-SNPs-explain-a-large-proportion-of-the-heritability-for-human-height/">Common SNPs explain a large proportion of the heritability for human height</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/24/Functional-architecture-of-low-frequency-variants-highlights-strength-of-negative-selection-across-coding-and-non-coding-annotations/">Functional architecture of low-frequency variants highlights strength of negative selection across coding and non-coding annotations</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">4</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Cell-Systems/">Cell Systems</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Molecular-Plant/">Molecular Plant</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">5</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/APA预测/" style="font-size: 15px;">APA预测</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/数量遗传/" style="font-size: 15px;">数量遗传</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/迁移学习/" style="font-size: 15px;">迁移学习</a> <a href="/tags/驯化/" style="font-size: 15px;">驯化</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/机器学习/" style="font-size: 15px;">机器学习</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/GWAS/" style="font-size: 15px;">GWAS</a> <a href="/tags/Arabidopsis/" style="font-size: 15px;">Arabidopsis</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/Dark-genes/" style="font-size: 15px;">Dark genes</a> <a href="/tags/Camouflaged-genes/" style="font-size: 15px;">Camouflaged genes</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2020/02/">二月 2020</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/09/">九月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/08/">八月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/07/">七月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/05/">五月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>