<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</h1><div class="post-meta"><a href="/2019/03/22/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/#comments" class="comment-count"></a><p><span class="date">Mar 22, 2019</span><span><a href="/categories/Cell/" class="category">Cell</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是2018年12月发表在Cell期刊上关于突变对剪切效应影响的文章。通讯作者是来自于西班牙巴塞罗那科学技术学院的<a href="https://www.crg.eu/ben_lehner" target="_blank" rel="noopener">Ben Lehner</a>。 <a id="more"></a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867418316246" target="_blank" rel="noopener">文章链接</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>尽管分子生物学的知识很丰富，但准确预测生物学现象的定量法则仍然很少见。可变剪切是在人类疾病中干扰基因表达的重要步骤；</li>
<li>为了理解在进化过程中变异的组合效应，作者量化了人类中一个可变剪接的外显子出现时积累的所有组合突变的影响。结果表明突变效应与外显子包含水平是非单调地，每个突变在<strong>predictable intermediate inclusion level</strong>上具有最大效应；</li>
<li>这种scale在整个全基因组范围内的顺式和反式剪切干扰中（包括天然和疾病相关变体）都观察到了。数学模型表明可变剪切位点之间的竞争足够导致这种非线性的图谱；</li>
<li>把这种全局的scaling law与特定的相互作用（neighborint mutations）结合将会使得精确的预测复杂基因型（&gt;10 mutations）成为可能。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>生物学系统行为的定量预测依旧很罕见。举个例子，从基因型的改变预测表型的变化依旧是遗传学、进化、农业以及精准医疗中的核心挑战。然而，即使是目前非常好的研究，预测基因甚至SNP的影响都非常困难；</li>
<li>一个原因是：突变的结果依赖于遗传背景。考虑到基因组内的完全变异，以及仅考虑单个基因内的其他变异时，这是正确的。单个蛋白质和RNA的全面诱变揭示了基因内突变之间丰富的pairwise interactions；当组合两个以上的突变时，也会发生表型的变化，这些突变无法从成分配对组合的表型中预测出来。 突变之间的这些相互作用被称为遗传相互作用或上位性，成对（二级）和高级（第三，第四等级）相互作用对于在系统评价的少数情况下对于准确的遗传预测都是重要的；</li>
<li><strong>深度诱变</strong>结合<strong>功能选择</strong>和<strong>深度测序</strong>也已用于量化突变对基因表达的影响，包括：基因启动子，转录增强子，5’和3’UTR以及调节剪接的内含子和外显子区域的诱变；</li>
<li>选择性剪接是人类疾病中干扰基因表达的一个关键调节步骤，研究表明其中10％的致病外显子突变改变了剪接，并且据估计：所有与疾病相关的等位基因中有1/3改变了剪接；</li>
<li>量化模型中所有可能的SNP变化对FAS基因外显子外显子6的影响，我们先前报道超过60％的单突变改变了外显子的包含。 此外，测试双突变体揭示了突变对之间频繁的非线性相互作用，使得难以预测突变组合时拼接的确切水平；</li>
<li>FAS基因的外显子6（编码FAS/CD95致死手提的跨膜结构与）在人类中是alternatively spliced，在不同的细胞类型和条件下都不一样。跳过这个外显子的mRNA编码缺乏跨膜结构域的分泌蛋白，该跨膜结构域充当诱饵受体。 因此，FAS的可变剪接将蛋白质从促凋亡分子转变为抗细胞凋亡分子；</li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-重新构建FAS外显子6的进化-Reconstructing-the-Evolution-of-FAS-exon-6"><a href="#3-1-重新构建FAS外显子6的进化-Reconstructing-the-Evolution-of-FAS-exon-6" class="headerlink" title="3.1 重新构建FAS外显子6的进化 (Reconstructing the Evolution of FAS exon 6)"></a>3.1 重新构建FAS外显子6的进化 (Reconstructing the Evolution of FAS exon 6)</h4><ul>
<li>与标题对应，这一部分主要就是想说明FAS基因Exon 6的保留在进化上是由几个碱基的替换导致的；</li>
<li><p>在人类中，FAS中exon 6的可变剪切使得FAS蛋白从膜结合的促凋亡同种型转换为可溶性抗凋亡同种型，这个外显子的可变剪切在不同组织中是不同的，这里用PSI (percentage spliced-in)值来评估（<strong>Figure 1A</strong>），在五个人类组织中从～70%（in lung）变化到～95% (in kidney)，接下来作者开始分析不同灵长类动物中五个组织的RNA-Seq数据来探究FAS exon 6可变剪切的进化情况，发现在<strong>大猩猩</strong>和<strong>Old World monkeys</strong>中，外显子保留在不同组织中变化很大，但是在<strong>New world monkeys</strong>和<strong>lemurs</strong>中不同组织中几乎都是一致的（<strong>Figure 1A</strong>），这表明，这个外显子的可变跳跃在灵长类物种中是发生了进化的。<br><img src="/images/20190322-Fig1.jpg" alt="Figure 1"><br><img src="/images/20190322-Fig2.png" alt="Figure 1-legend"></p>
</li>
<li><p>FAS exon 6的5’和3’内含子序列在不同灵长类物种中基本上不变的。但是与灵长类最后一个共同祖先相比，仍在11位置处有12nt的替换（<strong>Figure 1A，1B</strong>），<strong>Figure 1B</strong>展示的就是与祖先相比，灵长类物种中的替换情况，但是这些替换在物种进化树上只对应了6个节点（<strong>Figure S1B</strong>），因此剩下的组合（去除进化树节点上的变异）都是inferred evolutionary intermediates, 作者就量化了这些inferred evolutionary intermediates的PSI。然而在人类中，外显子保留的水平是60%，处在物种进化树的底端（<strong>Figure 1A</strong>），但是如图所示，<strong>node 2</strong>和<strong>node 3</strong>几乎是一致的，而对于更近一些的intermediates的exon inclusion水平也处于中间状态。</p>
</li>
<li><strong>!!!</strong>这里才理解作者在摘要中说<strong>This revealed that mutation effects scale non-monotonically with the inclusion level of an exon</strong>，在进化过程中外显子保留的水平是非单调的。<br><img src="/images/20190322-Fig3.jpg" alt="Figure S1"><br><img src="/images/20190323-Fig1.png" alt="Figure S1-legend"></li>
</ul>
<h4 id="3-2-组合完整的基因型空间定位-Combinatorially-complete-mapping-of-a-genotype-space"><a href="#3-2-组合完整的基因型空间定位-Combinatorially-complete-mapping-of-a-genotype-space" class="headerlink" title="3.2 组合完整的基因型空间定位 (Combinatorially complete mapping of a genotype space)"></a>3.2 组合完整的基因型空间定位 (Combinatorially complete mapping of a genotype space)</h4><ul>
<li>虽然在不同物种以及不同组织中外显子的保留水平变化很大，但是却只有12个突变，这就可以分析哪一个突变具有独立的效应。这个外显子的进化是通过数百万种可能的进化途径之一发生的，这种途径将原始祖先的基因型与当前的人类FAS外显子6基因型联系起来。<strong>在进化过程中这12个替换在每一步中效应是一样的吗？</strong>作者为了回答这个问题，作者构建了3,072个基因型（2^10*3,10个位置可以有2个变化，有一个可以有3个变化；<strong>Figure 1B</strong>）。然后作者就自己构建了一个小基因 (minigene cassette), 覆盖了FAS基因的5-7个外显子，然后把它转到KEK293细胞中(<strong>Figure 1C</strong>)，然后在最终成熟的mRNA中定量每个对应基因型的exon inclusion量，作者在正文中只分析了794个基因型；</li>
<li>从<strong>Figure 1C</strong>可以看出作者计算得失enrichment scores (Ess), 但是<strong>Figure S1F</strong>表明ESc与PSI有很强的线性相关性，而且ESs的9个生物学重负相关系数很高（0.57-0.74； <strong>Figure S1G</strong>），但是对于SD小于10的PSI相关系数更高（0.97，0.98；<strong>Figure S1G</strong>），因此在接下来的正文中作者focus到794个high confidence genotypes, 其他genotypes的分析再补充材料中，结论相似；</li>
<li><strong>Figure 1D:</strong> 不同基因型中PSI的分布，是一个双峰分布；</li>
<li><strong>Figure 1E:</strong> 离祖先不同远近的基因型的PSI分布。</li>
</ul>
<h4 id="3-3-突变对剪切有非独立的影响-Mutations-have-non-independent-effects-on-splicing"><a href="#3-3-突变对剪切有非独立的影响-Mutations-have-non-independent-effects-on-splicing" class="headerlink" title="3.3 突变对剪切有非独立的影响 (Mutations have non-independent effects on splicing)"></a>3.3 突变对剪切有非独立的影响 (Mutations have non-independent effects on splicing)</h4><ul>
<li>作者在这部分主要分析突变的效应是不是独立的 (independent), 所谓<strong>independent</strong>, 就是说，我们施加一个突变，不管这个刚开始这个序列上有没有其他突变，对splicing的影响都是一样的。这部分结果表明，<strong>Mutations have non-independent effects on splicing</strong>；<br><img src="/images/20190322-Fig4.jpg" alt="Figure 2"><br><img src="/images/20190322-Fig5.png" alt="Figure 2-legend"></li>
<li><strong>Figure 2B:</strong> 举个例子，在祖先基因型中又一个T19G突变，也有11个基因型中也有这个突变（与祖先1nt的不同），54个基因型中与祖先相比有2nt的不同，对应这个图的的灰色bar；</li>
<li><strong>Figure 2C:</strong> 所有的12个突变，当他们的starting genotypes不同时，效应都会发生很大的变化，但是在同一方向上（比如外显子包含水平不断增加），突变倾向于具有不同效应；</li>
<li><strong>Figure 2D:</strong> 作者在这里证明不同的genotypes对应的突变有不同的效应，量化祖先外显子的每个突变的影响并用具有12个参数的线性模型拟合，发现拟合效果很差，RMSE(root-mean-square-error)只有45；</li>
<li><strong>Figure 2E:</strong> 当考虑所有基因型中每个突变的平均效果时，预测性能大大提升，RMSE降低到了17.8，但仍然存在与实际值的重要偏差。</li>
</ul>
<h4 id="3-4-突变效应与起始包含水平是非单调的-Mutation-effects-scale-non-monotonically-with-starting-inclusion-levels"><a href="#3-4-突变效应与起始包含水平是非单调的-Mutation-effects-scale-non-monotonically-with-starting-inclusion-levels" class="headerlink" title="3.4 突变效应与起始包含水平是非单调的 (Mutation effects scale non-monotonically with starting inclusion levels)"></a>3.4 突变效应与起始包含水平是非单调的 (Mutation effects scale non-monotonically with starting inclusion levels)</h4><ul>
<li><strong>Figure 3A:</strong> 为了研究为什么单个突变在不同的数据集中有变化，在这部分作者分析了在突变加入之前与之后的外显子包含水平的变化，外显子包含的范围在0-100；</li>
<li><strong>Figure 3B-3C:</strong> 因此作者就假设了一系列的模型单调模型；<br><img src="/images/20190323-Fig2.jpg" alt="Figure 3"><br><img src="/images/20190323-Fig3.png" alt="Figure 3-legend"></li>
<li><strong>Figure 3D-3E:</strong> 但是，作者惊奇的发现，二者的关系并不是单调的，PSI首先增加到最大然后在开始降低。这一个细胞系当然不足以说明，作者又在其他三个细胞系中证明了这个结果；</li>
</ul>
<h4 id="3-5-数学建模表明竞争是突变效应中非单调性的起源-Mathematical-modeling-suggests-competition-as-the-origin-of-non-monotonicity-in-the-effects-of-mutations"><a href="#3-5-数学建模表明竞争是突变效应中非单调性的起源-Mathematical-modeling-suggests-competition-as-the-origin-of-non-monotonicity-in-the-effects-of-mutations" class="headerlink" title="3.5 数学建模表明竞争是突变效应中非单调性的起源 (Mathematical modeling suggests competition as the origin of non-monotonicity in the effects of mutations)"></a>3.5 数学建模表明竞争是突变效应中非单调性的起源 (Mathematical modeling suggests competition as the origin of non-monotonicity in the effects of mutations)</h4><ul>
<li>这部分作者使用泊松过程模拟突变效应的非单调性；<br><img src="/images/20190323-Fig4.png" alt="Figure 4ABC"><br><img src="/images/20190323-Fig5.png" alt="Figure 4DEFGH"></li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/03/22/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/">https://jingjingzhai.github.io/2019/03/22/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/可变剪切/">可变剪切</a><a href="/tags/进化/">进化</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/" class="next">Predicting splicing from primary sequence with deep learning</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-重新构建FAS外显子6的进化-Reconstructing-the-Evolution-of-FAS-exon-6"><span class="toc-text">3.1 重新构建FAS外显子6的进化 (Reconstructing the Evolution of FAS exon 6)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-组合完整的基因型空间定位-Combinatorially-complete-mapping-of-a-genotype-space"><span class="toc-text">3.2 组合完整的基因型空间定位 (Combinatorially complete mapping of a genotype space)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-突变对剪切有非独立的影响-Mutations-have-non-independent-effects-on-splicing"><span class="toc-text">3.3 突变对剪切有非独立的影响 (Mutations have non-independent effects on splicing)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-突变效应与起始包含水平是非单调的-Mutation-effects-scale-non-monotonically-with-starting-inclusion-levels"><span class="toc-text">3.4 突变效应与起始包含水平是非单调的 (Mutation effects scale non-monotonically with starting inclusion levels)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-5-数学建模表明竞争是突变效应中非单调性的起源-Mathematical-modeling-suggests-competition-as-the-origin-of-non-monotonicity-in-the-effects-of-mutations"><span class="toc-text">3.5 数学建模表明竞争是突变效应中非单调性的起源 (Mathematical modeling suggests competition as the origin of non-monotonicity in the effects of mutations)</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/03/22/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/">Predicting splicing from primary sequence with deep learning</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/">DeepTACT: Predicting 3D chromatin contacts via bootstrapping deep learning</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/13/Network-propagation-a-universal-amplifier-of-genetic-associations/">Network propagation: a universal amplifier of genetic associations</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/12/Functional-protein-representations-from-biological-networks-enable-diverse-cross-species-inference/">Functional protein representations from biological networks enable diverse cross-species inference (未完待续...)</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/09/A-transcription-factor-hierarchy-defines-an-environmental-stress-response-network/">A transcription factor hierarchy defines an environmental stress response network</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/08/Cistrome-and-epicistrome-features-shape-the-regulatory-DNA-landscape/">Cistrome and epicistrome features shape the regulatory DNA landscape (未完待续...)</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/07/Dysregulation-of-expression-correlates-with-rare-allele-burden-and-fitness-loss-in-maize/">Dysregulation of expression correlates with rare-allele burden and fitness  loss in maize (未完待续...)</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/07/Evolutionarily-informed-deep-learning-methods-forpredicting-relative-transcript-abundance-fromDNA-sequence/">Evolutionarily informed deep learning methods for predicting relative transcript abundance from DNA sequence</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/04/A-resource-of-variant-effect-predictions-of-SNV-in-model-organism/">A resource of variant effect predictions of single nucleotide variants in model organisms</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-reviews-genetics/">Nature reviews genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">1</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>