<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Deep-learning augmented RNA-seq analysis of transcript splicing | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Deep-learning augmented RNA-seq analysis of transcript splicing</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Deep-learning augmented RNA-seq analysis of transcript splicing</h1><div class="post-meta"><a href="/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/#comments" class="comment-count"></a><p><span class="date">Mar 27, 2019</span><span><a href="/categories/Nature-Methods/" class="category">Nature Methods</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是发表在Nature Methods上的使用深度学习来扩充可变剪切的分析。通讯作者是来自于UCLA的<a href="http://xinglab.github.io" target="_blank" rel="noopener">邢毅</a>老师。<a id="more"></a>这篇文章创新点在于将贝叶斯假设检验框架和深度学习结合在一起，首先使用贝叶斯假设检验来检验可变差异剪切，并用来作为训练模型的标签，然后根据深度学习预测的结果重新作为贝叶斯假设检验的先验概率从而估计后验概率；除此之外，作者针对于类别不平衡预测结果还做了rank-transformation。</p>
<p><a href="https://www.nature.com/articles/s41592-019-0351-9" target="_blank" rel="noopener">文章链接</a><br><a href="https://github.com/Xinglab/DARTS" target="_blank" rel="noopener">文章代码</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>可变剪切的RNA-Seq分析最大的局限在于非常依赖测序深度；</li>
<li>作者在这里提出DARTS, 通过整合先验的RNA-Seq证据以及深度学习预测结果来推断不同生物学样本中的差异可变剪切；<br>DARTS利用公共数据库中的大量的RNA-Seq数据通过深度学习提供可变剪切调节的knowledge base, 因此可以用来帮助研究人员即使在中等测序深度的情况下更好的刻画可变剪切。</li>
</ul>
<h3 id="2-结果"><a href="#2-结果" class="headerlink" title="2. 结果"></a>2. 结果</h3><h4 id="2-1-DARTS方法简介"><a href="#2-1-DARTS方法简介" class="headerlink" title="2.1 DARTS方法简介"></a>2.1 DARTS方法简介</h4><ul>
<li>RNA-Seq测序使得分析转录组水平的可变剪切成为可能，RNA-Seq的快速积累也为分析可变剪切提供了前所未有的机遇。但是可变剪切的分析却受限于测序深度。</li>
<li>因此作者提出了<strong>DARTS</strong>，如下图所示：DARTS由两个成份组成，DNN（Deep neural network）基于外显子特异性的序列特征以及样本特异性的调控特征预测两个条件是不是<strong>差异可变剪切</strong>；BHT(Bayesian hypothesis testing)统计模型通过整合特定RNA-Seq数据的经验证据（<strong>由DNN预测得来</strong>）与可变剪切的先验概率推断差异可变剪切；</li>
<li>在训练的时候，首先使用无信息先验模型只基于RNA-Seq数据生成高置信度差异以及非差异的标签，然后再用于训练，然后预测的结果又用于DARTS BHT（info），用预测结果作为先验；</li>
<li><strong>Figure 1B:</strong> 与其他方法不同的是，DARTS不仅使用了cis sequence还整合了两种情况下trans RNA-binding proteins(RBPs)的mRNA水平。这可以允许DARTS分析RBPs的表达如何影响剪切，作者使用了2926个序列特征以及1498个注释的RBPs的mRNA水平（trans RBP特征）；<br><img src="/images/20190327-Fig1.png" alt="Figure 1"><br><img src="/images/20190327-Fig2.png" alt="Figure 1-legend"></li>
<li><strong>Figure 1C:</strong> 作者使用了两个cell lines(K562和HepG2)对应的RBP-depletion RNA-seq，在两个cell lines中都有的196个 RBPs depleted by short-harpin RNA (shRNA), 对应着408对（knockdown-versus-control）;对于只在一个细胞系中出现的58组数据用于leave-out test;</li>
<li><strong>Figure 1D:</strong> DARTS在leave-out数据集上的表现，与其他模型进行比较；</li>
</ul>
<h4 id="2-2-DARTS-BHT框架的性能评估以及训练数据集对于DARTS-DNN的影响"><a href="#2-2-DARTS-BHT框架的性能评估以及训练数据集对于DARTS-DNN的影响" class="headerlink" title="2.2 DARTS BHT框架的性能评估以及训练数据集对于DARTS DNN的影响"></a>2.2 DARTS BHT框架的性能评估以及训练数据集对于DARTS DNN的影响</h4><ul>
<li><strong>Figure 2A:</strong> DARTS BHT (info)与DARTS BHT (flat)的性能比较, 测序深度越浅，性能提升的越显著；</li>
<li><strong>Figure 2B:</strong> DARTS在leave0out数据集上的预测效果；<br><img src="/images/20190327-Fig4.png" alt="Figure 2"></li>
</ul>
<h4 id="2-3-DARTS在上皮-间质转化中的可变剪切分析"><a href="#2-3-DARTS在上皮-间质转化中的可变剪切分析" class="headerlink" title="2.3 DARTS在上皮-间质转化中的可变剪切分析"></a>2.3 DARTS在上皮-间质转化中的可变剪切分析</h4><ul>
<li><strong>Figure 3A:</strong> 将DARTS DNN用于时序的RNA0Seq数据，底部的bar plot是无先验模型的结果，线图是DNN的预测结果；<br><img src="/images/20190327-Fig5.png" alt="Figure 3"><br><img src="/images/20190327-Fig6.png" alt="Figure 3-legend"></li>
</ul>
<h3 id="3-方法"><a href="#3-方法" class="headerlink" title="3. 方法"></a>3. 方法</h3><h4 id="3-1-DARTS贝叶斯假设检验框架"><a href="#3-1-DARTS贝叶斯假设检验框架" class="headerlink" title="3.1 DARTS贝叶斯假设检验框架"></a>3.1 DARTS贝叶斯假设检验框架</h4><ul>
<li>这部分使用贝叶斯假设检验来确定差异可变剪切事件，DARTS的核心是先利用贝叶斯假设检验确定样本的标签（<strong>differential splicing</strong> or <strong>unchanged splicing</strong>）,然后使用深度学习建模预测，把深度学习预测结果作为先验信息，再结合RNA-Seq数据（实验证据）得到样本最终的标签。以最简单的情况（检测两个条件下exon-inclusion levels; PSI values）为例, 认为在特定PSI的情况下，外显子保留的read count符合二项分布，其中\(n\)为外显子保留的read数目加上外显子跳跃read数目，概率\(p=f_i(\psi_{ij})\)，详情如下：</li>
<li><p><strong>首先声明：</strong>\(i\) 表示第\(i\)个外显子，\(j\in{1,2}\) 分别代表差异splicing和unchanged splicing；</p>
<script type="math/tex; mode=display">I_{ij}|\psi_{ij}\sim Binomial(n=I_{ij}+S_{ij},p=f_i(\psi_{ij}))</script><script type="math/tex; mode=display">\psi_{i1}=\mu_i</script><script type="math/tex; mode=display">\psi_{i2}=\mu_i+\delta_i</script><script type="math/tex; mode=display">\mu_i\sim Unif(0,1)</script><script type="math/tex; mode=display">\delta_i\sim N(0,\tau^2)</script></li>
<li><p>\(I_{ij}\): <strong>外显子保留的read count</strong>；</p>
</li>
<li>\(S_{ij}\): <strong>外显子跳跃的read count</strong>；</li>
<li>\(\psi_{ij}\): <strong>外显子 \(i\) 在sample group \(j\in{1,2}\)的外显子保留水平,也就是PSI值</strong>；</li>
<li>\(f_i\): <strong>外显子 \(i\) 的长度标准化函数，这个函数考虑了外显子保留和跳跃的有效长度</strong>；</li>
<li>\(\mu_i\): <strong>外显子 \(i\) 的baseline inclusion level</strong>；</li>
<li>\(\delta_i\): <strong>两个条件下期望的外显子保留水平之差</strong>；</li>
<li>差异可变剪切分析的目标就是检验两个条件下是否有很高的概率使得\(\delta_i\)大于指定的阈值\(c\) (例如：0.05)。对应如下公式：<script type="math/tex; mode=display">p(|\delta_i|>c|I_{ij},S_{ij})\approx1</script></li>
<li><p>在贝叶斯统计中，可以通过使用参数\(\delta\)的<a href="https://www.zhihu.com/question/51980224" target="_blank" rel="noopener">spike-and-slab先验</a>来估计结果。spike-and-slab先验是双组分混合分布，其中“spike”描绘了模型参数\(\delta\)被约束在零附近的概率，而“slab”分量描绘了模型参数\(\delta\)的无约束分布。</p>
</li>
<li><p>在贝叶斯统计框架中，为了考虑到随机的技术干扰对PSI值的影响，作者利用了<strong>spike prior \(H_0\) with a small variance \(\tau=\tau_0\)</strong>; <strong>slab prior \(H_1\) with a small variance \(\tau=\tau_1\)</strong>。设置\(\tau0=0.03\), 对应着90%的密度约束在\(\delta\in[-0.05,0.05]\), \(\tau1=0.3\), 因此后验概率可以写做：</p>
<script type="math/tex; mode=display">p(H_1|I_{ij},S_{ij})=\frac1Zp(H_1)*p(I_{ij},S_{ij}|H_1)</script><script type="math/tex; mode=display">p(I_{ij},S_{ij}|H_1)=\int_\tau\int_\mu p(I_{ij},S_{ij}|\mu_i,\delta_i)*p(\mu_i,\delta_i|H_1)d\mu_id\delta_i</script><script type="math/tex; mode=display">p(H_0|I_{ij},S_{ij})=\frac1Zp(H_0)*p(I_{ij},S_{ij}|H_0)</script><script type="math/tex; mode=display">p(I_{ij},S_{ij}|H_0)=\int_\tau\int_\mu p(I_{ij},S_{ij}|\mu_i,\delta_i)*p(\mu_i,\delta_i|H_0)d\mu_id\delta_i</script></li>
<li><p><strong>说明：</strong> </p>
<ul>
<li>\(p(H_1)\)：<strong>外显子\(i\)差异剪切的先验概率，在模型训练之前这个采用无信息先验分布，训练模型之后用预测分值作为先验信息, 因此这里还是无信息先验</strong>；</li>
<li>\(p(H_0)=1-p(H_1)\): <strong>外显子\(i\)unchanged的先验概率</strong>；</li>
<li>\(p(I_{ij},S_{ij}|H_1)\)和\(p(I_{ij},S_{ij}|H_0)\)分别表示在<strong>differential splicing</strong> and <strong>unchanged splicing</strong>模型下的likelihoods；</li>
<li>\(Z\)：标准常数；</li>
</ul>
</li>
<li>由于是比较两个模型，因此又可以写成以下形式：<script type="math/tex; mode=display">\frac{p(H_1|I_{ij},S_{ij})}{p(H_0|I_{ij},S_{ij})}=\frac{p(H_1)}{p(H_0)}*\frac{p(I_{ij},S_{ij}|H_1)}{p(I_{ij},S_{ij}|H_0)}</script></li>
<li>由于采用<strong>无信息先验</strong>，所以：\(p(H_0)=p(H_1)=0.5\); 因此上式<strong>等价于</strong>似然比检验，这里作者称为DARTS BHT (flat); 当整合基于深度学习的预测结果作为先验信息时，作者称为DARTS BHT (info)。</li>
<li>最终，使用上述公式，我们可以得到参数\(\delta_i\)的边际后验概率\(p(\delta_i|I_{ij},S_{ij})\)，即两个模型的后验混合：<script type="math/tex; mode=display">p(\delta_i|I_{ij},S_{ij})=p(\delta_i|H_1,I_{ij},S_{ij})*p(H_1|I_{ij},S_{ij})+p(\delta_i|H_0,I_{ij},S_{ij})*p(H_0|I_{ij},S_{ij})</script></li>
<li>最终就是推断\(p(|\delta_i|&gt;c|I_{ij},S_{ij})\), 作者在分析中，设置\(c=0.05\), 即：外显子保留水平有5%的变化。认为\(p(|\delta_i|&gt;0.05|I_{ij},S_{ij})&gt;0.9\)为显著差异剪切事件；\(p(|\delta_i|&gt;0.05|I_{ij},S_{ij})&lt;0.1\)为显著的unchanged splicing事件。</li>
</ul>
<h4 id="3-2-DARTS预测差异可变剪切的深度神经网络模型-DARTS-DNN-model-for-predicting-differential-alternative-splicing"><a href="#3-2-DARTS预测差异可变剪切的深度神经网络模型-DARTS-DNN-model-for-predicting-differential-alternative-splicing" class="headerlink" title="3.2 DARTS预测差异可变剪切的深度神经网络模型(DARTS DNN model for predicting differential alternative splicing)"></a>3.2 DARTS预测差异可变剪切的深度神经网络模型(DARTS DNN model for predicting differential alternative splicing)</h4><ul>
<li>DARTS的一个核心就是利用DNN检测两个条件下一个特定的外显子是不是差异剪切，这里作者利用了两个层面的特征，第一个是序列层面的特征，第二个是在两个条件下样本特异性的反式RBP表达水平的特征，因此预测问题可以表述为以下公式：<script type="math/tex; mode=display">p(Y_{ik}=1)=F(Y_{ik};E_i,G_k)</script></li>
<li>其中\(Y_{ik}\)就是外显子\(i\)在比较\(k\)中的标签；\(E_i\)是一个向量，包含2936个进化保守型特征、2973个剪切点的长度、2971个调控motif的组成以及1748个RNA二级结构特征（ skipped exons, alternative 5′ splice sites, alternative 3′ splice sites, and retained introns）；而\(G_k\)是2996维的向量代表1498个RBPs在两个条件下的标准化基因表达量；DNN的预测结果可以背进一步整合为贝叶斯假设检验框架中的先验信息；</li>
<li>模型由4个全连接隐层分别包含1200，500，300和200个神经元均使用ReLu激活函数；RMSprop优化，minibatch设置为1000，加入Dropout防止过拟合。</li>
</ul>
<h4 id="3-3-RNA-seq数据处理以及DARTS-DNN模型的训练"><a href="#3-3-RNA-seq数据处理以及DARTS-DNN模型的训练" class="headerlink" title="3.3 RNA-seq数据处理以及DARTS DNN模型的训练"></a>3.3 RNA-seq数据处理以及DARTS DNN模型的训练</h4><ul>
<li>使用rMATS处理bam文件检测可变剪切；</li>
<li>Kallisto定基因表达量，使用Gencode生成index；</li>
<li>使用DARTS BHT(flat)生成训练标签，后验概率\(p(|\Delta\psi|&gt;0.05)&gt;0.9\)作为正样本；后验概率\(p(|\Delta\psi|&gt;0.05)&lt;0.1\)作为负样本；</li>
<li>10%的样本用于测试数据，剩下90%数据用于五倍交叉验证；</li>
<li>同时也收集了只在一个细胞系中存在的ENCODE RBP-knockdown实验作为leave-out datasets来评估模型的精度；</li>
</ul>
<h4 id="3-4-DARTS信息先验的Rank-transformation"><a href="#3-4-DARTS信息先验的Rank-transformation" class="headerlink" title="3.4 DARTS信息先验的Rank-transformation"></a>3.4 DARTS信息先验的Rank-transformation</h4><ul>
<li>由于unchanged splicing要比differential splicing大几个数量级，因此机器学习算法会偏向于类别多的样本，作者为了解决这个问题，使用了非监督的rank-tranformation重新scale DARTS DNN的预测分值来获得DARTS BHT框架的先验信息；</li>
<li>具体来说，作者将所有DARTS DNN的预测分值拟合成的双峰（两个component）的高斯混合模型，然后得出两个混合高斯峰的均值和方差，以及属于特定成分的每个DARTS DNN得分的后验概率\(\lambda\)；</li>
<li>每个高斯成分的均值和方差分别设置为: \(\mu_0\), \(\mu_1\), \(\sigma_0\), \(\sigma_1\)，然后每个原始预测分值通过rank-transformed成新的高斯成分然后通过权重参数\(\lambda\)加权平均；</li>
<li>最后，为了保持有限的先验概率，把transformed DARTS DNN score scale到\([\alpha, 1-\alpha]\), 其中\(\alpha\in[0,0.5)\)；</li>
<li>在实际操作中，用到的参数设置为：\(\mu_0=0.05\), \(\mu_1=0.95\), \(\sigma_0=\sigma_1=0.1\), \(\alpha=0.05\)。</li>
</ul>
</div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/">https://jingjingzhai.github.io/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/深度学习/">深度学习</a><a href="/tags/可变剪切/">可变剪切</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/" class="pre">Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation</a><a href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/" class="next">Predicting splicing from primary sequence with deep learning</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-结果"><span class="toc-text">2. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#2-1-DARTS方法简介"><span class="toc-text">2.1 DARTS方法简介</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#2-2-DARTS-BHT框架的性能评估以及训练数据集对于DARTS-DNN的影响"><span class="toc-text">2.2 DARTS BHT框架的性能评估以及训练数据集对于DARTS DNN的影响</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#2-3-DARTS在上皮-间质转化中的可变剪切分析"><span class="toc-text">2.3 DARTS在上皮-间质转化中的可变剪切分析</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-方法"><span class="toc-text">3. 方法</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-DARTS贝叶斯假设检验框架"><span class="toc-text">3.1 DARTS贝叶斯假设检验框架</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-DARTS预测差异可变剪切的深度神经网络模型-DARTS-DNN-model-for-predicting-differential-alternative-splicing"><span class="toc-text">3.2 DARTS预测差异可变剪切的深度神经网络模型(DARTS DNN model for predicting differential alternative splicing)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-RNA-seq数据处理以及DARTS-DNN模型的训练"><span class="toc-text">3.3 RNA-seq数据处理以及DARTS DNN模型的训练</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-4-DARTS信息先验的Rank-transformation"><span class="toc-text">3.4 DARTS信息先验的Rank-transformation</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/04/03/Dysregulation-of-expression-correlates-with-rare-allele-burden-and-fitness-loss-in-maize/">Dysregulation of expression correlates with rare-allele burden and fitness  loss in maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/03/DeepCAPE-a-deep-convolutional-neural-network-for-the-accurate-prediction-of-enhancers/">DeepCAPE: a deep convolutional neural network for the accurate prediction of enhancers</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/">Comprehensive functional genomic resource and integrative model for the human brain</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/01/A-universal-sNP-and-small-indel-variant-caller-using-deep-neural-networks/">A universal SNP and small-indel variant caller using deep neural networks</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/31/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/">Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/">Deep-learning augmented RNA-seq analysis of transcript splicing</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/20/Predicting-splicing-from-primary-sequence-with-deep-learning/">Predicting splicing from primary sequence with deep learning</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/18/Predicting 3D chromatin contacts via bootstrapping deep learning/">DeepTACT: Predicting 3D chromatin contacts via bootstrapping deep learning</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/13/Network-propagation-a-universal-amplifier-of-genetic-associations/">Network propagation: a universal amplifier of genetic associations</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">1</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>