<!DOCTYPE html><html lang="zh-CN"><head><meta name="generator" content="Hexo 3.8.0"><meta http-equiv="content-type" content="text/html; charset=utf-8"><meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" name="viewport"><meta content="yes" name="apple-mobile-web-app-capable"><meta content="black-translucent" name="apple-mobile-web-app-status-bar-style"><meta content="telephone=no" name="format-detection"><meta name="description"><title>Evolutionarily informed deep learning methods for predicting relative transcript abundance from DNA sequence | 学而不思则惘</title><link rel="stylesheet" type="text/css" href="//fonts.loli.net/css?family=Source+Code+Pro"><link rel="stylesheet" type="text/css" href="/css/style.css?v=2.0.3"><link rel="stylesheet" type="text/css" href="/css/highlight.css?v=2.0.3"><link rel="Shortcut Icon" href="/favicon.ico"><link rel="bookmark" href="/favicon.ico"><link rel="apple-touch-icon" href="/apple-touch-icon.png"><link rel="apple-touch-icon-precomposed" href="/apple-touch-icon.png"><link rel="alternate" type="application/atom+xml" href="/atom.xml"></head><body><div class="body_container"><div id="header"><div class="site-name"><h1 class="hidden">Evolutionarily informed deep learning methods for predicting relative transcript abundance from DNA sequence</h1><a id="logo" href="/.">学而不思则惘</a><p class="description"></p></div><div id="nav-menu"><a href="/." class="current"><i class="fa fa-home"> 首页</i></a><a href="/archives/"><i class="fa fa-archive"> 归档</i></a><a href="/about/"><i class="fa fa-user"> 关于</i></a><a href="/atom.xml"><i class="fa fa-rss"> 订阅</i></a></div><div id="search-form"><div id="result-mask" class="hide"></div><label><input id="search-key" type="text" autocomplete="off" placeholder="Arama"></label><div id="result-wrap" class="hide"><div id="search-result"></div></div><div class="hide"><template id="search-tpl"><div class="item"><a href="/{path}" title="{title}"><div class="title">{title}</div><div class="time">{date}</div><div class="tags">{tags}</div></a></div></template></div></div></div><div id="layout" class="layout-g"><div class="layout-l"><div class="content_container"><div class="post"><h1 class="post-title">Evolutionarily informed deep learning methods for predicting relative transcript abundance from DNA sequence</h1><div class="post-meta"><a href="/2019/03/07/Evolutionarily-informed-deep-learning-methods-forpredicting-relative-transcript-abundance-fromDNA-sequence/#comments" class="comment-count"></a><p><span class="date">Mar 07, 2019</span><span><a href="/categories/PNAS/" class="category">PNAS</a></span><span><i id="busuanzi_container_page_pv"><i id="busuanzi_value_page_pv"></i><i>点击</i></i></span></p></div><div class="post-content"><h3 id="0-简介"><a href="#0-简介" class="headerlink" title="0. 简介"></a>0. 简介</h3><p>这篇文章是康奈尔大学<strong>Edward S. Buckler</strong>课题组在PNAS发表的关于预测相对表达量的文章，虽然题目说是深度学习方法，但其实本文最大的亮点在于训练数据集、测试数据集的分割。作者认为，由于生物体在进化过程中，基因之间会共享一些进化信息，因此在分割数据集的时候不能像CV领域里一样，将数据集随机分割，这样容易导致过拟合。因此这篇文章就把以预测转录本相对表达量为例，在分割数据集的时候考虑到进化信息（也就是基因家族）。至于在模型方法方面，还是采用传统的CNN。<br><a id="more"></a><br><a href="https://www.pnas.org/content/early/2019/03/05/1814551116" target="_blank" rel="noopener">文章链接</a></p>
<h3 id="1-摘要"><a href="#1-摘要" class="headerlink" title="1. 摘要"></a>1. 摘要</h3><ul>
<li>深度学习在许多领域彻底改变了预测，并且有着与遗传学与分子生物学同样的潜力。然而，以当前形式应用这些方法忽略了生物系统内的进化依赖性，因此可能导致假阳性以及错误的结论；</li>
<li>作者因此开发了两种考虑了进化相关的机器学习模型：（i）gene-family-guided splitting; (ii）ortholog contrasts。第一种方法通过在训练集与测试集中包含不同家族信息来约束模型；第二种方法使用直系同源基因之间的进化知识比较来控制和利用训练过程中的进化分歧。</li>
<li>用mRNA表达水平预测这两种方法，得到auROC值在0.75至0.94的范围内。模型权重显示了可解释的生物学模式，导致假设3’UTR对于微调mRNA丰度水平更为重要，而5’UTR对于大规模变化更为重要。</li>
</ul>
<h3 id="2-前言"><a href="#2-前言" class="headerlink" title="2. 前言"></a>2. 前言</h3><ul>
<li>深度学习尤其是CNN在图像识别以及NLP领域都取得了很大进步，这些方法也随后被应用于分子生物学、基因组学的研究当中，但是进化关系使得在生物学中正确训练和测试模型比上面提到的图像或文本分类问题更具挑战性；</li>
<li>举个例子，如果一个人想预测mRNA的表达水平，标准的图像识别方法就是将数据集随机分成Training、Validation、Testing，但是由于基因之间的共享的进化历史（基因家族相关性、基因重复等），随机分会导致各组数据之间有依赖，测试数据集不够独立。因此没有考虑进化关系的模型就会很容易出现过拟合；</li>
<li>因此作者就提出了两个基于CNN的方法（如下<strong>Figure 1A</strong>所示，但其实就是关于数据集的分割不同）。第一种方法，作者起名为<strong>gene-family-guided splitting</strong>,实则就是在分割训练集与测试集的时候将同一家族的基因分到一起，这样训练集与测试集合不存在基因家族的overlap；第二种方法为<strong>ortholog contrasts</strong>, 如<strong>Figure 1B</strong>所示，这种方法主要针对于有两个物种的预测问题，就将直系同源的基因放到一起。<br><img src="/images/20190307-Fig1.png" alt="Figure 1"></li>
</ul>
<h3 id="3-结果"><a href="#3-结果" class="headerlink" title="3. 结果"></a>3. 结果</h3><h4 id="3-1-基于DNA序列区分表达基因与为表达基因-Differentiating-Between-Expressed-and-Unexpressed-Genes-Based-on-DNA-Sequence"><a href="#3-1-基于DNA序列区分表达基因与为表达基因-Differentiating-Between-Expressed-and-Unexpressed-Genes-Based-on-DNA-Sequence" class="headerlink" title="3.1 基于DNA序列区分表达基因与为表达基因 (Differentiating Between Expressed and Unexpressed Genes Based on DNA Sequence)"></a>3.1 基于DNA序列区分表达基因与为表达基因 (Differentiating Between Expressed and Unexpressed Genes Based on DNA Sequence)</h4><ul>
<li>这部分作者构建了一个模型用来区分基因是expressed or unexpressed, 也就是最常见的二分类模型，在训练的时候采取了图1中的第一种方法。<br><img src="/images/20190307-Fig2.png" alt="Figure 2"></li>
<li><strong>Figure 2A:</strong> 作者首先构造了一个叫做<strong>pseudogene</strong>的模型，也基因的上下游1k的序列作为模型的输入；</li>
<li><strong>Figure 2B:</strong> 作者首先对玉米中所有的基因进行分类，这里是用了7篇文章中的422个组织的RNA-Seq数据，红色、绿色和蓝色分别代表高表达、中等表达和不表达；</li>
<li><strong>Figure 2C:</strong> 作者<strong>Figure 2B</strong>中基因的分类，分别训练了两个<strong>pseudogene</strong>模型，一个是<strong>Off/On</strong>, 也就是用中等表达和高表达作为正样本；另外一个是<strong>Off/High</strong>，用高表达的作为正样本，使用downsampling使得正负样本比例1:1。随机十次五折交叉验证来评估模型的准确性；</li>
<li>同时作者在补充材料<strong>Figure S1</strong>中证明了如果采用随机样本分割，auROC和accuracy都会提升，进一步反映了之前的基于随机分割的方法会造成over-fitting。</li>
</ul>
<h4 id="3-2-使用直系同源对比预测两个基因中的哪一个更高表达-Predicting-Which-of-Two-Genes-Is-More-Highly-Expressed-Using-Ortholog-Contrasts"><a href="#3-2-使用直系同源对比预测两个基因中的哪一个更高表达-Predicting-Which-of-Two-Genes-Is-More-Highly-Expressed-Using-Ortholog-Contrasts" class="headerlink" title="3.2 使用直系同源对比预测两个基因中的哪一个更高表达 (Predicting Which of Two Genes Is More Highly Expressed Using Ortholog Contrasts)"></a>3.2 使用直系同源对比预测两个基因中的哪一个更高表达 (Predicting Which of Two Genes Is More Highly Expressed Using Ortholog Contrasts)</h4><ul>
<li>这部分主要用来讲述如何预测直系同源的两个基因哪个基因的表达量更高，当然我并没有理解预测这个有啥用，附上文章原话<strong>The ortholog contrast model follows a simple approach derived from phylogenetics, where the most recent common ancestor of two closely related genes can be represented as a contrast between the two. Contrasting genes in this manner directly accounts for statistical dependencies between the genes that would otherwise hamper comparison with other genes. Building on this idea, the ortholog contrast method com- pares two genes from different genomes (or alleles from the same species) to each other and predicts the difference between the expression levels of the two</strong>；<br><img src="/images/20190307-Fig3.png" alt="Figure 3"></li>
<li><strong>Figure 3A:</strong> 作者在这里仍然使用一个二进制模型，1代表gene1表达比gene2高，0相反，训练的时候去掉表达为0的基因；</li>
<li><strong>Figure 3B:</strong> 同样适用accuracy和auROC来比较不同特征是模型的精度；</li>
</ul>
<h4 id="3-3-对CNN模型的解读揭示了对转录本丰度预测的重要特征和元件-Interpretation-of-CNN-Models-Reveals-Elements-and-Motifs-Important-for-Transcript-Abundance"><a href="#3-3-对CNN模型的解读揭示了对转录本丰度预测的重要特征和元件-Interpretation-of-CNN-Models-Reveals-Elements-and-Motifs-Important-for-Transcript-Abundance" class="headerlink" title="3.3 对CNN模型的解读揭示了对转录本丰度预测的重要特征和元件 (Interpretation of CNN Models Reveals Elements and Motifs Important for Transcript Abundance)"></a>3.3 对CNN模型的解读揭示了对转录本丰度预测的重要特征和元件 (Interpretation of CNN Models Reveals Elements and Motifs Important for Transcript Abundance)</h4><ul>
<li>前面的分析都比较套路，这部分作者主要根据CNN中每个神经元的权重分析哪些特征对于预测转录本的相对丰度很重要。类似于传统机器学习分析中的feature importance；首先目前的模型并不能区分转录本丰度是由于合成还是降解导致的，但是未来如果用GRO-Seq，PRO-Seq等数据训练，将可能实现这个目标。</li>
</ul>
<p><img src="/images/20190307-Fig4.png" alt="Figure 5"></p>
<ul>
<li><strong>Figure 5A-5B:</strong> 作者在这部分用了两种CNN可视化的方法，一个是<a href="https://arxiv.org/abs/1704.02685" target="_blank" rel="noopener">DeepLIFT</a>和<a href="https://arxiv.org/abs/1312.6034" target="_blank" rel="noopener">Occlusion</a>.因此这个图展示的就是用promoter构建的pseudo模型中3‘UTR的贡献以及terminator构建模型中5’UTR的贡献，二者是互补的。A图<strong>Off/On</strong>模型；B图是<strong>Off/High</strong>模型；而且nonpseudogene的promoter信号比terminator更强，这也与<strong>Figure 2</strong>结论一致（单独基于promoter序列预测比单独基于terminator序列预测效果更好）。</li>
</ul>
<p><img src="/images/20190307-Fig5.png" alt="Figure 6"></p>
<ul>
<li><strong>Figure 6:</strong> 接下来是<strong>ortholog contrasts</strong>模型的可视化，与上一个不同，terminator区域表现出更强的信号；</li>
</ul>
<h3 id="4-讨论"><a href="#4-讨论" class="headerlink" title="4. 讨论"></a>4. 讨论</h3><h4 id="4-1-3’UTR对于RNA丰度的小范围变化可能更重要"><a href="#4-1-3’UTR对于RNA丰度的小范围变化可能更重要" class="headerlink" title="4.1 3’UTR对于RNA丰度的小范围变化可能更重要"></a>4.1 3’UTR对于RNA丰度的小范围变化可能更重要</h4><h4 id="4-2-不同模式和训练方法的优点和缺点"><a href="#4-2-不同模式和训练方法的优点和缺点" class="headerlink" title="4.2 不同模式和训练方法的优点和缺点"></a>4.2 不同模式和训练方法的优点和缺点</h4><h4 id="4-3-结论与展望"><a href="#4-3-结论与展望" class="headerlink" title="4.3 结论与展望"></a>4.3 结论与展望</h4></div><div class="post-copyright"><blockquote><p>原文作者: 翟晶晶</p><p>原文链接: <a href="https://jingjingzhai.github.io/2019/03/07/Evolutionarily-informed-deep-learning-methods-forpredicting-relative-transcript-abundance-fromDNA-sequence/">https://jingjingzhai.github.io/2019/03/07/Evolutionarily-informed-deep-learning-methods-forpredicting-relative-transcript-abundance-fromDNA-sequence/</a></p><p>版权声明: 转载请注明出处(必须保留作者署名及链接)</p></blockquote></div><div class="tags"><a href="/tags/深度学习/">深度学习</a><a href="/tags/Transcript-abundance-prediction/">Transcript abundance prediction</a></div><div class="post-share"><div class="social-share"><span>分享到:</span></div></div><div class="post-nav"><a href="/2019/03/08/Cistrome-and-epicistrome-features-shape-the-regulatory-DNA-landscape/" class="pre">Cistrome and epicistrome features shape the regulatory DNA landscape (未完待续...)</a><a href="/2019/03/04/A-resource-of-variant-effect-predictions-of-SNV-in-model-organism/" class="next">A resource of variant effect predictions of single nucleotide variants in model organisms</a></div><div id="comments"><div id="lv-container" data-id="city" data-uid="MTAyMC80MzIwNy8xOTc1Mw=="></div></div></div></div></div><div class="layout-r"><div id="sidebar"><div class="search-pla"></div><div id="toc" class="widget"><div class="widget-title"><i class="fa fa-fei">文章目录</i></div><ol class="toc"><li class="toc-item toc-level-3"><a class="toc-link" href="#0-简介"><span class="toc-text">0. 简介</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#1-摘要"><span class="toc-text">1. 摘要</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#2-前言"><span class="toc-text">2. 前言</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#3-结果"><span class="toc-text">3. 结果</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#3-1-基于DNA序列区分表达基因与为表达基因-Differentiating-Between-Expressed-and-Unexpressed-Genes-Based-on-DNA-Sequence"><span class="toc-text">3.1 基于DNA序列区分表达基因与为表达基因 (Differentiating Between Expressed and Unexpressed Genes Based on DNA Sequence)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-2-使用直系同源对比预测两个基因中的哪一个更高表达-Predicting-Which-of-Two-Genes-Is-More-Highly-Expressed-Using-Ortholog-Contrasts"><span class="toc-text">3.2 使用直系同源对比预测两个基因中的哪一个更高表达 (Predicting Which of Two Genes Is More Highly Expressed Using Ortholog Contrasts)</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#3-3-对CNN模型的解读揭示了对转录本丰度预测的重要特征和元件-Interpretation-of-CNN-Models-Reveals-Elements-and-Motifs-Important-for-Transcript-Abundance"><span class="toc-text">3.3 对CNN模型的解读揭示了对转录本丰度预测的重要特征和元件 (Interpretation of CNN Models Reveals Elements and Motifs Important for Transcript Abundance)</span></a></li></ol></li><li class="toc-item toc-level-3"><a class="toc-link" href="#4-讨论"><span class="toc-text">4. 讨论</span></a><ol class="toc-child"><li class="toc-item toc-level-4"><a class="toc-link" href="#4-1-3’UTR对于RNA丰度的小范围变化可能更重要"><span class="toc-text">4.1 3’UTR对于RNA丰度的小范围变化可能更重要</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-2-不同模式和训练方法的优点和缺点"><span class="toc-text">4.2 不同模式和训练方法的优点和缺点</span></a></li><li class="toc-item toc-level-4"><a class="toc-link" href="#4-3-结论与展望"><span class="toc-text">4.3 结论与展望</span></a></li></ol></li></ol></div><div class="widget"><div class="widget-title"><i class="fa fa-xie"> 最新文章</i></div><ul class="post-list"><li class="post-list-item"><a class="post-list-link" href="/2019/04/15/Single-cell-RNA-seq-denoising-using-a-deep-count-autoencoder/">Single-cell RNA-seq denoising using a deep count autoencoder</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/12/Opportunities-and-challenges-for-transcriptome-wide-association-studies/">Opportunities and challenges for transcriptome- wide association studies</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/09/Virtual-ChIP-seq-predicting-transcription-factor-binding-by-learning-from-the-transcriptome/">Virtual ChIP-seq: predicting transcription factor binding by learning from the transcriptome</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/03/Dysregulation-of-expression-correlates-with-rare-allele-burden-and-fitness-loss-in-maize/">Dysregulation of expression correlates with rare-allele burden and fitness  loss in maize</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/03/DeepCAPE-a-deep-convolutional-neural-network-for-the-accurate-prediction-of-enhancers/">DeepCAPE: a deep convolutional neural network for the accurate prediction of enhancers</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/01/Comprehensive-functional-genomicresource-and-integrative-model-forthe-human-brain/">Comprehensive functional genomic resource and integrative model for the human brain</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/04/01/A-universal-sNP-and-small-indel-variant-caller-using-deep-neural-networks/">A universal SNP and small-indel variant caller using deep neural networks</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/31/Combinatorial-Genetics-Reveals-a-Scaling-Law-for-the-Effects-of-Mutations-on-Splicing/">Combinatorial Genetics Reveals a Scaling Law for the Effects of Mutations on Splicing</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/29/Phylogenomic-synteny-network-analysis-of-MADS-Box-transcription-factor-genes-reveals-lineage-specific-transpositions-ancient-tandem-duplications-and-deep-positional-conservation/">Phylogenomic synteny network analysis of MADS-Box transcription factor genes reveals lineage-specific transpositions ancient tandem duplications and deep positional conservation</a></li><li class="post-list-item"><a class="post-list-link" href="/2019/03/27/Deep-learning-augmented-RNA-seq-analysis-of-transcript-splicing/">Deep-learning augmented RNA-seq analysis of transcript splicing</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-gui"> 期刊</i></div><ul class="category-list"><li class="category-list-item"><a class="category-list-link" href="/categories/Cell/">Cell</a><span class="category-list-count">3</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Biology/">Genome Biology</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Genome-Research/">Genome Research</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/MSB/">MSB</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/NAR/">NAR</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature/">Nature</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Biotechnology/">Nature Biotechnology</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Communications/">Nature Communications</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Genetics/">Nature Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Methods/">Nature Methods</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Nature-Reviews-Genetics/">Nature Reviews Genetics</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/PNAS/">PNAS</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/Science/">Science</a><span class="category-list-count">2</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/The-Plant-Cell/">The Plant Cell</a><span class="category-list-count">1</span></li><li class="category-list-item"><a class="category-list-link" href="/categories/bioRxiv/">bioRxiv</a><span class="category-list-count">2</span></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-biao"> 标签</i></div><div class="tagcloud"><a href="/tags/Enhancers-prediction/" style="font-size: 15px;">Enhancers prediction</a> <a href="/tags/SNP-Annotation/" style="font-size: 15px;">SNP Annotation</a> <a href="/tags/SNP-Calling/" style="font-size: 15px;">SNP Calling</a> <a href="/tags/深度学习/" style="font-size: 15px;">深度学习</a> <a href="/tags/可变剪切/" style="font-size: 15px;">可变剪切</a> <a href="/tags/进化/" style="font-size: 15px;">进化</a> <a href="/tags/Human-brain/" style="font-size: 15px;">Human brain</a> <a href="/tags/Maize/" style="font-size: 15px;">Maize</a> <a href="/tags/Arabidopsis-TF/" style="font-size: 15px;">Arabidopsis TF</a> <a href="/tags/Transcript-abundance-prediction/" style="font-size: 15px;">Transcript abundance prediction</a> <a href="/tags/TFBS-Prediction/" style="font-size: 15px;">TFBS Prediction</a> <a href="/tags/Network/" style="font-size: 15px;">Network</a> <a href="/tags/TWAS/" style="font-size: 15px;">TWAS</a> <a href="/tags/Synteny-Network/" style="font-size: 15px;">Synteny Network</a> <a href="/tags/scRNA-Seq/" style="font-size: 15px;">scRNA-Seq</a> <a href="/tags/chromatin-contacts-prediction/" style="font-size: 15px;">chromatin contacts prediction</a></div></div><div class="widget"><div class="widget-title"><i class="fa fa-archive"> 归档</i></div><ul class="archive-list"><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/04/">四月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/03/">三月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2019/02/">二月 2019</a></li><li class="archive-list-item"><a class="archive-list-link" href="/archives/2018/11/">十一月 2018</a></li></ul></div><div class="widget"><div class="widget-title"><i class="fa fa-you"> 友情链接</i></div><ul></ul><a href="http://bioinfo.nwafu.edu.cn" title="malab" target="_blank">malab</a><ul></ul><a href="http://github.com/cma2015" title="Github" target="_blank">Github</a><ul></ul><a href="https://stackoverflow.com" title="Stackoverflow" target="_blank">Stackoverflow</a><ul></ul><a href="https://www.bioinfo.info" title="生信坑" target="_blank">生信坑</a><ul></ul><a href="https://www.researchgate.net" title="ResearchGate" target="_blank">ResearchGate</a><ul></ul><a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64" title="UCSC tools" target="_blank">UCSC tools</a><ul></ul><a href="http://zh.gluon.ai" title="动手学深度学习" target="_blank">动手学深度学习</a></div></div></div></div><a id="totop" href="#top"></a><div id="footer"><div class="footer-info"><p><a href="/baidusitemap.xml">网站地图</a> |  <a href="/atom.xml">订阅本站</a> |  <a href="/about/">联系博主</a></p><p>本站总访问量：<i id="busuanzi_container_site_pv"><i id="busuanzi_value_site_pv"></i></i>次，本站总访客数:<i id="busuanzi_container_site_uv"><i id="busuanzi_value_site_uv"></i></i>人</p><p><span> Copyright &copy;<a href="/." rel="nofollow">翟晶晶.</a></span><span> Theme by<a rel="nofollow" target="_blank" href="https://github.com/chaooo/hexo-theme-BlueLake"> BlueLake.</a></span><span> Count by<a href="http://busuanzi.ibruce.info/"> busuanzi.</a></span><span> Powered by<a rel="nofollow" target="_blank" href="https://hexo.io"> Hexo.</a></span></p></div></div></div><script src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js" async></script><script type="text/javascript" src="/js/search.json.js?v=2.0.3"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  }
});</script><script type="text/javascript" src="//cdn.bootcss.com/mathjax/2.7.1/latest.js?config=TeX-AMS-MML_HTMLorMML" async></script><script type="text/javascript" src="/js/toctotop.js?v=2.0.3" async></script><link rel="stylesheet" type="text/css" href="/share/css/share.css"><script type="text/javascript" src="/share/js/social-share.js" charset="utf-8"></script><script type="text/javascript" src="/share/js/qrcode.js" charset="utf-8"></script><script>(function(d, s) {
  var j, e = d.getElementsByTagName('body')[0];
  if (typeof LivereTower === 'function') { return; }
  j = d.createElement(s);
  j.src = 'https://cdn-city.livere.com/js/embed.dist.js';
  j.async = true;
  e.appendChild(j);
})(document, 'script');
</script></body></html>